---
title: Liver and muscle circadian clocks cooperate to support glucose tolerance in mice
source: https://www.cell.com/cell-reports/fulltext/S2211-1247(23)00599-5
author:
  - "[[Permissions]]"
  - "[[Information for authors]]"
  - "[[STAR Methods]]"
  - "[[Search for articles by this author]]"
  - "[[Highlights]]"
  - "[[Summary]]"
  - "[[Jacob G. Smith]]"
  - "[[jacob.smith@upf.edu]]"
  - "[[1]]"
  - "[[2]]"
published:
created: 2026-01-18
description: Smith et al. rescued circadian clock function in liver and muscle in otherwise clock-lessmice, finding that glucose tolerance is achieved only when feeding rhythms and Bmal1function in liver and muscle are engaged simultaneously. The authors posit that disruptionof this spatiotemporal mechanism contributes to circadian-disruption-induced metabolicdisease.
tags:
  - clippings
  - sleep
---
## Highlights

- Muscle *Bmal1* is partially sufficient to drive local glucose metabolism
- Diurnal transcriptomes of liver and muscle are largely independent
- Even in combination, liver and muscle clocks do not impart systemic glucose tolerance
- Synergy between feeding rhythms and liver and muscle clocks enables glucose tolerance

## Summary

Physiology is regulated by interconnected cell and tissue circadian clocks. Disruption of the rhythms generated by the concerted activity of these clocks is associated with metabolic disease. Here we tested the interactions between clocks in two critical components of organismal metabolism, liver and skeletal muscle, by rescuing clock function either in each organ separately or in both organs simultaneously in otherwise clock-less mice. Experiments showed that individual clocks are partially sufficient for tissue glucose metabolism, yet the connections between both tissue clocks coupled to daily feeding rhythms support systemic glucose tolerance. This synergy relies in part on local transcriptional control of the glucose machinery, feeding-responsive signals such as insulin, and metabolic cycles that connect the muscle and liver. We posit that spatiotemporal mechanisms of muscle and liver play an essential role in the maintenance of systemic glucose homeostasis and that disrupting this diurnal coordination can contribute to metabolic disease.

## Graphical abstract

![Graphical abstract undfig1](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/a7239750-3c0c-4e0a-a740-f4f690b2194d/main.assets/fx1_lrg.jpg)

Graphical abstract undfig1

## Research topic(s)

1. [CP: Metabolism](https://www.cell.com/action/doSearch?AllField=%22CP%3A+Metabolism%22&ISSN=2211-1247)

## Introduction

Circadian clocks are defining features of bacteria, plants, and animals.

1.

Rosbash, M.

**The implications of multiple circadian clock origins**

*PLoS Biol.* 2009; **7**:e62

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

2.

Bass, J. ∙ Lazar, M.A.

**Circadian time signatures of fitness and disease**

*Science.* 2016; **354**:994-999

In mammals, the clock broadly impacts physiology and drives rhythms of behavior, organ system function, and cellular pathways, thereby aligning them with the solar cycle. This process is linked with overall health when robust but with disease when disrupted.

3.

Guan, D. ∙ Lazar, M.A.

**Interconnections between circadian clocks and metabolism**

*J. Clin. Invest.* 2021; **131**:e148278

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

4.

Cederroth, C.R. ∙ Albrecht, U. ∙ Bass, J....

**Medicine in the fourth dimension**

*Cell Metabol.* 2019; **30**:238-250

Studies in humans and model organisms have made it clear that metabolism is a central aspect of circadian control. In mice, genetic and environmental circadian disruption impairs lipid and carbohydrate metabolism, induces hallmarks of metabolic syndrome, and leads to diabetes-like phenotypes.

5.

Fishbein, A.B. ∙ Knutson, K.L. ∙ Zee, P.C.

**Circadian disruption and human health**

*J. Clin. Invest.* 2021; **131**:e148286

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

6.

Shi, S.Q. ∙ Ansari, T.S. ∙ McGuinness, O.P....

**Circadian disruption leads to insulin resistance and obesity**

*Curr. Biol.* 2013; **23**:372-381

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

7.

Kettner, N.M. ∙ Voicu, H. ∙ Finegold, M.J....

**Circadian homeostasis of liver metabolism suppresses hepatocarcinogenesis**

*Cancer Cell.* 2016; **30**:909-924

Circadian control originates in virtually all cells of the body, propagates anatomically, and incorporates differential layers of regulation. Pioneering studies using tissue explants and cell type-specific clock mutant mice have established the existence of peripheral clocks that interact with circadian cues produced by central clocks in the brain.

8.

Schibler, U. ∙ Sassone-Corsi, P.

**A web of circadian pacemakers**

*Cell.* 2002; **111**:919-922

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

9.

Kornmann, B. ∙ Schaad, O. ∙ Bujard, H....

**System-driven and oscillator-dependent circadian transcription in mice with a conditionally active liver clock**

*PLoS Biol.* 2007; **5**:e34

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

10.

Nagoshi, E. ∙ Saini, C. ∙ Bauer, C....

**Circadian gene expression in individual fibroblasts: cell-autonomous and self-sustained oscillators pass time to daughter cells**

*Cell.* 2004; **119**:693-705

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

11.

Yoo, S.H. ∙ Yamazaki, S. ∙ Lowrey, P.L....

**PERIOD2::LUCIFERASE real-time reporting of circadian dynamics reveals persistent circadian oscillations in mouse peripheral tissues**

*Proc. Natl. Acad. Sci. USA.* 2004; **101**:5339-5346

The liver, a highly rhythmic and metabolic tissue, has autonomous clock functionality but requires inputs from central clocks, for example, daily feeding and fasting behavior, to express more of its rhythmic potential and maintain local homeostasis.

12.

Guan, D. ∙ Xiong, Y. ∙ Trinh, T.M....

**The hepatocyte clock and feeding control chronophysiology of multiple liver cell types**

*Science.* 2020; **369**:1388-1394

[Crossref](https://doi.org/10.1126/science.aba8984)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/32732282/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba8984&pmid=32732282)

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

14.

Greenwell, B.J. ∙ Trott, A.J. ∙ Beytebiere, J.R....

**Rhythmic food intake drives rhythmic gene expression more potently than the hepatic circadian clock in mice**

*Cell Rep.* 2019; **27**:649-657.e5

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

Considering that the liver interacts with many partner tissues (e.g., muscle), each of which contains its own local clock, we sought to test whether peripheral-to-peripheral clock interactions are critical, much like brain-to-peripheral connections are. Determining such mechanisms will provide a better understanding of how metabolism is regulated in time and space.

Here, we investigated the interaction between clocks in liver and skeletal muscle, two organs that are connected metabolically through the exchange of metabolites and regulatory signals, but whose coordinated daily actions are poorly understood. By manipulating endogenous *Bmal1*, the main transcriptional activator of the molecular clock, we rescued clock function in liver, in muscle, or in both tissues, in otherwise clock-less mice. Through analyses of transcriptomics and metabolism, we defined muscle clock autonomy and teased apart key spatiotemporal aspects of glucose homeostasis. Our major finding is that systemic glucose tolerance can be achieved from the concerted actions of three circadian system components—the liver clock, the muscle clock, and a feeding-fasting rhythm—yet no component alone is sufficient. We propose that these spatiotemporal aspects of glucose homeostasis contribute to the relationship between circadian disruption and metabolic disease.

## Results

### Characterization of muscle and liver+muscle clock reconstituted mice

To interrogate the functions of liver and muscle clocks, we used *Bmal1-stopFL* mice, in which endogenous expression of the critical clock component *Bmal1*

16.

Bunger, M.K. ∙ Wilsbacher, L.D. ∙ Moran, S.M....

**Mop3 is an essential component of the master circadian pacemaker in mammals**

*Cell.* 2000; **103**:1009-1017

can be reconstituted exclusively in Cre recombinase-expressing tissues in otherwise *Bmal1* -deficient mice.

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

17.

Welz, P.S. ∙ Zinna, V.M. ∙ Symeonidi, A....

**BMAL1-Driven tissue clocks Respond independently to light to maintain homeostasis**

*Cell.* 2019; **177**:1436-1447.e12

*Bmal1-stopFL* mice were crossed with hepatocyte-specific *Alfp* -Cre and myofiber-specific *Hsa* -Cre lines to generate the following genotypes: wild-type (WT; all clocks), full *Bmal1* knockout (KO; no clocks), hepatocyte *Bmal1* -reconstituted (Liver-RE; only liver clock),

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

myofiber *Bmal1* -reconstituted (Muscle-RE; only skeletal muscle clock), and both hepatocyte and myofiber *Bmal1* -reconstituted (Liver+Muscle-RE; only liver and skeletal muscle clocks) ([Figure 1](https://www.cell.com/cell-reports/fulltext/#fig1) A). Western blotting and qPCR confirmed the expected expression of BMAL1 across genotypes ([Figures S1](https://www.cell.com/cell-reports/fulltext/#mmc1) A–S1E). Even though total food intake was similar across genotypes, KO mice exhibited a lower body weight and a higher fat-to-lean mass ratio than WT mice, and the presence of liver and/or muscle clocks did not alter these phenotypes ([Figures 1](https://www.cell.com/cell-reports/fulltext/#fig1) B–1D). As reported previously for Liver-RE mice,

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

18.

Sinturel, F. ∙ Gos, P. ∙ Petrenko, V....

**Circadian hepatocyte clocks keep synchrony in the absence of a master pacemaker in the suprachiasmatic nucleus or other extrahepatic clocks**

*Genes Dev.* 2021; **35**:329-334

[Crossref](https://doi.org/10.1101/gad.346460.120)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/33602874/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1101%2Fgad.346460.120&pmid=33602874)

KO and all RE mice lacked robust 24 h rhythms of food intake and displayed severely blunted dark phase locomotor activity under 12 h light/12 h dark conditions with *ad libitum* (AL) feeding ([Figure 1](https://www.cell.com/cell-reports/fulltext/#fig1) E). Only WT mice had consistent and robust diurnal metabolic cycles, as exemplified by respiratory exchange ratio (RER) and energy expenditure ([Figures 1](https://www.cell.com/cell-reports/fulltext/#fig1) F and [S1](https://www.cell.com/cell-reports/fulltext/#mmc1) F). Only six of the 23 KO and RE mice had significant rhythms of both parameters ([Figure S1](https://www.cell.com/cell-reports/fulltext/#mmc1) F). Compared with KO, mice with liver and/or muscle clocks did not show significant differences in RER ([Figure 1](https://www.cell.com/cell-reports/fulltext/#fig1) F).

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/4fd255e3-95db-4bab-8ce4-45a0855a24f0/main.assets/gr1_lrg.jpg)

Figure 1 Characterization of muscle or liver+muscle clock reconstituted mice

### Liver and muscle core clocks oscillate in an independent manner

To assess muscle clock autonomy, and probe potential muscle-liver connections, we performed RNA sequencing (RNA-seq) on livers and gastrocnemius muscles (predominantly fast-twitch fibers

19.

Sher, J. ∙ Cardasis, C.

**Skeletal muscle fiber types in the adult mouse**

*Acta Neurol. Scand.* 1976; **54**:45-56

) harvested every 4 h over the diurnal cycle. For comparisons with Liver-RE, we used data from our previous study featuring mice of identical age, sex, diet, and conditions.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

We used the differential rhythmicity algorithm *dryR*

20.

Weger, B.D. ∙ Gobet, C. ∙ David, F.P.A....

**Systematic analysis of differential rhythmic liver gene expression mediated by the circadian clock and feeding rhythms**

*Proc. Natl. Acad. Sci. USA.* 2021; **118**

e2015803118

[Crossref](https://doi.org/10.1073/pnas.2015803118)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/33452134/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2015803118&pmid=33452134)

to compare genotypes ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) A–2D, [Tables S1](https://www.cell.com/cell-reports/fulltext/#mmc2) and [S2](https://www.cell.com/cell-reports/fulltext/#mmc3)) alongside JTK\_CYCLE

21.

Hughes, M.E. ∙ Hogenesch, J.B. ∙ Kornacker, K.

**JTK\_CYCLE: an efficient nonparametric algorithm for detecting rhythmic components in genome-scale data sets**

*J. Biol. Rhythm.* 2010; **25**:372-380

for analysis of peak phases and amplitudes for core clock genes ([Figures S2](https://www.cell.com/cell-reports/fulltext/#mmc1) A and S2B). As both single and double Cre-positive WT mice (i.e., Hsa-Cre, Alfp-Cre, Hsa+Alfp-Cre) were similar according to dryR models, double Cre-positive WTs are shown in the figures.

In muscle, loss of *Bmal1* disrupted clock gene oscillations as expected ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) A, 2B, [S2](https://www.cell.com/cell-reports/fulltext/#mmc1) A, and S2C). Even with highly efficient Cre-mediated recombination ([Figure S1](https://www.cell.com/cell-reports/fulltext/#mmc1) C), reconstitution of muscle *Bmal1* led to only a partial rescue, with clear differences between Muscle-RE and WT. On average, amplitudes of clock genes in Muscle-RE were 56.4% lower ([Figure S2](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Reflecting a lack of robustness, *dryR* assigned most clock genes to a model of genes oscillating with advanced phase and reduced amplitude in Muscle-RE or Liver+Muscle-RE mice and indicated that *Cry1* did not oscillate ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) A and [S2](https://www.cell.com/cell-reports/fulltext/#mmc1) C). In Liver-RE mice, nearly all clock genes were phase advanced with reduced amplitude ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) C, 2D, [S2](https://www.cell.com/cell-reports/fulltext/#mmc1) B, and S2D), as we have shown previously

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

; exceptions to this were *Per1* and *Bhlhe41*.

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/904858ed-4465-48c5-a1e7-a031783c2ef1/main.assets/gr2_lrg.jpg)

Figure 2 Liver and muscle core clocks oscillate in an independent manner

Next, we considered that liver and muscle molecular clocks may influence each other. The additional rescue of liver *Bmal1* had little effect on muscle clock genes ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) A and 2B). Like Muscle-RE mice, clock gene amplitudes remained lower in Liver+Muscle-RE mice compared with WT mice ([Figure S2](https://www.cell.com/cell-reports/fulltext/#mmc1) A). In Liver+Muscle-RE mice, *dryR* models indicated that one clock gene, *Bhlhe40*, gained oscillation in muscle in the presence of the liver clock, whereas another gene, *Tef*, lost oscillation ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) A and 2B). Inversely, dual clock rescue had little impact on liver clock genes. Nonetheless, *dryR* models indicated that the oscillation of *Rev-Erbα* and *Bhlhe40* in liver was fully rescued in double but not in either single RE mouse ([Figures 2](https://www.cell.com/cell-reports/fulltext/#fig2) C and 2D). Together, these data show that liver and muscle clocks are largely independent of one another.

### Muscle Bmal1 is partially sufficient to drive the local diurnal transcriptome

Previous rescue experiments suggested that the muscle clock can have effects on behavior.

22.

McDearmon, E.L. ∙ Patel, K.N. ∙ Ko, C.H....

**Dissecting the functions of the mammalian clock protein BMAL1 by tissue-specific rescue in mice**

*Science.* 2006; **314**:1304-1308

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

23.

Ehlen, J.C. ∙ Brager, A.J. ∙ Baggs, J....

**Bmal1 function in skeletal muscle regulates sleep**

*Elife.* 2017; **6**:e26557

However, whether endogenous muscle *Bmal1* is sufficient to drive rhythmic functions within the muscle has not been defined. To explore this question, we assessed the state of clock-controlled output genes, focusing on genes that oscillated in both single and double Cre-positive WT mice. Considering the top 10 *dryR* models (ranked according to the number of genes captured in each) we found sizable differences in the oscillatory nature of liver and muscle. About three times more genes oscillated in WT liver (3,750) as in WT muscle (1,150) ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A and 3B). The ability of each local clock to drive diurnal transcription was also different; reconstitution of *Bmal1* induced oscillation of ∼1,000 genes in liver but only ∼250 genes in muscle ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A and 3B).

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/de5fb0bb-2709-45f3-b94a-103c2571f03b/main.assets/gr3_lrg.jpg)

Figure 3 Local clocks in muscle and liver drive distinct autonomous outputs

Next, we identified two *dryR* models representing autonomous genes. We defined autonomous as a loss of oscillation in KO mice with the reappearance of oscillation in single and double RE mice. One model contained 81 genes that oscillated identically in WT, Muscle-RE, and Liver+Muscle-RE mice, peaking mainly around zeitgeber time (ZT) 1–ZT5 and ZT10–ZT12 ([Figure 3](https://www.cell.com/cell-reports/fulltext/#fig3) A and 3C). These genes were involved in a range of biological processes, including development, protein catabolism, and intracellular signaling. Notably, the presence of muscle *Bmal1* was sufficient to drive oscillation of *Myod1*, a principal regulator of the skeletal muscle lineage

24.

Andrews, J.L. ∙ Zhang, X. ∙ McCarthy, J.J....

**CLOCK and BMAL1 regulate MyoD and are necessary for maintenance of skeletal muscle phenotype and function**

*Proc. Natl. Acad. Sci. USA.* 2010; **107**:19090-19095

([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Another model, which included most core clock genes, comprised 45 genes that oscillated in the three genotypes but with advanced phase and reduced amplitude in Muscle-RE and Liver+Muscle-RE mice ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A, 3C, and [S3](https://www.cell.com/cell-reports/fulltext/#mmc1) G). In addition to circadian-rhythm-related terms, these genes were enriched for pathways involving transcription, glycogen metabolic process, phosphorylation, and Wnt signaling, among others.

Regulation of glucose metabolism is a central function of the muscle clock.

25.

Harfmann, B.D. ∙ Schroder, E.A. ∙ Kachman, M.T....

**Muscle-specific loss of Bmal1 leads to disrupted tissue glucose metabolism and systemic glucose homeostasis**

*Skelet. Muscle.* 2016; **6**:12

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

26.

Rudic, R.D. ∙ McNamara, P. ∙ Curtis, A.M....

**BMAL1 and CLOCK, two essential components of the circadian clock, are involved in glucose homeostasis**

*PLoS Biol.* 2004; **2**:e377

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

27.

Dyar, K.A. ∙ Ciciliot, S. ∙ Wright, L.E....

**Muscle insulin sensitivity and glucose metabolism are controlled by the intrinsic muscle clock**

*Mol. Metabol.* 2014; **3**:29-41

Here, we observed differential expression of several glucose-related genes ([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Although the oscillation of *Slc2a4* (Glut4), the primary insulin-dependent glucose transporter, was rescued in muscle from Muscle-RE mice, average expression remained considerably lower than WT ([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Other examples are *Pfkfb3*, encoding a glycolysis-promoting enzyme, and *Pdp1*, encoding a phosphatase that activates pyruvate dehydrogenase (PDH) phosphatase ([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) A).

In liver, genes that oscillated identically in WT, Liver-RE, and Liver+Muscle-RE mice mostly peaked between ZT6 and ZT10, with an additional peak around ZT20, and were enriched for metabolic pathways including mitochondrial function, lipid metabolism, amino acid metabolism, and transcription ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) B and 3D). Like muscle, liver contained an additional set of genes that oscillated in all three genotypes but had advanced phase and reduced amplitude in RE mice ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) B and 3D). Consistent with our previous reports,

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

*dryR* analysis showed that these liver genes were functionally enriched for circadian rhythm and terms including NAD <sup xmlns="http://www.w3.org/1999/xhtml">+</sup> biosynthesis, and carbohydrate metabolism ([Figure 3](https://www.cell.com/cell-reports/fulltext/#fig3) D).

Manipulation of *Bmal1* may also shape tissue phenotypes by altering the expression of non-rhythmic genes. We used *dryR* mean models, which test for differences in average daily expression, to assess the expression of non-rhythmic WT genes. In liver and muscle, the largest model contained genes with identical expression between WT, KO, and RE mice ([Tables S3](https://www.cell.com/cell-reports/fulltext/#mmc4) and [S4](https://www.cell.com/cell-reports/fulltext/#mmc5)). The next largest model contained genes that were deregulated in KO and restored to WT levels in RE mice, representing 24.65% and 30.54% of non-rhythmic genes in muscle and liver, respectively ([Figures S3](https://www.cell.com/cell-reports/fulltext/#mmc1) B and S3D). The top enrichments of these genes were pathways involving immune function in liver and mitochondrial function in muscle ([Figures S3](https://www.cell.com/cell-reports/fulltext/#mmc1) C and S3E). [Figures S3](https://www.cell.com/cell-reports/fulltext/#mmc1) These analyses suggests that changes in the expression of non-rhythmic genes may also contribute to the state of metabolism in KO and RE mice.

### Liver and muscle diurnal transcriptomes are influenced by Bmal1 function in distal tissues

Even though liver and muscle core clocks are largely independent, we considered that they may still influence clock-controlled output genes across the two tissues. Generally, heightened rhythmicity was not observed with dual versus single reconstitution, as evidenced by a similar number of oscillating genes in single and double RE mice ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A and 3B). Fewer genes oscillated in muscles of Liver+Muscle-RE compared with Muscle-RE mice, showing a minor inhibitory effect of liver *Bmal1* on oscillation of the muscle. However, sets of genes that oscillated exclusively in WT and Liver+Muscle-RE mice were among the top *dryR* models in liver (484 genes) and muscle (112 genes) ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A and 3B), indicating a degree of cross-tissue regulation. In muscle, genes rescued by dual reconstitution were confined to the dark phase and were related to intracellular signal transduction, phosphorylation, proliferation, and development ([Figure 4](https://www.cell.com/cell-reports/fulltext/#fig4) A). In liver, the genes had more widely distributed phases and pertained to pathways involving ribosomes, cytoskeletal regulation, and NF-κB activity ([Figure 4](https://www.cell.com/cell-reports/fulltext/#fig4) B).

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/75e57693-9dbe-4d2e-b5a2-f906e3fadd8f/main.assets/gr4_lrg.jpg)

Figure 4 Liver and muscle diurnal transcriptomes are influenced by Bmal1 function in distal tissues

To complement *dryR* analysis, we performed two-way ANOVA genome-wide using Nitecap.

28.

Brooks, T.G. ∙ Mrčela, A. ∙ Lahens, N.F....

**Nitecap: an exploratory circadian analysis web application**

*J. Biol. Rhythm.* 2022; **37**:43-52

Mirroring *dryR* analysis, we observed a similar number of significant changes in single and double RE tissues compared with WT ([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) F). This analysis also indicated a greater similarity between Muscle-RE and WT mice than Liver+Muscle-RE and WT mice.

We next interrogated the *dryR* model containing genes that oscillated exclusively in WT mice, which we referred to as genes that oscillate non-autonomously. These genes require a function of *Bmal1* in additional tissue(s), for example the timed release of a soluble signaling molecule. Interestingly, in muscle, response to hypoxia, response to hormone, and cellular glucose homeostasis were enriched pathways of genes oscillating non-autonomously ([Figure 4](https://www.cell.com/cell-reports/fulltext/#fig4) A). Notable genes included *Tbc1d1*, which aids in GLUT4 trafficking to the plasma membrane,

29.

An, D. ∙ Toyoda, T. ∙ Taylor, E.B....

**TBC1D1 regulates insulin- and contraction-induced glucose transport in mouse skeletal muscle**

*Diabetes.* 2010; **59**:1358-1365

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

30.

Szekeres, F. ∙ Chadt, A. ∙ Tom, R.Z....

**The Rab-GTPase-activating protein TBC1D1 regulates skeletal muscle glucose metabolism**

*Am. J. Physiol. Endocrinol. Metab.* 2012; **303**:E524-E533

and *Hk2*, which phosphorylates glucose in the first step of glycolysis ([Figure S3](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Genes oscillating non-autonomously in liver were also enriched in response to hypoxia and pathways related to protein phosphorylation ([Figure 4](https://www.cell.com/cell-reports/fulltext/#fig4) B), which is linked with extrinsic signaling events.

Additional *dryR* models contained a subset of genes that were rescued in Liver-RE or Muscle-RE mice but then lost in double RE mice ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A, 3B, [S4](https://www.cell.com/cell-reports/fulltext/#mmc1) A, and S4B). Although the biological significance of this subset of genes is unclear, many functional enrichments in liver and muscle were distinct from other models. *DryR* also constructed models with genes that oscillated irrespective of the presence of *Bmal1* ([Figures 3](https://www.cell.com/cell-reports/fulltext/#fig3) A, 3B, [S4](https://www.cell.com/cell-reports/fulltext/#mmc1) C, and S4D). Together, these data show that liver and muscle clocks influence clock-controlled output genes across the two tissues in specific ways.

### Liver and muscle clocks are partially sufficient for control of glucose metabolism at the tissue but not systemic level

The metabolism of glucose within muscle and liver helps achieve whole-body glucose homeostasis.

31.

Kalsbeek, A. ∙ la Fleur, S. ∙ Fliers, E.

**Circadian control of glucose metabolism**

*Mol. Metabol.* 2014; **3**:372-383

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

32.

Lamia, K.A. ∙ Storch, K.F. ∙ Weitz, C.J.

**Physiological significance of a peripheral tissue circadian clock**

*Proc. Natl. Acad. Sci. USA.* 2008; **105**:15172-15177

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

33.

Gachon, F. ∙ Loizides-Mangold, U. ∙ Petrenko, V....

**Glucose homeostasis: regulation by peripheral circadian clocks in rodents and humans**

*Endocrinology.* 2017; **158**:1074-1084

Muscle is the primary site of post-prandial glucose disposal, and the liver is the main producer of glucose.

34.

Hui, S. ∙ Cowan, A.J. ∙ Zeng, X....

**Quantitative fluxomics of circulating metabolites**

*Cell Metabol.* 2020; **32**:676-688.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

36.

DeFronzo, R.A. ∙ Jacot, E. ∙ Jequier, E....

**The effect of insulin on the disposal of intravenous glucose. Results from indirect calorimetry and hepatic and femoral venous catheterization**

*Diabetes.* 1981; **30**:1000-1007

As previous studies have shown that BMAL1 is necessary for maximal muscle glucose uptake,

25.

Harfmann, B.D. ∙ Schroder, E.A. ∙ Kachman, M.T....

**Muscle-specific loss of Bmal1 leads to disrupted tissue glucose metabolism and systemic glucose homeostasis**

*Skelet. Muscle.* 2016; **6**:12

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

27.

Dyar, K.A. ∙ Ciciliot, S. ∙ Wright, L.E....

**Muscle insulin sensitivity and glucose metabolism are controlled by the intrinsic muscle clock**

*Mol. Metabol.* 2014; **3**:29-41

we asked whether it is sufficient. We performed isotope tracing with U- <sup xmlns="http://www.w3.org/1999/xhtml">13</sup> C-glucose to follow the metabolism of glucose into tissues ([Figure 5](https://www.cell.com/cell-reports/fulltext/#fig5) A).

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/07fc2dd3-5f71-4c9c-bc08-e0d94a624e05/main.assets/gr5_lrg.jpg)

Figure 5 Liver and muscle clocks are partially sufficient for control of glucose metabolism at the tissue but not systemic level

Gavage of 1.5 g/kg, a typical dose of a glucose tolerance test, sufficiently labeled serum glucose ([Figure 5](https://www.cell.com/cell-reports/fulltext/#fig5) B). Compared with WT muscles, KO muscles showed substantially lower labeling of glycolytic and tricarboxylic acid (TCA) cycle intermediates, indicating impaired glucose uptake and oxidation ([Figures 5](https://www.cell.com/cell-reports/fulltext/#fig5) C and 5D). The presence of muscle BMAL1 in Muscle-RE mice restored labeling of these metabolites, but labeling was still considerably lower than in WT, highlighting the requirement of other tissue clocks for muscle glucose metabolism. Liver clock reconstitution did not alter muscle glucose metabolism, as Muscle-RE and Liver+Muscle-RE labeling patterns were similar ([Figures 5](https://www.cell.com/cell-reports/fulltext/#fig5) C and 5D). Although the liver does take up glucose during feeding to support anabolic pathways (e.g., glycogen and lipid synthesis), it also takes up muscle-derived metabolites of glucose (e.g., lactate and alanine) for oxidation or recycling.

37.

Cori, C.F.

**The glucose-lactic acid cycle and gluconeogenesis**

*Curr. Top. Cell. Regul.* 1981; **18**:377-387

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

38.

Felig, P. ∙ Pozefsky, T. ∙ Marliss, E....

**Alanine: key role in gluconeogenesis**

*Science.* 1970; **167**:1003-1004

In contrast to muscle, liver from KO mice displayed increased labeling of glycolytic intermediates and a modestly reduced labeling of TCA cycle intermediates ([Figures 5](https://www.cell.com/cell-reports/fulltext/#fig5) C and 5D). In Liver-RE mice, we observed normalized labeling of glycolytic but not TCA cycle intermediates, suggesting disparate regulatory mechanisms ([Figure 5](https://www.cell.com/cell-reports/fulltext/#fig5) D). Taken together, these data show partial rescue of tissue glucose metabolism by autonomous clocks in muscle and liver, with no evidence of improved metabolism with both clocks in tandem.

By balancing the consumption and production of glucose during periods of feeding and fasting, liver and muscle help maintain blood glucose within a physiological range.

32.

Lamia, K.A. ∙ Storch, K.F. ∙ Weitz, C.J.

**Physiological significance of a peripheral tissue circadian clock**

*Proc. Natl. Acad. Sci. USA.* 2008; **105**:15172-15177

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

34.

Hui, S. ∙ Cowan, A.J. ∙ Zeng, X....

**Quantitative fluxomics of circulating metabolites**

*Cell Metabol.* 2020; **32**:676-688.e4

To determine whether any RE mice had better glycemic control than KO mice, we performed oral glucose tolerance tests. Remarkably, glucose tolerance was similarly impaired in all genotypes compared with WT, in both male and female mice ([Figures 5](https://www.cell.com/cell-reports/fulltext/#fig5) E, [S5](https://www.cell.com/cell-reports/fulltext/#mmc1) A, and S5B). Our results demonstrate that despite partially rescuing tissue glucose metabolism, BMAL1 function in liver and muscle is insufficient to improve the glucose intolerance of KO mice.

### Feeding rhythms and autonomous clocks bolster time-dependent gene expression in liver and muscle

Next, we considered which additional diurnal signals might invigorate liver and muscle clocks and enable improvement of glucose homeostasis at the systemic level. WT mice, but not KO and RE mice, exhibit rhythmic feeding-fasting behavior ([Figure 1](https://www.cell.com/cell-reports/fulltext/#fig1) E), which affects local clocks and transcript oscillations in peripheral organs.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

14.

Greenwell, B.J. ∙ Trott, A.J. ∙ Beytebiere, J.R....

**Rhythmic food intake drives rhythmic gene expression more potently than the hepatic circadian clock in mice**

*Cell Rep.* 2019; **27**:649-657.e5

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

39.

Petrus, P. ∙ Smith, J.G. ∙ Koronowski, K.B....

**The central clock suffices to drive the majority of circulatory metabolic rhythms**

*Sci. Adv.* 2022; **8**:eabo2896

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

40.

Damiola, F. ∙ Le Minh, N. ∙ Preitner, N....

**Restricted feeding uncouples circadian oscillators in peripheral tissues from the central pacemaker in the suprachiasmatic nucleus**

*Genes Dev.* 2000; **14**:2950-2961

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

41.

Vollmers, C. ∙ Gill, S. ∙ DiTacchio, L....

**Time of feeding and the intrinsic circadian clock drive rhythms in hepatic gene expression**

*Proc. Natl. Acad. Sci. USA.* 2009; **106**:21453-21458

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

42.

de Goede, P. ∙ Sen, S. ∙ Oosterman, J.E....

**Differential effects of diet composition and timing of feeding behavior on rat brown adipose tissue and skeletal muscle peripheral clocks**

*Neurobiol. Sleep Circadian Rhythms.* 2018; **4**:24-33

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

43.

Manella, G. ∙ Sabath, E. ∙ Aviram, R....

**The liver-clock coordinates rhythmicity of peripheral tissues in response to feeding**

*Nat. Metab.* 2021; **3**:829-842

We have demonstrated that feeding-fasting behavior bolsters the phase and output of the liver clock including glucose pathways.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

To determine the effect of feeding rhythms, we performed experiments under night feeding (NF) conditions, giving mice access to food only during their active phase (the 12 h dark period; ZT12–ZT24/0) for 2 weeks. As expected, NF induced rhythms of feeding and fasting, as well as RER, in KO and RE mice ([Figures 6](https://www.cell.com/cell-reports/fulltext/#fig6) A, [S6](https://www.cell.com/cell-reports/fulltext/#mmc1) B, and S6C). Only small changes in body weight were observed under NF (WT, +2.14%; KO and RE, −3.48% to −6.81%) ([Figure S6](https://www.cell.com/cell-reports/fulltext/#mmc1) A).

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/58fe6476-a4e3-4419-b3e3-e766ca7677e7/main.assets/gr6_lrg.jpg)

Figure 6 Feeding rhythms and autonomous clocks bolster time-dependent gene expression in liver and muscle

We performed bulk RNA-seq on each tissue at ZT4 and ZT16 under NF. Using Liver-RE mice, we previously showed that NF corrects the phase and amplitude of core clock oscillations

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

; thus, we now focused on muscle. Assessing day-night differences, we found that NF made the expression profile of *Rev-Erbɑ* more similar to WT in both Muscle-RE and Liver+Muscle-RE mice ([Figures 6](https://www.cell.com/cell-reports/fulltext/#fig6) B and S6D). Although we cannot infer rhythmicity from two time point data, several other clock genes showed significant day-night changes under NF ([Figure S6](https://www.cell.com/cell-reports/fulltext/#mmc1) D) and these changes likely reflect enhancement of oscillatory phase and/or amplitude. Notably, for liver and muscle, we observed no additional benefit of dual reconstitution on clock gene expression ([Figure S6](https://www.cell.com/cell-reports/fulltext/#mmc1) D). These data indicate that as in liver, imposed feeding-fasting rhythms bolster clock gene expression in muscle.

Next, we assessed the genome-wide effect of NF. In liver, Liver-RE and Liver+Muscle-RE mice were most similar to WT mice ([Figures S7](https://www.cell.com/cell-reports/fulltext/#mmc1) A and S7B). In muscle, Liver+Muscle-RE showed the least number of changes versus WT at ZT4 and exhibited significant changes versus Muscle-RE mice (Figures  [S7](https://www.cell.com/cell-reports/fulltext/#mmc1) A and [S8](https://www.cell.com/cell-reports/fulltext/#mmc1) A). Changes included genes involved in lipid and glucose metabolism and are supportive of a limited yet significant impact of the liver clock on the muscle at this time point ([Figure S8](https://www.cell.com/cell-reports/fulltext/#mmc1) B). We also considered that reconstitution of *Bmal1* in liver could alter gene expression in muscles (e.g., livers of Muscle-RE mice), and vice versa. However, we observed that without local *Bmal1* also present, the cross-tissue impact was negligible ([Figures S7](https://www.cell.com/cell-reports/fulltext/#mmc1) A and S7B).

We then asked whether feeding-fasting rhythms rescued the temporal expression of genes that oscillated non-autonomously in WT mice under AL conditions. From 57 genes that exhibited higher expression at ZT4 vs. ZT16 in WT NF mice, only 3 were significantly different in KO, and there was no further benefit of muscle clock reconstitution ([Figures 6](https://www.cell.com/cell-reports/fulltext/#fig6) C). One hundred eighteen genes exhibited lower expression at ZT4 vs. ZT16 in WT NF mice. Of these, only 18 were significantly different in KO. Again, we observed no further benefit of muscle clock reconstitution on this set of genes ([Figures 6](https://www.cell.com/cell-reports/fulltext/#fig6) D). Next, we functionally annotated genes that showed no significant difference vs. WT (i.e., rescued genes) at ZT4 and ZT16. Although we observed relatively weak enrichments, many functions were identified, including those relevant to the metabolic phenotypes at hand. Notably, at ZT16, glucose metabolic process and two terms related to insulin signaling were identified as rescued specifically in Liver+Muscle-RE muscle ([Figures 6](https://www.cell.com/cell-reports/fulltext/#fig6) C and 6D), suggesting an interaction of the two clocks on this process. Furthermore, identification of these pathways did not depend on the use of a specific algorithm; calling AL non-autonomously oscillating genes with JTK\_CYCLE also led to identification of similar gene classes as rescued in Liver+Muscle-RE muscles ([Figure S9](https://www.cell.com/cell-reports/fulltext/#mmc1)). Further inspection of glucose genes revealed that NF also led to small improvements in the expression of *Glut4* and *Pfkm* when the muscle clock was intact ([Figure S10](https://www.cell.com/cell-reports/fulltext/#mmc1)). These data show that NF improves the temporal profiles of gene expression in muscle, an effect that depends on *Bmal1* in the muscle, and involves certain genes involved in glucose metabolism.

Analyses of liver agreed with our previous report.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

Considering genome-wide day-night differences within genotypes, we observed 2 to 3 times the number of changes in Liver-RE and Liver+Muscle-RE, respectively, compared with KO liver ([Figure S11](https://www.cell.com/cell-reports/fulltext/#mmc1)). Like in muscle, there was still a considerably sized subset of genes whose expression was significantly different than in WT liver, indicating that there are other inputs besides feeding that maximize temporal gene expression. This was particularly evident at ZT4. Genes rescued by NF in Liver-RE and Liver+Muscle-RE at ZT4 were modestly enriched for actin cytoskeleton, immune function, and MAPK signaling, among others ([Figure S11](https://www.cell.com/cell-reports/fulltext/#mmc1)). Of note, some genes involved in mitochondrial translation were rescued in KO and Liver-RE mice at ZT16, showing that feeding-fasting cycles are sufficient to modulate this pathway. In addition, genes involved in autophagy were uniquely rescued in Liver+Muscle-RE mice, suggesting that an interaction of the two clocks is important for this function. Finally, using JTK\_CYCLE to define AL non-autonomously oscillating genes, we found that genes involved in glucose metabolism were rescued by NF at ZT16 in Liver-RE and Liver+Muscle-RE mice ([Figure S12](https://www.cell.com/cell-reports/fulltext/#mmc1)).

Overall, these data show that feeding-fasting rhythms enhance the temporal expression of clock genes and sets of clock-controlled output genes in liver and muscle. Similar to AL conditions, there was no major benefit of dual clock reconstitution under NF. However, rescue of certain genes involved in glucose metabolism was observed in Liver+Muscle-RE mice, indicating a potential role for the interaction between feeding rhythms and liver and muscle clocks in driving this process from a transcriptional standpoint.

### Synergy between feeding rhythms and liver and muscle clocks enables systemic glucose homeostasis

Next, we asked whether tissue clock function bolstered by feeding rhythms would improve serum glucose responses. Indeed, different glucose tolerances were uncovered in KO and RE mice under NF. The initial peak of glucose was blunted in Muscle-RE compared with KO, suggesting that muscle *Bmal1* alone was able to improve glucose clearance ([Figures 7](https://www.cell.com/cell-reports/fulltext/#fig7) A and [S13](https://www.cell.com/cell-reports/fulltext/#mmc1) A). More strikingly, however, blood glucose curves of WT and Liver+Muscle-RE mice were indistinguishable; blood glucose peaked 30 min post-gavage in Liver-RE and Muscle-RE, yet it peaked just 15 min post-gavage in Liver+Muscle-RE mice ([Figure 7](https://www.cell.com/cell-reports/fulltext/#fig7) A), indicating a more rapid rate of glucose clearance in the presence of both liver and muscle clocks. Comparison of areas under the curve revealed improved glucose tolerance in Liver+Muscle-RE mice compared with KO mice ([Figure 7](https://www.cell.com/cell-reports/fulltext/#fig7) A). These results demonstrate a functional connection between liver and muscle clocks for systemic glucose metabolism in the presence of a feeding-fasting rhythm.

![](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/asset/713cd88d-98b9-4ca5-a87c-8164a97b689e/main.assets/gr7_lrg.jpg)

Figure 7 Synergy between feeding rhythms and liver and muscle clocks enables systemic glucose homeostasis

During feeding, most glucose is cleared via insulin-induced uptake into tissues.

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

44.

TeSlaa, T. ∙ Bartman, C.R. ∙ Jankowski, C.S.R....

**The source of glycolytic intermediates in mammalian tissues**

*Cell Metabol.* 2021; **33**:367-378.e5

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

45.

DeFronzo, R.A. ∙ Tripathy, D.

**Skeletal muscle insulin resistance is the primary defect in type 2 diabetes**

*Diabetes Care.* 2009; **32**:S157-S163

[Crossref](https://doi.org/10.2337/dc09-S302)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/19875544/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.2337%2Fdc09-S302&pmid=19875544)

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

46.

Merz, K.E. ∙ Thurmond, D.C.

**Role of skeletal muscle in insulin resistance and glucose uptake**

*Compr. Physiol.* 2020; **10**:785-809

NF induced a rhythm of serum insulin in Liver+Muscle-RE mice, evidenced by a peak during feeding at ZT16 and a trough during fasting at ZT4 ([Figure 7](https://www.cell.com/cell-reports/fulltext/#fig7) B). We have previously demonstrated serum insulin rhythms under NF in KO and Liver-RE mice.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

Serum insulin is noteworthy because it can act as a synchronizing cue for peripheral oscillators, inhibit glucose production in liver, and drive glucose uptake into muscle.

27.

Dyar, K.A. ∙ Ciciliot, S. ∙ Wright, L.E....

**Muscle insulin sensitivity and glucose metabolism are controlled by the intrinsic muscle clock**

*Mol. Metabol.* 2014; **3**:29-41

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

29.

An, D. ∙ Toyoda, T. ∙ Taylor, E.B....

**TBC1D1 regulates insulin- and contraction-induced glucose transport in mouse skeletal muscle**

*Diabetes.* 2010; **59**:1358-1365

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

47.

Fougeray, T. ∙ Polizzi, A. ∙ Régnier, M....

**The hepatocyte insulin receptor is required to program the liver clock and rhythmic gene expression**

*Cell Rep.* 2022; **39**:110674

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

48.

Puigserver, P. ∙ Rhee, J. ∙ Donovan, J....

**Insulin-regulated hepatic gluconeogenesis through FOXO1-PGC-1alpha interaction**

*Nature.* 2003; **423**:550-555

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

49.

Crosby, P. ∙ Hamnett, R. ∙ Putker, M....

**Insulin/IGF-1 drives PERIOD synthesis to entrain circadian rhythms with feeding time**

*Cell.* 2019; **177**:896-909.e20

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

50.

Yamajuku, D. ∙ Inagaki, T. ∙ Haruma, T....

**Real-time monitoring in three-dimensional hepatocytes reveals that insulin acts as a synchronizer for liver clock**

*Sci. Rep.* 2012; **2**:439

*Bmal1* -null mice have a defect in glucose-stimulated insulin secretion and are hypersensitive to insulin.

26.

Rudic, R.D. ∙ McNamara, P. ∙ Curtis, A.M....

**BMAL1 and CLOCK, two essential components of the circadian clock, are involved in glucose homeostasis**

*PLoS Biol.* 2004; **2**:e377

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

51.

Petrenko, V. ∙ Saini, C. ∙ Giovannoni, L....

**Pancreatic alpha- and beta-cellular clocks have distinct molecular properties and impact on islet hormone secretion and gene expression**

*Genes Dev.* 2017; **31**:383-398

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

52.

Jouffe, C. ∙ Weger, B.D. ∙ Martin, E....

**Disruption of the circadian clock component BMAL1 elicits an endocrine adaption impacting on insulin sensitivity and liver disease**

*Proc. Natl. Acad. Sci. USA.* 2022; **119**

e2200083119

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

53.

Marcheva, B. ∙ Ramsey, K.M. ∙ Buhr, E.D....

**Disruption of the clock components CLOCK and BMAL1 leads to hypoinsulinaemia and diabetes**

*Nature.* 2010; **466**:627-631

Here, from insulin tolerance tests, we observed that RE mice are similarly hypersensitive to insulin under AL, although not all genotypes were statistically significant ([Figures 7](https://www.cell.com/cell-reports/fulltext/#fig7) C and [S13](https://www.cell.com/cell-reports/fulltext/#mmc1) B). Interestingly, NF reduced insulin sensitivity similarly in KO and RE mice ([Figures 7](https://www.cell.com/cell-reports/fulltext/#fig7) D, [S13](https://www.cell.com/cell-reports/fulltext/#mmc1) C, and S13F). Thus, we concluded that while insulin levels and action contributed to glucose tolerances under NF, the two factors did not explain why glucose tolerance was rescued specifically in Liver+Muscle-RE mice.

Because muscle is the main site of post-prandial glucose clearance,

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

45.

DeFronzo, R.A. ∙ Tripathy, D.

**Skeletal muscle insulin resistance is the primary defect in type 2 diabetes**

*Diabetes Care.* 2009; **32**:S157-S163

[Crossref](https://doi.org/10.2337/dc09-S302)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/19875544/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.2337%2Fdc09-S302&pmid=19875544)

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

46.

Merz, K.E. ∙ Thurmond, D.C.

**Role of skeletal muscle in insulin resistance and glucose uptake**

*Compr. Physiol.* 2020; **10**:785-809

improved glucose tolerance in Liver+Muscle-RE mice likely reflects faster and/or more efficient muscle oxidation of glucose. Moreover, as improved glucose tolerance was maximal in the presence of both liver and muscle *Bmal1*, liver *Bmal1* must aid in glucose clearance. Glycolytic activity in muscle converts glucose to lactate and alanine, which are subsequently released, taken up by the liver, and recycled to glucose (i.e., the Cori and Cahil cycles).

37.

Cori, C.F.

**The glucose-lactic acid cycle and gluconeogenesis**

*Curr. Top. Cell. Regul.* 1981; **18**:377-387

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

38.

Felig, P. ∙ Pozefsky, T. ∙ Marliss, E....

**Alanine: key role in gluconeogenesis**

*Science.* 1970; **167**:1003-1004

Recent studies show that these cycles continue during feeding and that there are additional fates of lactate in the liver. As a preferred TCA cycle substrate, lactate can be fully oxidized or generate TCA cycle-derived amino acids. Lactate can also contribute to glycogen storage.

34.

Hui, S. ∙ Cowan, A.J. ∙ Zeng, X....

**Quantitative fluxomics of circulating metabolites**

*Cell Metabol.* 2020; **32**:676-688.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

44.

TeSlaa, T. ∙ Bartman, C.R. ∙ Jankowski, C.S.R....

**The source of glycolytic intermediates in mammalian tissues**

*Cell Metabol.* 2021; **33**:367-378.e5

Consistent with reports that muscle-specific *Bmal1* KO mice have normal glycogen content over the diurnal cycle,

54.

Dyar, K.A. ∙ Hubert, M.J. ∙ Mir, A.A....

**Transcriptional programming of lipid and amino acid metabolism by the skeletal muscle circadian clock**

*PLoS Biol.* 2018; **16**:e2005886

we observed relatively small changes in the expression of glycogen synthase 1 (*Gys1*) and glycogen phosphorylase (*Pygm*) in KO or Muscle-RE mice vs. WT ([Figure S13](https://www.cell.com/cell-reports/fulltext/#mmc1) E). We have shown previously that rhythms of glycogen synthesis and breakdown are restored to a significant degree in Liver-RE mice

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

([Figure S13](https://www.cell.com/cell-reports/fulltext/#mmc1) E). However, the contribution of glycogen synthesis to glucose clearance is estimated at ∼6% and thus is not likely to explain our results.

44.

TeSlaa, T. ∙ Bartman, C.R. ∙ Jankowski, C.S.R....

**The source of glycolytic intermediates in mammalian tissues**

*Cell Metabol.* 2021; **33**:367-378.e5

Instead, oxidation of carbohydrate is responsible for ∼90% of post-prandial glucose clearance.

44.

TeSlaa, T. ∙ Bartman, C.R. ∙ Jankowski, C.S.R....

**The source of glycolytic intermediates in mammalian tissues**

*Cell Metabol.* 2021; **33**:367-378.e5

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

45.

DeFronzo, R.A. ∙ Tripathy, D.

**Skeletal muscle insulin resistance is the primary defect in type 2 diabetes**

*Diabetes Care.* 2009; **32**:S157-S163

[Crossref](https://doi.org/10.2337/dc09-S302)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/19875544/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.2337%2Fdc09-S302&pmid=19875544)

To assess these mechanisms, we leveraged pyruvate tolerance tests. Classically, the pyruvate tolerance test has been used as a measure of liver gluconeogenesis, yet current studies show that it also reflects liver lactate oxidation, as injected pyruvate is rapidly reduced to lactate in the blood.

55.

Patgiri, A. ∙ Skinner, O.S. ∙ Miyazaki, Y....

**An engineered enzyme that targets circulating lactate to alleviate intracellular NADH:NAD(+) imbalance**

*Nat. Biotechnol.* 2020; **38**:309-313

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

56.

Rabinowitz, J.D. ∙ Enerbäck, S.

**Lactate: the ugly duckling of energy metabolism**

*Nat. Metab.* 2020; **2**:566-571

We observed a larger and more sustained increase in blood glucose in response to injected pyruvate in KO mice, whereas the curves of Liver-RE and Liver+Muscle-RE mice were similar to WT ([Figures 7](https://www.cell.com/cell-reports/fulltext/#fig7) E and [S13](https://www.cell.com/cell-reports/fulltext/#mmc1) D). These data suggest that liver *Bmal1* is sufficient to limit gluconeogenic flux and/or promote oxidation of pyruvate/lactate. Taken together, our results suggest that improved glucose tolerance in Liver+Muscle-RE mice under NF relies on the coordinated actions of muscle BMAL1 to metabolize incoming glucose, aided by higher insulin levels, and of liver BMAL1 to promote the clearance of muscle-derived metabolites of glucose ([Figure 7](https://www.cell.com/cell-reports/fulltext/#fig7) F).

## Discussion

Interconnected molecular clocks in cells of the brain and body regulate metabolism. Here, we tested the interactions between clocks in liver and muscle, revealing that rescuing clock function in liver, muscle, or both tissues in combination, was not sufficient for glucose tolerance, a phenotype used to assess impairment of glucose homeostasis in rodents and humans. This insufficiency was observed despite well-established roles for liver and muscle clocks in the regulation of glucose pathways.

25.

Harfmann, B.D. ∙ Schroder, E.A. ∙ Kachman, M.T....

**Muscle-specific loss of Bmal1 leads to disrupted tissue glucose metabolism and systemic glucose homeostasis**

*Skelet. Muscle.* 2016; **6**:12

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

27.

Dyar, K.A. ∙ Ciciliot, S. ∙ Wright, L.E....

**Muscle insulin sensitivity and glucose metabolism are controlled by the intrinsic muscle clock**

*Mol. Metabol.* 2014; **3**:29-41

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

32.

Lamia, K.A. ∙ Storch, K.F. ∙ Weitz, C.J.

**Physiological significance of a peripheral tissue circadian clock**

*Proc. Natl. Acad. Sci. USA.* 2008; **105**:15172-15177

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

43.

Manella, G. ∙ Sabath, E. ∙ Aviram, R....

**The liver-clock coordinates rhythmicity of peripheral tissues in response to feeding**

*Nat. Metab.* 2021; **3**:829-842

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

57.

Chaix, A. ∙ Lin, T. ∙ Le, H.D....

**Time-restricted feeding prevents obesity and metabolic syndrome in mice lacking a circadian clock**

*Cell Metabol.* 2019; **29**:303-319.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

58.

Doi, R. ∙ Oishi, K. ∙ Ishida, N.

**CLOCK regulates circadian rhythms of hepatic glycogen synthesis through transcriptional activation of Gys2**

*J. Biol. Chem.* 2010; **285**:22114-22121

We reconciled this discrepancy by showing that a daily feeding-fasting rhythm is a key input to liver and muscle clocks that enables synergy between the two tissues for control of glucose metabolism at the systemic level.

Analysis of transcriptomes and metabolism revealed that the observed synergy among the three circadian system components was multi-faceted. Genes involved in glycolysis, gluconeogenesis, glucose oxidation, and related pathways, are driven daily by clock transcription factors, including *Bmal1*.

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

27.

Dyar, K.A. ∙ Ciciliot, S. ∙ Wright, L.E....

**Muscle insulin sensitivity and glucose metabolism are controlled by the intrinsic muscle clock**

*Mol. Metabol.* 2014; **3**:29-41

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

31.

Kalsbeek, A. ∙ la Fleur, S. ∙ Fliers, E.

**Circadian control of glucose metabolism**

*Mol. Metabol.* 2014; **3**:372-383

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

32.

Lamia, K.A. ∙ Storch, K.F. ∙ Weitz, C.J.

**Physiological significance of a peripheral tissue circadian clock**

*Proc. Natl. Acad. Sci. USA.* 2008; **105**:15172-15177

This basal transcriptional state, which results in a partial ability to metabolize glucose, is further shaped by the interaction between clock transcription factors and feeding-fasting rhythms.

12.

Guan, D. ∙ Xiong, Y. ∙ Trinh, T.M....

**The hepatocyte clock and feeding control chronophysiology of multiple liver cell types**

*Science.* 2020; **369**:1388-1394

[Crossref](https://doi.org/10.1126/science.aba8984)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/32732282/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1126%2Fscience.aba8984&pmid=32732282)

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

14.

Greenwell, B.J. ∙ Trott, A.J. ∙ Beytebiere, J.R....

**Rhythmic food intake drives rhythmic gene expression more potently than the hepatic circadian clock in mice**

*Cell Rep.* 2019; **27**:649-657.e5

We have previously shown that integration of feeding signals by the autonomous liver clock enhances glucose homeostasis in the liver.

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

Here, we show a similar response in muscle. Feeding rhythms also overcame the defect in glucose-stimulated insulin release observed for mice deficient for pancreatic clock function

51.

Petrenko, V. ∙ Saini, C. ∙ Giovannoni, L....

**Pancreatic alpha- and beta-cellular clocks have distinct molecular properties and impact on islet hormone secretion and gene expression**

*Genes Dev.* 2017; **31**:383-398

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

59.

Perelis, M. ∙ Marcheva, B. ∙ Ramsey, K.M....

**Pancreatic beta cell enhancers regulate rhythmic transcription of genes controlling insulin secretion**

*Science.* 2015; **350**:aac4250

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

60.

Brown, M.R. ∙ Sen, S.K. ∙ Mazzone, A....

**Time-restricted feeding prevents deleterious metabolic effects of circadian disruption through epigenetic control of beta cell function**

*Sci. Adv.* 2021; **7**:eabg6856

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

61.

Sadacca, L.A. ∙ Lamia, K.A. ∙ deLemos, A.S....

**An intrinsic circadian clock of the pancreas is required for normal insulin release and glucose homeostasis in mice**

*Diabetologia.* 2011; **54**:120-124

*,* including both KO and RE mice, by boosting serum insulin. Insulin, in turn, drives glucose uptake into muscle, wherein local *Bmal1* temporally primes the machinery to process it. Rescue of muscle *Bmal1* alone modestly improved glucose clearance, yet simultaneous actions involving *Bmal1* -dependent functions in liver were required for maximal clearance. Liver takes up muscle-derived metabolites of glucose, including lactate and alanine, thereby clearing glucose-derived carbons from the circulation,

34.

Hui, S. ∙ Cowan, A.J. ∙ Zeng, X....

**Quantitative fluxomics of circulating metabolites**

*Cell Metabol.* 2020; **32**:676-688.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

37.

Cori, C.F.

**The glucose-lactic acid cycle and gluconeogenesis**

*Curr. Top. Cell. Regul.* 1981; **18**:377-387

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

38.

Felig, P. ∙ Pozefsky, T. ∙ Marliss, E....

**Alanine: key role in gluconeogenesis**

*Science.* 1970; **167**:1003-1004

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

62.

Jang, C. ∙ Hui, S. ∙ Zeng, X....

**Metabolite exchange between mammalian organs quantified in pigs**

*Cell Metabol.* 2019; **30**:594-606.e3

conceivably freeing muscle to take up more glucose. The synchronized engagement of these events presents a plausible explanation for improved glucose tolerance. Additionally, reconstitution of *Bmal1* in the suprachiasmatic nucleus restores a feeding rhythm and partially restores glucose tolerance.

39.

Petrus, P. ∙ Smith, J.G. ∙ Koronowski, K.B....

**The central clock suffices to drive the majority of circulatory metabolic rhythms**

*Sci. Adv.* 2022; **8**:eabo2896

Thus, the central clock in the brain is also a critical piece of the glucose homeostasis framework, even in the absence of intact clocks in liver and muscle. The synergy between liver and muscle clocks may be aided by additional mechanisms, such as soluble factors or extracellular vesicles released by muscle and liver.

63.

Seldin, M.M. ∙ Peterson, J.M. ∙ Byerly, M.S....

**Myonectin (CTRP15), a novel myokine that links skeletal muscle to systemic lipid homeostasis**

*J. Biol. Chem.* 2012; **287**:11968-11980

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

64.

Seldin, M.M. ∙ Lei, X. ∙ Tan, S.Y....

**Skeletal muscle-derived myonectin activates the mammalian target of rapamycin (mTOR) pathway to suppress autophagy in liver**

*J. Biol. Chem.* 2013; **288**:36073-36082

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

65.

Severinsen, M.C.K. ∙ Pedersen, B.K.

**Muscle-organ crosstalk: the emerging roles of myokines**

*Endocr. Rev.* 2020; **41**:594-609

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

66.

Bookout, A.L. ∙ de Groot, M.H.M. ∙ Owen, B.M....

**FGF21 regulates metabolism and circadian behavior by acting on the nervous system**

*Nat. Med.* 2013; **19**:1147-1152

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

67.

Ma, X. ∙ Zhou, Z. ∙ Chen, Y....

**RBP4 functions as a hepatokine in the regulation of glucose metabolism by the circadian clock in mice**

*Diabetologia.* 2016; **59**:354-362

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

68.

Chen, S. ∙ Feng, M. ∙ Zhang, S....

**Angptl8 mediates food-driven resetting of hepatic circadian clock in mice**

*Nat. Commun.* 2019; **10**:3518

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

69.

Priest, C. ∙ Tontonoz, P.

**Inter-organ cross-talk in metabolic syndrome**

*Nat. Metab.* 2019; **1**:1177-1188

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

70.

Whitham, M. ∙ Parker, B.L. ∙ Friedrichsen, M....

**Extracellular vesicles provide a means for tissue crosstalk during exercise**

*Cell Metabol.* 2018; **27**:237-251.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

71.

Perrin, L. ∙ Loizides-Mangold, U. ∙ Skarupelova, S....

**Human skeletal myotubes display a cell-autonomous circadian clock implicated in basal myokine secretion**

*Mol. Metabol.* 2015; **4**:834-845

These mechanisms remain to be investigated.

This study also sheds light on the contributions of different cell types and systemic signals to circadian physiology in skeletal muscle. Single nuclei sequencing experiments show that ∼70% of muscle nuclei are myonuclei, which constitute the differentiated muscle fibers, while the additional ∼30% comprises fibrogenic progenitors, tenocytes, stem cells, endothelial cells, and immune cells.

72.

Dos Santos, M. ∙ Backer, S. ∙ Auradé, F....

**A fast Myosin super enhancer dictates muscle fiber phenotype through competitive interactions with Myosin genes**

*Nat. Commun.* 2022; **13**:1039

Here, we used *Hsa* -Cre to reconstitute *Bmal1* expression specifically in the myofiber lineage.

73.

Miniou, P. ∙ Tiziano, D. ∙ Frugier, T....

**Gene targeting restricted to mouse striated muscle lineage**

*Nucleic Acids Res.* 1999; **27**:e27

Interestingly, whole-tissue analysis revealed weaker oscillation of clock genes in this model, indicating that either additional cell types and/or additional systemic signals contribute significantly to the tissue ensemble rhythm. It is unlikely that this is a technical artifact of the model, as we observed high Cre-mediated recombination efficiency. In addition, our analysis showed that the temporal expression pattern of clock genes, such as *Rev-Erbα*, was restored under NF conditions, meaning that lack of this systemic signal, rather than *Rev-Erbα* expression in other cell types, was responsible for the lack of *Rev-Erbα* oscillation under AL conditions. Still, not all clock genes were rescued. Another systemic signal of importance to muscle is daily periods of locomotion or exercise. Exercise can entrain the muscle clock and affect its functional outputs.

74.

Sato, S. ∙ Basse, A.L. ∙ Schönke, M....

**Time of exercise specifies the impact on muscle metabolic pathways and systemic energy homeostasis**

*Cell Metabol.* 2019; **30**:92-110.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

75.

Small, L. ∙ Altıntaş, A. ∙ Laker, R.C....

**Contraction influences Per2 gene expression in skeletal muscle through a calcium-dependent pathway**

*J. Physiol.* 2020; **598**:5739-5752

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

76.

Wolff, G. ∙ Esser, K.A.

**Scheduled exercise phase shifts the circadian clock in skeletal muscle**

*Med. Sci. Sports Exerc.* 2012; **44**:1663-1670

As KO and RE mice do not exhibit the robust locomotor activity of WT, restoring this additional behavior is likely important for further restoration of muscle rhythms in RE mice. Nevertheless, *Bmal1* is expressed in other skeletal-muscle-based cell types, for example muscle stem cells; therefore, these non-myofiber lineage cell types may also contribute. A more thorough interrogation at the single-cell level is warranted.

### Limitations of the study

Factors that define systemic glucose metabolism, such as insulin secretion and sensitivity, the release of peptides from the gut, and neuronal circuits that control energy expenditure, are temporally regulated by the clock system.

31.

Kalsbeek, A. ∙ la Fleur, S. ∙ Fliers, E.

**Circadian control of glucose metabolism**

*Mol. Metabol.* 2014; **3**:372-383

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

61.

Sadacca, L.A. ∙ Lamia, K.A. ∙ deLemos, A.S....

**An intrinsic circadian clock of the pancreas is required for normal insulin release and glucose homeostasis in mice**

*Diabetologia.* 2011; **54**:120-124

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

77.

Gil-Lozano, M. ∙ Mingomataj, E.L. ∙ Wu, W.K....

**Circadian secretion of the intestinal hormone GLP-1 by the rodent L cell**

*Diabetes.* 2014; **63**:3674-3685

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

78.

Ding, G. ∙ Li, X. ∙ Hou, X....

**REV-ERB in GABAergic neurons controls diurnal hepatic insulin sensitivity**

*Nature.* 2021; **592**:763-767

Accordingly, studies report that glucose tolerance displays circadian rhythmicity in rodents.

78.

Ding, G. ∙ Li, X. ∙ Hou, X....

**REV-ERB in GABAergic neurons controls diurnal hepatic insulin sensitivity**

*Nature.* 2021; **592**:763-767

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

79.

la Fleur, S.E. ∙ Kalsbeek, A. ∙ Wortel, J....

**A daily rhythm in glucose tolerance: a role for the suprachiasmatic nucleus**

*Diabetes.* 2001; **50**:1237-1243

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

80.

Zhong, L.X. ∙ Li, X.N. ∙ Yang, G.Y....

**Circadian misalignment alters insulin sensitivity during the light phase and shifts glucose tolerance rhythms in female mice**

*PLoS One.* 2019; **14**:e0225813

A limitation of this study is that tolerance tests were performed at a single diurnal time point. Therefore, results may not necessarily represent glucose responses at all diurnal time points. Because fasting is required for glucose tolerance tests (to ensure the clearance of acute food intake from the gut), we sought to align this experimental fast with the natural fasting period of the mouse (i.e., the light phase). With this setup, we avoided subjecting mice to starvation-like states associated with long fasts, and results were more reflective of physiological mechanisms.

81.

Virtue, S. ∙ Vidal-Puig, A.

**GTTs and ITTs in mice: simple tests, complex answers**

*Nat. Metab.* 2021; **3**:883-886

In addition, the metabolic activities of liver and muscle are relevant to both light and dark phase glucose control. For example, shuttling of glucose-derived metabolites from muscle to liver (including recycling of lactate back to glucose) occurs in both the fed and fasted states.

34.

Hui, S. ∙ Cowan, A.J. ∙ Zeng, X....

**Quantitative fluxomics of circulating metabolites**

*Cell Metabol.* 2020; **32**:676-688.e4

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

35.

Hui, S. ∙ Ghergurovich, J.M. ∙ Morscher, R.J....

**Glucose feeds the TCA cycle via circulating lactate**

*Nature.* 2017; **551**:115-118

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

44.

TeSlaa, T. ∙ Bartman, C.R. ∙ Jankowski, C.S.R....

**The source of glycolytic intermediates in mammalian tissues**

*Cell Metabol.* 2021; **33**:367-378.e5

Therefore, it is relevant to interrogate liver-muscle cross talk in the light phase.

## STAR★Methods

### Key resources table

<table><thead><tr><th>REAGENT or RESOURCE</th><th>SOURCE</th><th>IDENTIFIER</th></tr></thead><tbody><tr><td colspan="3"><b>Antibodies</b></td></tr><tr><td>Anti-BMAL1</td><td>Abcam</td><td>Cat# ab93806; RRID: <a href="https://antibodyregistry.org/AB_10675117">AB_10675117</a></td></tr><tr><td>Anti-p84</td><td>Genetex</td><td>Cat# GTX70220; RRID: <a href="https://antibodyregistry.org/AB_372637">AB_372637</a></td></tr><tr><td>Anti-RFP</td><td>Rockland</td><td>Cat# 600-401-379; RRID: <a href="https://antibodyregistry.org/AB_2209751">AB_2209751</a></td></tr><tr><td>Goat anti-rabbit IgG H&amp;L (Alexa Fluor® 647)</td><td>Abcam</td><td>Cat# ab150079; RRID: <a href="https://antibodyregistry.org/AB_2722623">AB_2722623</a></td></tr><tr><td>Anti-Tubulin</td><td>Sigma</td><td>Cat# T6199; RRID: <a href="https://antibodyregistry.org/AB_477583">AB_477583</a></td></tr><tr><td colspan="3"><b>Chemicals, peptides, and recombinant proteins</b></td></tr><tr><td>U− <sup>13</sup> C <sub>6</sub> glucose</td><td>Cambridge isotope Laboratories</td><td>CLM-1396-PK</td></tr><tr><td colspan="3"><b>Critical commercial assays</b></td></tr><tr><td>Ultra Sensitive mouse insulin ELISA kit</td><td>Crystal Chem</td><td>90080</td></tr><tr><td colspan="3"><b>Deposited data</b></td></tr><tr><td>RNA sequencing – WT and double RE liver and muscle</td><td>This paper</td><td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE197455">GSE197455</a></td></tr><tr><td>RNA sequencing – WT, <i>Bmal1</i> KO, and Muscle-RE muscle</td><td>This paper</td><td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE197726">GSE197726</a></td></tr><tr><td>RNA sequencing – Night feeding dataset, all genotypes, liver and muscle</td><td>This paper</td><td><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228389">GSE228389</a></td></tr><tr><td colspan="3"><b>Experimental models: Organisms/strains</b></td></tr><tr><td>Mouse: Wild Type (WT) – 1.<i>Bmal1</i> <sup><i>wt/wt</i></sup>, Alfp <i>-</i> cre <sup>tg/0</sup>; 2.<br><i>Bmal1</i> <sup><i>wt/wt</i></sup>, Hsa <i>-</i> cre <sup>tg/0</sup>; 3. <i>Bmal1</i> <sup><i>wt/wt</i></sup>, Alfp <i>-</i> cre <sup>tg/0</sup>, Hsa <i>-</i> cre <sup>tg/0</sup></td><td>S.A. Benitah lab, P.S. Welz, This study</td><td></td></tr><tr><td>Mouse: <i>Bmal1</i> knockout (KO)&nbsp;– <i>Bmal1</i> <sup><i>stopFL/stopFL</i></sup></td><td>S.A. Benitah lab, P.S. Welz, This study</td><td></td></tr><tr><td>Mouse: <i>Bmal1</i> hepatocyte-reconstituted (Liver-RE)<br>– <i>Bmal1 <sup>stopFL/stopFL</sup></i>; Alfp-cre <sup>tg/0</sup></td><td>S.A. Benitah lab, P.S. Welz, This study</td><td></td></tr><tr><td>Mouse: <i>Bmal1</i> skeletal muscle-reconstituted (Muscle-RE)&nbsp;– <i>Bmal1 <sup>stopFL/stopFL</sup></i>; Hsa-cre <sup>tg/0</sup></td><td>S.A. Benitah lab, P.S. Welz, This study</td><td></td></tr><tr><td>Mouse: <i>Bmal1</i> hepatocyte&nbsp;+ skeletal muscle- reconstituted (Liver+Muscle-RE)&nbsp;– <i>Bmal1</i> <sup><i>stopFL/stopFL</i></sup>; Alfp- cre <sup>tg/0</sup>; Hsa-cre <sup>tg/0</sup></td><td>S.A. Benitah lab, P.S. Welz, This study</td><td></td></tr><tr><td colspan="3"><b>Oligonucleotides</b></td></tr><tr><td>BMAL1 For - GCAGTGCCACTGACTACCAAGA</td><td>This study</td><td></td></tr><tr><td>BMAL1 Rev - TCCTGGACATTGCATTGCAT</td><td>This study</td><td></td></tr><tr><td>Rn18S For - CGCCGCTAGAGGTGAAATTC</td><td>This study</td><td></td></tr><tr><td>Rn18S Rev - CGAACCTCCGACTTTCGTTCT</td><td>This study</td><td></td></tr><tr><td colspan="3"><b>Software and algorithms</b></td></tr><tr><td><i>dryR</i></td><td>Weger et al., 2021,<br><i>PNAS</i></td><td></td></tr><tr><td>JTK_CYCLE</td><td>Hughes et al., 2010, <i>J. Biol. Rhythms.</i></td><td></td></tr><tr><td>DAVID Bioinformatics Resource</td><td>Huang, Sherman and Lempicki, 2009, <i>Nat. Protoc.</i></td><td></td></tr><tr><td>Nitecap</td><td>Brooks et al., 2022, <i>J. Biol. Rhythms.</i></td><td></td></tr></tbody></table>

- [Open table in a new tab](https://www.cell.com/action/showFullTableHTML?isHtml=true&tableId=undtbl1&pii=S2211-1247%2823%2900599-5)

### Resource availability

#### Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kevin Koronowski ([Koronowski@uthscsa.edu](https://www.cell.com/cell-reports/fulltext/))

#### Materials availability

Mouse lines generated in this study are available from the [lead contact](https://www.cell.com/cell-reports/fulltext/#sec-4-2-1) with a completed materials transfer agreement.

### Experimental model and study participant details

#### Animals

Mice were bred and housed at the University of California Irvine vivarium, in accordance with the guidelines of the local Institutional Animal Care and Use Committee (IACUC). Animal experiments conformed to institutional and national regulatory standards and were performed considering ARRIVE (Animal Research: Reporting of In Vivo Experiments) guidelines. *Bmal1* - *stopFL* mice were generated on the C57BL/6J background as previously described

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

<sup xmlns="http://www.w3.org/1999/xhtml">,</sup>

17.

Welz, P.S. ∙ Zinna, V.M. ∙ Symeonidi, A....

**BMAL1-Driven tissue clocks Respond independently to light to maintain homeostasis**

*Cell.* 2019; **177**:1436-1447.e12

(see also [Figure 1](https://www.cell.com/cell-reports/fulltext/#fig1)). Liver reconstituted (Liver-RE; hepatocyte-specific reconstitution of *Bmal1*) mice were generated by crossing *Bmal1* - *stopFL* with *Alfp* -Cre mice as previously described.

15.

Koronowski, K.B. ∙ Kinouchi, K. ∙ Welz, P.S....

**Defining the independence of the liver circadian clock**

*Cell.* 2019; **177**:1448-1462.e14

Skeletal muscle reconstituted (Muscle-RE; myocyte-specific reconstitution of *Bmal1*) mice were generated by crossing *Bmal1* - *stopFL* with mice expressing *cre* recombinase driven by the human alpha-skeletal actin (*Hsa*; *Acta1*) promotor (*Hsa* -Cre). Crosses of *Bmal1* - *stopFL* with both Cre lines generated Liver+Muscle reconstituted (Liver+Muscle-RE) mice. Experimental genotypes were: wild type (WT) – 1. Bmal1 <sup xmlns="http://www.w3.org/1999/xhtml"><i>wt/wt</i></sup>, Alfp *\-* cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>; 2. Bmal1 <sup xmlns="http://www.w3.org/1999/xhtml"><i>wt/wt</i></sup>, Hsa *\-* cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>; 3. Bmal1 <sup xmlns="http://www.w3.org/1999/xhtml"><i>wt/wt</i></sup>, Alfp *\-* cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>, Hsa *\-* cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>. *Bmal1* knockout (KO) – *Bmal1* <sup xmlns="http://www.w3.org/1999/xhtml"><i>stopFL/sto-FL</i></sup>; Bmal1 hepatocyte-reconstituted (Liver-RE) – *Bmal1* <sup xmlns="http://www.w3.org/1999/xhtml"><i>stopFL/stopFL</i></sup>; Alfp-cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>. Bmal1 skeletal muscle-reconstituted (Muscle-RE) – *Bmal1* <sup xmlns="http://www.w3.org/1999/xhtml"><i>stopFL/stopFL</i></sup>; Hsa-cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>. Bmal1 hepatocyte + skeletal muscle-reconstituted (Liver+Muscle-RE) – *Bmal1* <sup xmlns="http://www.w3.org/1999/xhtml"><i>stopFL/stopFL</i></sup>; Alfp-cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>; Hsa-cre <sup xmlns="http://www.w3.org/1999/xhtml">tg/0</sup>. The specific genotypes of WT controls for each experiment are listed in the figure legends. All experiments utilized 8- to 12-week-old mice entrained to a 12 h light/12 h dark cycle and fed *ad libitum* or placed under night feeding (NF) wherein mice were given *ad libitum* access to food during the dark period from zeitgeber time (ZT)12 to ZT24/0. For feasibility purposes, males were used for transcriptomic analyses and females for functional experiments. Key functional experiments were confirmed in both male and female mice.

### Method details

#### Western blot

Frozen tissue was homogenized in lysis buffer (RIPA - 50 mM Tris-HCl \[pH 8\], 150 mM NaCl, 5 mM EDTA, 15 mM MgCl <sub>2</sub> and 1% NP-40) supplemented with Protease Inhibitor Cocktail (Roche, Basel, Switzerland), 500 μM PMSF (serine protease inhibitor), 10 mM nicotinamide (Sirtuin inhibitor) and 330 nM TSA (Class I and II HDAC inhibitor). Samples were lysed on ice with periodic mixing for 30 min. Next, samples were sonicated (5 s on, 5 s off, 4 cycles, 60% amp.) and centrifuged at 13,200 rpm at 4°C for 15 min and the supernatant was collected. We used the Braford method to determine protein concentration, using Protein Assay Dye (BioRad). 20 μg protein from each sample was separated on 6% or 8% gels by SDS-PAGE and subsequently transferred to a nitrocellulose membrane. Membranes were then blocked with 5% milk in TBS-T (0.1% Tween 20, TBS) at room temperature for 2 h. Primary antibodies were diluted in 5% milk TBS-T and incubated with blots at 4°C overnight (BMAL1, Abcam – ab93806; ACTIN, Abcam – ab3280, P84, Genetex – GTX70220). Blots were incubated with HRP-conjugated secondary antibodies (Mouse IgG-HRP conjugate, EMD Millipore – AP160P; Rabbit IgG-HRP linked, EMD Millipore – 12–348) in 5% milk in TBS-T at room temperature for 1 h and visualized with Immobilon Western chemiluminescent HRP substrate (Millipore, Burlington, MA). Finally, blots were developed on HyBlot CL autoradiography film (Denville Scientific, Holliston, MA).

#### mCHERRY Immunostaining analysis

Freshly harvested gastrocnemius muscle was fixed for 120 min in 2% PFA, then washed 2 x with PBS, before incubation in 1:1 sucrose/OCT for 30 min, then frozen in OCT in tissue blocks using liquid-nitrogen cooled isobutane. 10 μm transverse gastrocnemius muscle sections were incubated in 5% w/v BSA 5% w/v normal goat serum 5.0% v/v Triton in PBS for 2 h, washed 3 times with PBS then incubated with anti-RFP antibody (ROCKLAND, 600-401-379, 1:250, anti-RFP antibodies detect mCHERRY) in 5% w/v BSA 5% w/v normal goat serum 5.0% v/v Triton in PBS overnight at 4°C. Slides were washed 3 times in PBS then incubated in Alexa Fluor 647 anti-rabbit secondary antibody (Abcam ab150079) with DAPI 1/2000 in PBS for 2 h at RT, washed 3 times in PBS then cover slips mounted using Fluoromount G. Images were acquired using Cell Observer (Zeiss) and analyzed using FIJI. Fibers were traced manually using FIJI and each assigned as a ROI, integrated density was then calculated for each. A minimum of 100 ROI and 3 fields of view were used for each mouse.

#### Indirect calorimetry, food intake, and body composition

Metabolic parameters were measured using the Phenomaster metabolic cage system (TSE Systems Inc., Bad Homburg, Germany). The climate chamber was set to 21°C and 50% humidity with a 12h:12h light–dark cycle. Mice were maintained on standard chow (Teklad 2020x) *ad libitum* or time-restricted during the 12 h dark phase as indicated, singly housed, and acclimated for 48h prior to data collection. VO <sub>2</sub>, VCO <sub>2</sub>, respiratory exchange ratio (RER), energy expenditure and food intake were monitored for 3 min per cage and measured every 39 min for four to six consecutive days. Body composition (lean mass and fat mass) was measured in non-anesthetized mice via MRI (QNMR EchoMRI100; Echo Medical Systems) prior to recordings. Food intake values were calculated by multiplying average grams eaten per hour by the kcal/g of the chow diet.

#### Locomotor activity

Mice were individually housed in home cages equipped with optical beam motion detection capability (Starr Life Sciences). Data were collected using VitalView 5.0 software and analyzed using Clocklab software (Actimetrics).

#### RNA-sequencing for ad libitum samples

Total RNA was extracted by Trizol for both liver and muscle samples, with muscle samples further purified using Direct-Zol columns (Zymo). RNA was monitored for quality control using the Agilent Bioanalyzer Nano RNA chip and Nanodrop absorbance ratios for 260/280nm and 260/230nm. Library construction was performed according to the Illumina TruSeq Total RNA stranded protocol. rRNA was depleted using ribo-zero rRNA gold removal kit (human/mouse/rat). rRNA-depleted RNA was chemically fragmented for 3 min. First strand synthesis was completed using random primers and reverse transcriptase to make cDNA. After second strand synthesis, cDNA was purified using AMPure XP beads, end-repaired and 3′ ends were adenylated. Illumina unique dual indexed adapters were ligated on the ends and the adapter-ligated fragments were enriched by nine cycles of PCR. The resulting libraries were validated by qPCR and sized by Agilent Bioanalyzer DNA high sensitivity chip. The concentrations for the libraries were normalized and then multiplexed together. The multiplexed libraries were sequenced using paired-end 100 cycles chemistry on the NovaSeq 6000.

#### RNA-sequencing for NF samples

NGS experiments for NF samples were performed in the Genomics Unit of the CNIC. RNA extractions on liver samples were performed using RNeasy columns (Qiagen) and Direct-Zol columns (Zymo) were used for muscle samples. Quality control monitoring was performed as above for ad libitum samples, then single read sequencing was performed using the Illumina NextSeq 2000 System to achieve a minimum of 25 million reads per sample.

#### RNA-sequencing analysis for ad libitum samples

Sequencing results arrived in FastQ format. The pair-end reads from each replicate experiment were aligned to the reference genome assembly mm10 and the corresponding transcriptome using the Tuxedo protocol. Reads uniquely aligned to known exons, or splice junctions, extracted with no more than two mismatches were included in the transcriptome. Gene expression levels were directly computed from the read alignment results for each replicate. Standard FPKM (fragments per kilobase of exon per million mapped reads) values were extracted for each gene covered by the sequencing data and each replicate used in this study. Note, ad libitum RNA-sequencing data from livers of WT- *Alfp* -Cre, KO and Liver-RE is from our previous publication

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

and was reanalysed by *dryR*

20.

Weger, B.D. ∙ Gobet, C. ∙ David, F.P.A....

**Systematic analysis of differential rhythmic liver gene expression mediated by the circadian clock and feeding rhythms**

*Proc. Natl. Acad. Sci. USA.* 2021; **118**

e2015803118

[Crossref](https://doi.org/10.1073/pnas.2015803118)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/33452134/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2015803118&pmid=33452134)

alongside newly produced RNA-sequencing data of livers from WT- *Alfp* + *Hsa-Cre* and Liver+Muscle-RE mice. Fresh RNA extractions from tissue were performed on ZT4 and ZT16 liver samples of WT- *Alfp-* Cre, KO and Liver-RE night-fed mice collected for our previous study,

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

to minimise extraction and sequencing batch effects, and allow high confidence comparisons with WT- *Alfp-* Cre, KO, and Liver+Muscle-RE samples. Where shown, single RE or KO mice were normalized to the *Alfp* -Cre- or *Hsa* -Cre-positive WT control, double RE mice were normalized to the *Alfp+Hsa* -Cre-positive WT control, and data are displayed relative to WT controls within that feeding condition.

#### RNA-sequencing data analysis for NF samples

Quality assessment, alignment and differential gene expression analysis were performed by Sequentia Biotech (Barcelona). Output FASTQ files from the sequencer as well as the quality of the reads were assessed using FASTQC v0.11.9 ([http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)). Then, adapters and low quality bases were removed from the reads during a sequence-trimming step by means of BBDuk v35.85 ([https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)). Reads with less than 35 bp and less than a quality score of 25 were discarded.

High quality reads were mapped against the *Mus musculus* genome (GRCm38/Ensembl release 101 reference genome and gene annotation) employing STAR v2.7.1a.

82.

Dobin, A. ∙ Davis, C.A. ∙ Schlesinger, F....

**STAR: ultrafast universal RNA-seq aligner**

*Bioinformatics.* 2013; **29**:15-21

Subsequently, FeatureCounts v 1.5.1

83.

Liao, Y. ∙ Smyth, G.K. ∙ Shi, W.

**featureCounts: an efficient general purpose program for assigning sequence reads to genomic features**

*Bioinformatics.* 2014; **30**:923-930

was used to quantify gene expression in terms of raw fragment counts. The latter information was displayed as an expression matrix that was loaded into R v4.0.3. Uninformative genes were filtered out from the matrix using HTSFilter package v1.30.1.

84.

Rau, A. ∙ Gallopin, M. ∙ Celeux, G....

**Data-based filtering for replicated high-throughput transcriptome sequencing experiments**

*Bioinformatics.* 2013; **29**:2146-2152

Then, filtered raw counts were normalized by applying the TMM method. The differential gene expression analysis was conducted by the DESeq2 package v3.16

85.

Love, M.I. ∙ Huber, W. ∙ Anders, S.

**Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2**

*Genome Biol.* 2014; **15**:550

in order to identify upregulated/downregulated genes. Multiple testing correction was performed with the FDR method and significance level was set at FDR <0.05.

For AL vs. NF comparisons, non-autonomously oscillating genes were further narrowed down to those that exhibited a significant difference in expression between ZT4 and ZT16 in WT mice under NF.

#### qPCR

For liver and muscle, 500 ng-1 μg of Trizol-extracted RNA was reverse transcribed using Thermo Scientific Maxima Reverse Transcriptase and qPCR was performed using Syber Green Master Mix on a ThermoFisher Quantstudio 3. All gene expression data were normalized to 18S ribosomal RNA. Primer sequences are listed in the [key resources table](https://www.cell.com/cell-reports/fulltext/#sec-4-1).

#### Rhythmicity and pathway analysis

To identify oscillating transcripts, transcriptomic data for each tissue was processed via *dryR*.

20.

Weger, B.D. ∙ Gobet, C. ∙ David, F.P.A....

**Systematic analysis of differential rhythmic liver gene expression mediated by the circadian clock and feeding rhythms**

*Proc. Natl. Acad. Sci. USA.* 2021; **118**

e2015803118

[Crossref](https://doi.org/10.1073/pnas.2015803118)

[PubMed](https://pubmed.ncbi.nlm.nih.gov/33452134/)

[Google Scholar](https://scholar.google.com/scholar_lookup?doi=10.1073%2Fpnas.2015803118&pmid=33452134)

We used 5 group comparisons for each tissue. For liver: WT- *Alfp* -Cre, WT- *Alfp* + *Hsa* -Cre, KO, Liver-RE, Liver+Muscle-RE. For muscle: WT- *Hsa* -Cre, WT- *Alfp* + *Hsa* -Cre, KO, Muscle-RE, Liver+Muscle-RE. Genes with a BICW <0.25 and amplitude <0.25 were excluded. Only genes rhythmic in both WTs (single and double cre) were included, though WT- *Alfp* + *Hsa* -Cre is shown in heatmaps for clarity (see also [Tables S1](https://www.cell.com/cell-reports/fulltext/#mmc2) and [S2](https://www.cell.com/cell-reports/fulltext/#mmc3)). Genes within each heatmap were normalised within each genotype due to observed batch effect in regards to overall levels of mRNA expression for muscle samples harvested in either batch 1 (WT- *Hsa* -Cre, KO, Muscle-RE) or batch 2 (WT- *Alfp* + *Hsa* -Cre, Liver+Muscle-RE). As such, for both muscle and liver, 3 group *dryR* was used to compare mean levels within batch 1 for muscle (WT- *Hsa* -Cre, KO, Muscle) and within batch 1 for liver (WT- *Alfp* -Cre, KO, Liver-RE). Gene ontology analysis was performed using DAVID

87.

Huang, D.W. ∙ Sherman, B.T. ∙ Lempicki, R.A.

**Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources**

*Nat. Protoc.* 2009; **4**:44-57

for biological process for genes classifed into different *dryR* models. We performed select 2 group comparisons using Nitecap

28.

Brooks, T.G. ∙ Mrčela, A. ∙ Lahens, N.F....

**Nitecap: an exploratory circadian analysis web application**

*J. Biol. Rhythm.* 2022; **37**:43-52

to identify genes with a significant change in the shape of their profile as determined by two-way ANOVA p < 0.01.

#### Glucose, insulin, and pyruvate tolerance tests

Blood glucose was measured via the tail at 0, 15, 30, 60 and 120 min following gavage or intraperitoneal injection using an Accu-Chek Aviva Plus Blood Glucose Monitoring System. Glucose tolerance curves were calculated using current standards

81.

Virtue, S. ∙ Vidal-Puig, A.

**GTTs and ITTs in mice: simple tests, complex answers**

*Nat. Metab.* 2021; **3**:883-886

; baseline was subtracted from raw blood glucose values at each time point to give change in glucose concentration (delta) over time. Raw data are shown in the corresponding supplementary figures. Area under the curve was calculated using standard parameters. For oral glucose tolerance tests, mice were fasted from ZT0 to ZT6 and then gavaged with 1.5 g/kg glucose. For pyruvate tolerance tests, mice were fasted from ZT0 to ZT8 and then injected I.P. with 1.5 g/kg sodium pyruvate. For insulin tolerance tests, mice were fasted from ZT0 to ZT3 and then injected I.P. with 0.3 U/kg insulin.

#### Glucose tracing

Glucose tracing was performed identically to oral glucose tolerance tests, except gavaged glucose was uniformly labeled with carbon 13 (U-

13.

Greco, C.M. ∙ Koronowski, K.B. ∙ Smith, J.G....

**Integration of feeding behavior by the liver circadian clock reveals network dependency of metabolic rhythms**

*Sci. Adv.* 2021; **7**:eabi7828

C <sub xmlns="http://www.w3.org/1999/xhtml">6</sub>, Cambridge Isotope Laboratories, CLM-1396-PK). Tissues were harvested 25–30 min after gavage.

##### Glucose tracing sample preparation and metabolite extraction

Serum (5 μL) was mixed with 150 μL 4°C 40:40:20 methanol:acetonitrile:water (extraction solvent), vortexed and immediately centrifuged at 16,000g for 10 min at 4°C. The supernatant (∼100 μL) was collected for liquid chromatography-mass spectrometry (LC-MS) analysis. Frozen tissue samples were ground at liquid nitrogen temperature with a CryoMill (Retsch). To minimize data variation due to tissue heterogeneity, entire tissues were collected and ground. The resulting tissue powder (approximately 20 mg) was weighed and then extracted by adding 4°C extraction solvent, vortexed and centrifuged at 16,000g for 10 min at 4°C. The volume of the extraction solution (μL) was 40x the weight of tissue (mg) to make an extract of 25 mg of tissue per ml of solvent. The supernatant (40 μL) was collected for LC-MS analysis.

##### Metabolite measurements using LC-MS

A quadrupole orbitrap mass spectrometer (Q Exactive; Thermo Fisher Scientific) operating in negative or positive ion mode was coupled to a Vanquish UHPLC system (Thermo Fisher Scientific) with electrospray ionization and used to scan from m/z 70 to 1,000 at 1 Hz, with a 140,000 resolution. LC separation was achieved on an XBridge BEH Amide column (2.1 × 150 mm <sup>2</sup>, 2.5 μm particle size, 130 Å pore size; Waters Corporation) using a gradient of solvent A (95:5 water: acetonitrile with 20 mM of ammonium acetate and 20 mM of ammonium hydroxide, pH 9.45) and solvent B (acetonitrile). Flow rate was 150 μL/min. The LC gradient was: 0 min, 85% B; 2 min, 85% B; 3 min, 80% B; 5 min, 80% B; 6 min, 75% B; 7 min, 75% B; 8 min, 70% B; 9 min, 70% B; 10 min, 50% B; 12 min, 50% B; 13 min, 25% B; 16 min, 25% B; 18 min, 0% B; 23 min, 0% B; 24 min, 85% B; and 30 min, 85% B. The autosampler temperature was 5°C and the injection volume was 3 μL. Data were analyzed using the MAVEN software (build 682, [http://maven.princeton.edu/index.php](http://maven.princeton.edu/index.php)). Natural isotope correction was performed with AccuCor R code ([https://github.com/lparsons/accucor](https://github.com/lparsons/accucor)).

#### Insulin measurements

Collected blood was placed on ice and subsequently centrifuged at 3,000 rpm for 10 min at 4°C. The supernatant was stored at −80°C. All samples were thawed and used as starting material for the Ultra-Sensitive Mouse Insulin ELISA Kit (Cystal Chem 90080), which was utilized according to the manufacturer’s instructions.

### Quantification and statistical analysis

All data are displayed as mean ± S.E.M. unless otherwise noted. For each experiment, the number of biological replicates (n-value), statistical test and significance threshold can be found in the figure legend or main text. Complex statistical analyses of large-scale datasets are described within the corresponding methods section. Data were analyzed in Prism 6.0 software (GraphPad). The suitability of parametric vs. non-parametric tests was determined by data distribution analysis tools in the software. Sample size was determined by referencing literature standards for studies of circadian rhythms.

88.

Hughes, M.E. ∙ Abruzzi, K.C. ∙ Allada, R....

**Guidelines for genome-scale analysis of biological rhythms**

*J. Biol. Rhythm.* 2017; **32**:380-393

Data were checked for outliers with Grubbs’ test.

## Acknowledgments

This paper is dedicated to Paolo Sassone-Corsi, a hugely inspiring scientist and mentor who remains an important influence on our work. We also thank P.S.C. Lab animal technician S. Sato and laboratory manager W. Orquiz for their valued contributions, as well as Aintzane Rueda and Alfonso Saera-Vila at Sequentia Biotech (Barcelona) for their work on RNA sequencing read alignment and differential gene expression analysis. J.G.S. was supported by Zymo-CEM Postdoctoral Fellowship (Zymo Research) awarded at the University of California, Irvine. K.B.K. was supported by NIH, NIDDK F32 Fellowship – DK121425. T.S. was supported by a Japan Society for the Promotion of Science (JSPS) fellowship. C.M.G. was supported by the National Cancer Institute of the National Institutes of Health under award T32CA009054 and by the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement 749869. P.P. was funded by The Wenner-Gren Foundations, the Foundation Blanceflor Boncompagni Ludovisi, née Bildt and Tore Nilsson Foundation for Medical Science. V.S. was supported by the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement 895390. A.V. was supported by the Hitachi-Nomura postdoctoral fellowship awarded through the Department of Biological Chemistry at the University of California, Irvine. The work of S.C., M.S., and P.B. was in part supported by NIH grant GM123558. C.J. was supported by the AASLD Foundation Pinnacle Research Award in Liver Disease, the Edward Mallinckrodt, Jr. Foundation Award, and NIH/NIAAA R01 AA029124. P.-S.W. is supported by grant RYC2019026661-I funded by MCIN/AEI/10.13039/501100011033 and by “ESF Investing in your future.” Financial support for the S.M. laboratory is provided through the NIH/NCI (grants R01CA244519, R01CA259370, and K22CA212045). Research in the S.A.B. lab is supported partially by the European Research Council (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement 787041), the Government of Cataluña (SGR grant), the Government of Spain (MINECO), the La Marató/TV3 Foundation, the Foundation Lilliane Bettencourt, the Spanish Association for Cancer Research (AECC), and the Worldwide Cancer Research Foundation (WCRF). The IRB Barcelona is a Severo Ochoa Center of Excellence (MINECO award SEV-2015-0505). P.M.-C. acknowledges funding from MICINN-RTI2018-096068, ERC-2016-AdG-741966, LaCaixa-HEALTH-HR17-00040, MDA, UPGRADE-H2020-825825, AFM, DPP-Spain, Fundació La MaratóTV3- 80/19-202021, MWRF, and María-de-Maeztu Program for Units of Excellence to UPF (MDM-2014-0370), and the Severo-Ochoa Program for Centers of Excellence to CNIC (SEV2015-0505). Work in the P.S.-C. laboratory is supported by NIH grants R21DK114652 and R21AG053592, a Challenge Grant from the Novo Nordisk Foundation (NNF202585), KAUST funding (OSR-2019-CRG8-URF/1/4042), and via access to the Genomics High Throughput Facility Shared Resource of the Cancer Center Support Grant (CA-62203) and the UCI and NIH-shared instrumentation grants 1S10RR025496-01, 1S10OD010794-01, and 1S10OD021718-01.

### Author contributions

Conceptualization, J.G.S., K.B.K., P.S.-C., V.O., P.L., P.B., P.-S.W., T.M., C.J., S.M., S.A.B., and P.M.-C.; formal analysis, S.C., M.S., E.D., L.M., O.D., C.J., J.G.S., and K.B.K.; funding acquisition, P.S.-C., V.O., S.M., S.A.B., and P.M.-C.; investigation, J.G.S., K.B.K., T.S., C.M.G., P.P., A.V., S.C., M.S., E.D., L.M., and V.S.; methodology, P.-S.W., S.M., J.G.S., and K.B.K.; project administration, S.M.; resources, P.-S.W., S.M., S.A.B., and P.M.-C.; supervision, S.M., S.A.B., P.M.-C., P.S.-C., P.B., V.O., B.L.P., and C.J.; visualization, J.G.S., K.B.K., S.C., M.S., E.D., and P.B.; writing – original draft, J.G.S., K.B.K., S.M., S.A.B., and P.M.-C.; writing – review & editing, J.G.S., K.B.K., R.B., J.M., A.V., P.L., T.M., A.K., M.V.-D., B.L.P., P.B., S.M., S.A.B., and P.M.-C.

### Declaration of interests

S.A.B. is a co-founder and scientific advisor of ONA Therapeutics.

### Inclusion and diversity

We support inclusive, diverse, and equitable conduct of research.

## Supplemental information (6)

[PDF (8.52 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/1a5365c3-1f6b-4bfd-b621-35a01f4f5589/mmc1.pdf)

[Spreadsheet (54.32 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/cb3f511d-2738-4568-8017-8473f5db0618/mmc2.xlsx)

[Spreadsheet (52.73 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/5eaa2406-0699-4c4e-bfdc-409f1d131fc2/mmc3.xlsx)

[Spreadsheet (6.08 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/e2c00ab0-c9db-4229-ae3d-60e7133963fe/mmc4.xlsx)

[Spreadsheet (6.12 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/377c1ba5-4694-4c75-b3e7-07119912dc28/mmc5.xlsx)

[PDF (15.09 MB)](https://www.cell.com/cms/10.1016/j.celrep.2023.112588/attachment/e2bcab7d-f9fd-4521-bce8-85be03167d06/mmc6.pdf)

## Data and code availability

- RNA-sequencing data have been deposited at GEO and are publicly available as of the date of publication. Accession numbers are listed in the [key resources table](https://www.cell.com/cell-reports/fulltext/#sec-4-1).
- This paper does not report original code.
- Any additional information required to reanalyze the data reported in this paper is available from the [lead contact](https://www.cell.com/cell-reports/fulltext/#sec-4-2-1) upon request.