---
title: "Resistant starch intake facilitates weight loss in humans by reshaping the gut microbiota - PMC"
source: "https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/"
author:
  - "[[Google Scholar]]"
published:
created: 2026-01-07
description: "Emerging evidence suggests that modulation of gut microbiota by dietary fibre may offer solutions for metabolic disorders. In a randomized placebo-controlled crossover design trial (ChiCTR-TTRCC-13003333) in 37 participants with overweight or ..."
tags:
  - "clippings"
---
![Nature Portfolio logo](https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npgopen.png) 

. 2024 Feb 26;6(3):578–597. doi: [10.1038/s42255-024-00988-y](https://doi.org/10.1038/s42255-024-00988-y)

## Resistant starch intake facilitates weight loss in humans by reshaping the gut microbiota

[Huating Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D) <sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉,</sup><sup>#</sup>, [Lei Zhang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D) <sup>1,</sup><sup>4,</sup><sup>#</sup>, [Jun Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D) <sup>5,</sup><sup>6,</sup><sup>#</sup>, [Qian Wu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Q%22%5BAuthor%5D) <sup>1,</sup><sup>4,</sup><sup>#</sup>, [Lingling Qian](https://pubmed.ncbi.nlm.nih.gov/?term=%22Qian%20L%22%5BAuthor%5D) <sup>1,</sup><sup>4</sup>, [Junsheng He](https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20J%22%5BAuthor%5D) <sup>5</sup>, [Yueqiong Ni](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20Y%22%5BAuthor%5D) <sup>1,</sup><sup>6,</sup><sup>7</sup>, [Petia Kovatcheva-Datchary](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovatcheva-Datchary%20P%22%5BAuthor%5D) <sup>8</sup>, [Rui Yuan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuan%20R%22%5BAuthor%5D) <sup>9</sup>, [Shuangbo Liu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20S%22%5BAuthor%5D) <sup>9</sup>, [Li Shen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shen%20L%22%5BAuthor%5D) <sup>10</sup>, [Mingliang Zhang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D) <sup>1</sup>, [Bin Sheng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheng%20B%22%5BAuthor%5D) <sup>11</sup>, [Ping Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20P%22%5BAuthor%5D) <sup>12,</sup><sup>13</sup>, [Kang Kang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20K%22%5BAuthor%5D) <sup>6</sup>, [Liang Wu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20L%22%5BAuthor%5D) <sup>1,</sup><sup>4</sup>, [Qichen Fang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20Q%22%5BAuthor%5D) <sup>1</sup>, [Xiaoxue Long](https://pubmed.ncbi.nlm.nih.gov/?term=%22Long%20X%22%5BAuthor%5D) <sup>1,</sup><sup>4</sup>, [Xiaolin Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D) <sup>14</sup>, [Yanli Li](https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D) <sup>14</sup>, [Yaorui Ye](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20Y%22%5BAuthor%5D) <sup>14</sup>, [Jianping Ye](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20J%22%5BAuthor%5D) <sup>1,</sup><sup>15</sup>, [Yuqian Bao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bao%20Y%22%5BAuthor%5D) <sup>1</sup>, [Yueliang Zhao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Y%22%5BAuthor%5D) <sup>9,</sup><sup>16</sup>, [Guowang Xu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20G%22%5BAuthor%5D) <sup>14</sup>, [Xinyu Liu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20X%22%5BAuthor%5D) <sup>14,</sup><sup>✉</sup>, [Gianni Panagiotou](https://pubmed.ncbi.nlm.nih.gov/?term=%22Panagiotou%20G%22%5BAuthor%5D) <sup>6,</sup><sup>7,</sup><sup>17,</sup><sup>18,</sup><sup>✉</sup>, [Aimin Xu](https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20A%22%5BAuthor%5D) <sup>2,</sup><sup>3,</sup><sup>✉</sup>, [Weiping Jia](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20W%22%5BAuthor%5D) <sup>1,</sup><sup>✉</sup>

PMCID: PMC10963277 PMID: [38409604](https://pubmed.ncbi.nlm.nih.gov/38409604/)

## Abstract

Emerging evidence suggests that modulation of gut microbiota by dietary fibre may offer solutions for metabolic disorders. In a randomized placebo-controlled crossover design trial (ChiCTR-TTRCC-13003333) in 37 participants with overweight or obesity, we test whether resistant starch (RS) as a dietary supplement influences obesity-related outcomes. Here, we show that RS supplementation for 8 weeks can help to achieve weight loss (mean −2.8 kg) and improve insulin resistance in individuals with excess body weight. The benefits of RS are associated with changes in gut microbiota composition. Supplementation with *Bifidobacterium* *adolescentis*, a species that is markedly associated with the alleviation of obesity in the study participants, protects male mice from diet-induced obesity. Mechanistically, the RS-induced changes in the gut microbiota alter the bile acid profile, reduce inflammation by restoring the intestinal barrier and inhibit lipid absorption. We demonstrate that RS can facilitate weight loss at least partially through *B.**adolescentis* and that the gut microbiota is essential for the action of RS.

---

In a randomized placebo-controlled trial in 37 individuals with excess body weight, dietary supplementation with resistant starch lowers body weight and induces changes in gut microbiota composition. Mechanistic analysis in male mice shows that resistant starch at least partially facilitates weight loss through the action of *Bifidobacterium* *adolescentis*.

## Main

Owing to the current global obesity epidemic, research is directed towards new obesity prevention and weight-reduction strategies. This is crucial because obesity significantly contributes to comorbidities such as diabetes and cardiovascular diseases—major global mortality causes. Conversely, weight loss mitigates these comorbidities, underscoring weight management’s importance in preventing and treating these conditions <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR1">1</a></sup>.

The gut microbiota has been increasingly recognized as an important regulator of host physiology and pathophysiology <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR2">2</a></sup>. Specifically, previous studies have reported that gut microbiota regulates inflammation, fat storage and glucose metabolism <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR3">3</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR4">4</a></sup>. Although the results of faecal microbial transplantation (FMT) from healthy donors to individuals with obesity were inconsistent or short-term <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR5">5</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR6">6</a></sup>, combining dietary intervention with FMT could result in favourable alterations in the recipients’ microbiota and improvements in clinical outcomes <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. Thus, the rational manipulation of the gut microbiome by dietary interventions might be a promising anti-obesity strategy <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR2">2</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>.

Prebiotics, including polysaccharides, oligosaccharides and other fermentable dietary fibres, increase the amount of beneficial gut microbiota, notably certain *Bifidobacterium* and *Lactobacillus*  spp. These bacteria diminish pathogen populations, fortify the gut barrier and mitigate the inflammatory response <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR8">8</a></sup>. Moreover, snacks formulated with different fibre preparations could be tailored to alter functions associated with specific elements of the microbiome <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR9">9</a></sup>; however, most prebiotic studies were based on correlations without establishing a causal link between the modulation of the gut microbiota and the observed beneficial effects on metabolism <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR2">2</a></sup>. Insights from both human trials and mechanistic studies in gnotobiotic animals are crucial to establish the causality between microbiome alterations and host biological responses. Furthermore, these studies are vital to comprehend the mechanisms connecting microbiome changes to the physiological advantages of prebiotics or other fermentable dietary fibres <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR2">2</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR9">9</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR10">10</a></sup>.

RS refers to a kind of fermentable dietary fibre that cannot be digested by human amylases in the small intestine and moves into the colon, where it undergoes fermentation by gut microbiota <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR11">11</a></sup>. Studies in rodents have demonstrated RS could lead to a decrease in total body fat, particularly visceral fat, as opposed to digestible starch feeding <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR12">12</a></sup>. Diets low in protein and high in carbohydrate yield the most favourable metabolic outcomes when the carbohydrate component consists of RS in mice <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR13">13</a></sup>; however, human data showed that there was no impact on the total body weight of individuals with metabolic syndrome after being fed RS for a duration spanning 4 to 12 weeks <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR14">14</a> – <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR16">16</a></sup>. Low-fat diets supplemented with RS had beneficial effects on the hosts, but high-fat diets attenuated the RS fermentation and the beneficial effects <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR17">17</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR18">18</a></sup>. This may be one possible explanation as to why RS seemingly had no impact on body weight in the clinical trials described above, as those clinical trials did not have a high compliance rate to the diet <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR12">12</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR19">19</a></sup>. Moreover, this implies RS-associated gut microbiota’s vital role in RS’s therapeutic effects; however, RS’s potential as a functional, adaptable food ingredient for obesity treatment in humans and the modulation of metabolic benefits by RS-related gut microbiome alterations remain unclear. Thus, a robust trial in obese individuals is essential to substantiate claims about RS’s impact on diverse physiological aspects in consumers and the required dosage <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR9">9</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR19">19</a></sup>. Moreover, multi-omics approaches and gnotobiotic animal models should be used to systematically and mechanistically connect the influence of RS on the gut microbial community and the host’s metabolism <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR12">12</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR19">19</a></sup>.

Here we performed a crossover, randomized clinical trial in individuals with excess body weight to investigate the effect of RS as a dietary supplement on obesity and other metabolic phenotypes. The trial is a feeding study providing isoenergetic and balanced background diets. Metagenomics and metabolomics analyses were conducted to assess RS’s impact on gut microbiota composition and function. Additionally, we analysed the influence of RS-modified gut microbiota transferred from selected human donors to antibiotic-treated mice on host adiposity and glucose metabolism, we also explored the mechanisms underlying the metabolic advantages conferred by gut microbiota through RS.

## Results

### RS intervention facilitates weight loss

The investigation was a placebo-controlled, double-blinded, crossover design intervention (ChiCTR-TTRCC-13003333), involving 37 participants (average age of 33.43 ± 7.71 years). The participants had a body mass index (BMI) ≥ 24 kg m <sup>−2</sup> and/or increased waist circumference (≥85 cm for men and ≥80 cm for women). None had chronic disorders, ongoing treatments impacting glucose metabolism or recent use of antibiotics or probiotic (within 3 weeks). The 20-week study duration included two intervention periods of 8 weeks, with one each for high-amylose maize (HAM-RS2) (RS, 2.8 kcal g <sup>−1</sup>, 91.2 g, containing 40 g RS) and control starch (CS) (AMIOCA) (3.55 kcal g <sup>−1</sup>, 72 g, amylopectin, containing 0 g RS, with equal energy supply) and a 4-week washout period between the interventions (Fig. [1a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). Study participants were randomly allocated into two groups: (1) RS-Washout-CS or (2) CS-Washout-RS. Starch was provided as powder in pre-packaged sachets to be mixed with 300 ml water. Each participants consumed one sachet twice a day, 10–15 min before meals. Throughout the trial, encompassing the run-in, two interventions and the intervening washout period, we provided an isoenergetic and balanced background diet (three meals per day), according to the Chinese and American guidelines for prevention and management of adults with overweight and obesity (details in the study protocol of [Supplementary Information](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM1)) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR20">20</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR21">21</a></sup>. Participants showed no difference in skipping meals or starch consumption between RS and CS interventions, suggesting that there was no difference in dietary compliance between the RS and CS intervention. Except for differences in dietary fibre intake (RS versus CS, 53.84 ± 4.70 g versus 9.97 ± 5.17 g, *P*  < 0.001), total energy consumed and percentage of macronutrients were similar during the RS or CS intervention periods (Supplementary Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)). In total, 37 participants, including 22 male and 15 female participants finished the study and were incorporated into the analyses (Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Tab1) and Extended Data Fig. [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig8)). The study was conducted in Shanghai, China from 3 July 2013 to 14 October 2016 and reported no gastrointestinal side effects, such as nausea, vomiting, bloating, increased bowel movement or change of stool frequency.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/6ae0c9a0c88c/42255_2024_988_Fig1_HTML.jpg)

a, Diagram of the clinical trial. After enrolment, randomization and run-in period, participants consumed either RS or CS alternately and separated by a washout period. During the whole trial, all participants were provided with identical diets. The assessments at each visit are displayed in the diagram. b–d, RS intervention significantly reduced body weight ( b ), fat mass ( c ) and waist circumference ( d ). e, f, Change of VFA and SFA evaluated by MRI. g, Representative abdominal MRI of participants before (left) and after (right) the 8-week RS intervention. Raw (top) and marked (bottom) MRI at navel level. Yellow represents SFA and red represents VFA. h, Change of GIR evaluated by hyperinsulinemic–euglycemic clamp. i, Change of serum TNFα levels. j, Change of serum IL-1β levels. k, Daily faecal lipid excretion, including NEFA, TG and TC after the 8-week interventions with RS or CS. l, Change of serum ANGPTL4 levels. m, Change of serum FGF21 levels. n = 37 individuals ( – ), = 36 individuals ( = 35 individuals ( ) and = 17 individuals ( ) for either RS or CS. Analysis of covariance (ANCOVA) adjusted by baseline value was used for comparison between RS and CS at each visit ( ). Data are shown as mean (95% confidence interval (CI)). \*\*\* P < 0.001. Data are shown as median with IQR ( ). Nonparametric Wilcoxon rank-sum test was used to evaluate the significance between the two interventions. \*\*\* < 0.001. Data are shown as box-and-whisker plots ( ). Box plot, median and quartiles; whiskers, data range. \* = 0.025, 0.014 and 0.046 ( ), \*\* = 0.004 and 0.002 ( ), \*\*\* < 0.001 for the between-group difference assessed by the linear mixed model adjusted for intervention order. †† = 0.003 and 0.002 ( ††† < 0.001 for the within-group change by mixed linear model adjusted for intervention followed by Bonferroni’s test.

#### Table 1.

Summary of clinical parameters of the study participants before and after CS or RS intervention

<table><thead><tr><th colspan="1" rowspan="1">Clinical parameters</th><th colspan="1" rowspan="1">CS 0 W</th><th colspan="1" rowspan="1">CS 8 W</th><th colspan="1" rowspan="1">RS 0 W</th><th colspan="1" rowspan="1">RS 8 W</th><th colspan="1" rowspan="1"><em>P</em> value <sup>a</sup></th><th colspan="1" rowspan="1"><em>P</em> value <sup>b</sup></th><th colspan="1" rowspan="1"><em>P</em> value <sup>c</sup></th><th colspan="1" rowspan="1"><em>P</em> value <sup>d</sup></th></tr></thead><tbody><tr><td colspan="9" rowspan="1"><strong>Obesity-related indicators</strong></td></tr><tr><td colspan="1" rowspan="1">Body weight (kg)</td><td colspan="1" rowspan="1">82.74 ± 15.67</td><td colspan="1" rowspan="1">83.23 ± 16.01</td><td colspan="1" rowspan="1">83.24 ± 15.70</td><td colspan="1" rowspan="1">80.91 ± 15.39</td><td colspan="1" rowspan="1">0.328</td><td colspan="1" rowspan="1">0.339</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">BMI (kg m <sup>−2</sup>)</td><td colspan="1" rowspan="1">28.58 ± 3.79</td><td colspan="1" rowspan="1">28.74 ± 3.95</td><td colspan="1" rowspan="1">28.75 ± 3.79</td><td colspan="1" rowspan="1">27.93 ± 3.75</td><td colspan="1" rowspan="1">0.361</td><td colspan="1" rowspan="1">0.422</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">FM (kg)</td><td colspan="1" rowspan="1">26.21 ± 8.37</td><td colspan="1" rowspan="1">26.71 ± 8.45</td><td colspan="1" rowspan="1">26.59 ± 8.34</td><td colspan="1" rowspan="1">24.83 ± 8.29</td><td colspan="1" rowspan="1">0.486</td><td colspan="1" rowspan="1">0.136</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">FFM (kg)</td><td colspan="1" rowspan="1">56.54 ± 12.33</td><td colspan="1" rowspan="1">56.57 ± 12.46</td><td colspan="1" rowspan="1">56.66 ± 12.35</td><td colspan="1" rowspan="1">55.75 ± 11.99</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.011</td><td colspan="1" rowspan="1">0.029</td></tr><tr><td colspan="1" rowspan="1">TBW (kg)</td><td colspan="1" rowspan="1">39.55 ± 7.71</td><td colspan="1" rowspan="1">39.51 ± 7.82</td><td colspan="1" rowspan="1">39.61 ± 7.69</td><td colspan="1" rowspan="1">39.10 ± 7.68</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.101</td><td colspan="1" rowspan="1">0.155</td></tr><tr><td colspan="1" rowspan="1">Fat percentage</td><td colspan="1" rowspan="1">31.70 ± 7.72</td><td colspan="1" rowspan="1">31.80 ± 8.03</td><td colspan="1" rowspan="1">31.99 ± 7.64</td><td colspan="1" rowspan="1">31.06 ± 7.93</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.101</td><td colspan="1" rowspan="1">0.117</td></tr><tr><td colspan="1" rowspan="1">Waist circumference (cm)</td><td colspan="1" rowspan="1">95.52 ± 10.91</td><td colspan="1" rowspan="1">95.59 ± 11.16</td><td colspan="1" rowspan="1">96.28 ± 11.00</td><td colspan="1" rowspan="1">92.60 ± 10.89</td><td colspan="1" rowspan="1">0.731</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">Hip circumference (cm)</td><td colspan="1" rowspan="1">105.00 ± 8.05</td><td colspan="1" rowspan="1">105.49 ± 7.71</td><td colspan="1" rowspan="1">105.31 ± 8.08</td><td colspan="1" rowspan="1">104.16 ± 7.69</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.372</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">Waist-to-hip ratio</td><td colspan="1" rowspan="1">0.91 ± 0.06</td><td colspan="1" rowspan="1">0.91 ± 0.07</td><td colspan="1" rowspan="1">0.91 ± 0.06</td><td colspan="1" rowspan="1">0.89 ± 0.06</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">0.01</td></tr><tr><td colspan="1" rowspan="1">VFA (cm²)</td><td colspan="1" rowspan="1">104.75 ± 53.46</td><td colspan="1" rowspan="1">108.31 ± 49.71</td><td colspan="1" rowspan="1">108.58 ± 55.06</td><td colspan="1" rowspan="1">86.39 ± 45.41</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">&lt;0.001</td></tr><tr><td colspan="1" rowspan="1">SFA (cm²)</td><td colspan="1" rowspan="1">283.63 ± 92.87</td><td colspan="1" rowspan="1">286.68 ± 96.69</td><td colspan="1" rowspan="1">289.64 ± 95.38</td><td colspan="1" rowspan="1">262.50 ± 89.91</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">0.004</td></tr><tr><td colspan="9" rowspan="1"><strong>Anthropometric parameters</strong></td></tr><tr><td colspan="1" rowspan="1">Age (years)</td><td colspan="4" rowspan="1">33.43 ± 7.71</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">Sex (male:female)</td><td colspan="4" rowspan="1">22:15</td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td><td colspan="1" rowspan="1"></td></tr><tr><td colspan="1" rowspan="1">SBP (mm Hg)</td><td colspan="1" rowspan="1">112.65 ± 10.00</td><td colspan="1" rowspan="1">113.51 ± 8.70</td><td colspan="1" rowspan="1">112.41 ± 11.71</td><td colspan="1" rowspan="1">113.05 ± 11.19</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.918</td></tr><tr><td colspan="1" rowspan="1">DBP (mm Hg)</td><td colspan="1" rowspan="1">76.14 ± 7.08</td><td colspan="1" rowspan="1">77.11 ± 6.88</td><td colspan="1" rowspan="1">75.08 ± 7.69</td><td colspan="1" rowspan="1">74.46 ± 7.54</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.342</td></tr><tr><td colspan="9" rowspan="1"><strong>Liver enzymes and renal function</strong></td></tr><tr><td colspan="1" rowspan="1">ALT (U l <sup>−1</sup>)</td><td colspan="1" rowspan="1">16.80 (12.30, 29.00)</td><td colspan="1" rowspan="1">20.00 (10.28, 30.25)</td><td colspan="1" rowspan="1">16.00 (12.00, 30.00)</td><td colspan="1" rowspan="1">15.00 (12.00, 22.50)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.432</td><td colspan="1" rowspan="1">0.041</td></tr><tr><td colspan="1" rowspan="1">AST (U l <sup>−1</sup>)</td><td colspan="1" rowspan="1">18.60 (17.20, 22.00)</td><td colspan="1" rowspan="1">19.76 (16.03, 24.80)</td><td colspan="1" rowspan="1">19.00 (17.00, 22.50)</td><td colspan="1" rowspan="1">17.00 (14.50, 21.00)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.058</td><td colspan="1" rowspan="1">0.022</td></tr><tr><td colspan="1" rowspan="1">GGT (U l <sup>−1</sup>)</td><td colspan="1" rowspan="1">24.80 (17.50, 35.00)</td><td colspan="1" rowspan="1">25.20 (18.17, 34.26)</td><td colspan="1" rowspan="1">25.00 (18.00, 36.00)</td><td colspan="1" rowspan="1">21.00 (16.00, 35.50)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.033</td><td colspan="1" rowspan="1">0.03</td></tr><tr><td colspan="1" rowspan="1">BUN (mmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">4.40 ± 0.97</td><td colspan="1" rowspan="1">4.54 ± 1.17</td><td colspan="1" rowspan="1">4.38 ± 0.99</td><td colspan="1" rowspan="1">4.23 ± 1.06</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.149</td></tr><tr><td colspan="1" rowspan="1">Cr (μmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">70.52 ± 12.94</td><td colspan="1" rowspan="1">70.26 ± 16.11</td><td colspan="1" rowspan="1">70.62 ± 13.02</td><td colspan="1" rowspan="1">69.95 ± 14.00</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.809</td></tr><tr><td colspan="9" rowspan="1"><strong>Lipid profiles</strong></td></tr><tr><td colspan="1" rowspan="1">TC (mmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">4.65 ± 0.67</td><td colspan="1" rowspan="1">4.56 ± 0.80</td><td colspan="1" rowspan="1">4.69 ± 0.67</td><td colspan="1" rowspan="1">4.40 ± 0.71</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.008</td><td colspan="1" rowspan="1">0.146</td></tr><tr><td colspan="1" rowspan="1">TG (mmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">1.09 (0.86, 1.68)</td><td colspan="1" rowspan="1">1.23 (0.80, 1.52)</td><td colspan="1" rowspan="1">1.04 (0.85, 1.64)</td><td colspan="1" rowspan="1">1.01 (0.79, 1.38)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.559</td></tr><tr><td colspan="1" rowspan="1">HDL-C (mmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">1.14 ± 0.24</td><td colspan="1" rowspan="1">1.14 ± 0.23</td><td colspan="1" rowspan="1">1.13 ± 0.25</td><td colspan="1" rowspan="1">1.10 ± 0.20</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.456</td></tr><tr><td colspan="1" rowspan="1">LDL-C (mmol l <sup>−1</sup>)</td><td colspan="1" rowspan="1">3.01 ± 0.58</td><td colspan="1" rowspan="1">2.95 ± 0.67</td><td colspan="1" rowspan="1">3.02 ± 0.60</td><td colspan="1" rowspan="1">2.89 ± 0.54</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.424</td><td colspan="1" rowspan="1">0.482</td></tr><tr><td colspan="9" rowspan="1"><strong>Secretory cytokines</strong></td></tr><tr><td colspan="1" rowspan="1">A-FABP (ng ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">35.49 (30.71, 44.82)</td><td colspan="1" rowspan="1">36.83 (23.35, 46.99)</td><td colspan="1" rowspan="1">37.47 (29.33, 47.81)</td><td colspan="1" rowspan="1">31.12 (23.74, 45.12)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.072</td><td colspan="1" rowspan="1">0.12</td></tr><tr><td colspan="1" rowspan="1">Adiponectin (μg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">9.34 (7.49, 13.86)</td><td colspan="1" rowspan="1">9.38 (7.01, 12.39)</td><td colspan="1" rowspan="1">9.21 (7.04, 13.13)</td><td colspan="1" rowspan="1">10.46 (7.38, 13.68)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.376</td><td colspan="1" rowspan="1">0.038</td></tr><tr><td colspan="1" rowspan="1">FGF21 (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">167.61 (99.71, 211.00)</td><td colspan="1" rowspan="1">172.07 (104.63, 237.70)</td><td colspan="1" rowspan="1">188.46 (113.19, 224.07)</td><td colspan="1" rowspan="1">151.74 (51.62, 216.98)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">0.002</td></tr><tr><td colspan="9" rowspan="1"><strong>Inflammation-related factors</strong></td></tr><tr><td colspan="1" rowspan="1">TNFα (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">0.70 (0.60, 0.81)</td><td colspan="1" rowspan="1">0.68 (0.55, 0.86)</td><td colspan="1" rowspan="1">0.70 (0.55, 0.82)</td><td colspan="1" rowspan="1">0.66 (0.45, 0.72)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.053</td><td colspan="1" rowspan="1">0.014</td></tr><tr><td colspan="1" rowspan="1">IL-1β (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">0.18 (0.13, 0.24)</td><td colspan="1" rowspan="1">0.17 (0.10, 0.27)</td><td colspan="1" rowspan="1">0.16 (0.11, 0.26)</td><td colspan="1" rowspan="1">0.11 (0.06, 0.17)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">&lt;0.001</td><td colspan="1" rowspan="1">0.046</td></tr><tr><td colspan="1" rowspan="1">MCP-1 (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">673.56 (535.55, 865.83)</td><td colspan="1" rowspan="1">630.62 (516.20, 768.22)</td><td colspan="1" rowspan="1">626.85 (513.23, 816.09)</td><td colspan="1" rowspan="1">631.97 (498.41, 822.20)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.925</td></tr><tr><td colspan="1" rowspan="1">IL-10 (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">1.72 (1.58, 2.07)</td><td colspan="1" rowspan="1">1.62 (1.49, 1.79)</td><td colspan="1" rowspan="1">1.66 (1.52, 1.95)</td><td colspan="1" rowspan="1">1.58 (1.48, 1.91)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.146</td></tr><tr><td colspan="1" rowspan="1">IL-6 (pg ml <sup>−1</sup>)</td><td colspan="1" rowspan="1">1.08 (0.65,1.85)</td><td colspan="1" rowspan="1">1.12 (0.57,1.64)</td><td colspan="1" rowspan="1">1.08 (0.77,1.58)</td><td colspan="1" rowspan="1">1.05 (0.71,1.49)</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.999</td><td colspan="1" rowspan="1">0.689</td></tr></tbody></table>

[Open in a new tab](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/table/Tab1/)

FM, fat mass; FFM, free fat mass; TBW, total body water; SBP, systolic blood pressure; DBP, diastolic blood pressure; ALT, alanine transaminase; AST, aspartate transaminase; GGT, γ-glutamyl transferase; HDL-C, high-density lipoprotein cholesterol; LDL-C, low-density lipoprotein cholesterol; BUN, blood urea nitrogen; Cr, creatinine; A-FABP; adipocyte fatty acid-binding protein. Data are expressed as mean ± s.d. or median (interquartile range (IQR)).

<sup>a</sup> Differences in baseline variables before the RS and CS interventions were assessed using a linear mixed model adjusted by intervention order followed by Bonferroni’s test.

<sup>b</sup> Differences in outcomes before and after CS intervention were assessed using a linear mixed model adjusted by intervention order followed by Bonferroni’s test.

<sup>c</sup> Differences in outcomes before and after RS intervention were assessed using a linear mixed model adjusted by intervention order followed by Bonferroni’s test.

<sup>d</sup> Differences in outcomes between the RS and CS intervention were assessed using a linear mixed model adjusted by intervention order.

![Extended Data Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/9b45c629e0e6/42255_2024_988_Fig8_ESM.jpg)

CONSORT flow diagram illustrating the procedures for enrolment, intervention allocation, follow-up and data analysis of the study. RS, resistant starch; CS, control starch.

We compared the actual changes in anthropometric parameters and biochemical indices between the interventions (RS versus CS) and the within-group change before and after the intervention using the linear mixed model adjusted by intervention order (Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Tab1) and Supplementary Table [2](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)). The primary outcome body weight was significantly decreased after the RS intervention and the net absolute change after RS intervention relative to CS intervention was −2.81 kg (95% CI −3.55 kg to −2.07 kg; *P*  < 0.001), whereas no significant change was observed after the CS intervention (Fig. [1b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). Moreover, fat mass and waist circumference reduced significantly after the RS intervention compared with the CS intervention (Fig. [1c,d](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). During the RS intervention period, the body weight, waist circumference and fat mass of participants significantly decreased from week 2 onwards. Both visceral fat areas (VFA) and subcutaneous fat areas (SFA), measured by abdominal magnetic resonance imaging (MRI) were lower following the RS consumption compared with those following CS consumption (*P*  < 0.001 and *P*  = 0.004, respectively; Fig. [1e–g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). Regardless of whether it was in the RS-Washout-CS group or the CS-Washout-RS group, significant reductions were observed in body weight and other obesity-related outcomes after the RS intervention (all *P*  < 0.05). In the RS-Washout-CS group, obesity-related outcomes showed a reversion to baseline levels after the washout period (Supplementary Table [3](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3) and Extended Data Fig. [2a–e](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig9)). Moreover, the two-way analysis of variance (ANOVA) showed significant difference in body weight and obesity-related outcomes (fat mass, waist circumference and VFA) on intervention (*P*  < 0.001) and no significant order effect or intervention-order interaction. These results demonstrated that an 8-week RS intervention reduced abdominal adiposity in individuals with excess body weight.

![Extended Data Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/51ed8e4921d5/42255_2024_988_Fig9_ESM.jpg)

( a-e ) Boxplots (with median) showing the dynamic changes of ( a ) body weight, ( b ) fat mass, and ( c ) waist circumference at every 2 weeks of intervention, as well as ( d ) visceral fat (VFA) and ( e ) subcutaneous fat area (SFA) at every 4 weeks of intervention. Lines connect the means of outcomes at different time pointes. The horizontal line represents the outcomes of the linear mixed model analysis followed by Bonferroni’s test conducted to compare the connected endpoints. RS-Washout-CS group, n = 19 individuals. CS-Washout-RS group, n = 18 individuals. \*\*\* P < 0.001. ( f-h ) Standardized meal tolerance tests (MTT) were performed at baseline (0 week) and after 8-week RS or CS intervention. ( f ) Blood glucose. ( g ) Change of area under curve (AUC) and analysis of blood glucose. ( h ) Serum insulin. ( i ) Change of serum adiponectin level evaluated by ELISA in subjects before and after 8-week RS or CS intervention. For -, n = 36 individuals per group, and for, n = 37 individuals per group. Data are expressed as mean ± s.e.m (, ). Data are shown as box-and-whisker plots ( ). The horizontal line in the middle of each box indicates the median value, the top and bottom borders of the boxes represent the 75th and 25th percentiles, and the whiskers denote the lowest and highest values. \* = 0.023 and 0.020 ( ), 0.016 ( ), 0.036 ( ), and 0.038 ( ), for the difference between RS and CS intervention by linear mixed model adjusted by intervention order and † = 0.022, †† = 0.001 ( ) and 0.001 ( ), ††† < 0.001 ( ) for the difference before and after RS intervention (RS 0W vs. RS 8W) by linear mixed model adjusted by intervention order followed by Bonferroni’s test.

Furthermore, glucose tolerance improved significantly after the RS intervention (Extended Data Fig. [2f,g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig9)). The insulin concentrations at 120 min following a meal tolerance test (MTT) in participants after the RS intervention were significantly lower than those after the CS intervention (Extended Data Fig. [2h](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig9)); however, the CS intervention did not induce any differences in glucose and insulin levels compared with baseline values (Extended Data Fig. [2f,h](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig9)). We further assessed insulin sensitivity by hyperinsulinemic–euglycemic clamp and found that the glucose infusion rate (GIR) was significantly increased after the RS intervention (with a median increase of 1.05 mg kg <sup>−1</sup>  min <sup>−1</sup>, 0.15 to 2.10) compared to the CS intervention (*P*  = 0.025) (Fig. [1h](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)), demonstrating a significant improvement in insulin sensitivity. Furthermore, a significant increase in serum adiponectin levels was observed after the RS intervention (Extended Data Fig. [2i](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig9)). In addition, after RS intervention, neither the first-phase nor the second-phase insulin secretion was significantly different from those after CS intervention (Supplementary Table [4](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)). All these results indicated that the 8-week RS intervention improved glucose tolerance and insulin sensitivity in individuals with excess body weight.

To investigate the potential mechanism by which RS facilitates weight loss, we determined the changes in chronic, low-grade inflammatory response and intestinal lipid digestion, which are closely related to obesity <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR22">22</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR23">23</a></sup>. Levels of pro-inflammatory cytokines such as serum tumour necrosis factor (TNF)α and interleukin (IL)-1β were found significantly lower in the study participants after RS consumption compared with CS consumption (*P*  = 0.014 and *P*  = 0.046, respectively), although no significant difference in monocyte chemoattractant protein-1 (MCP-1), IL-10 and IL-6 were observed during RS and CS consumption (Fig. [1i,j](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1) and Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Tab1)). Furthermore, we measured faecal lipids of the study participants and found that the daily excretion of faecal non-esterified fatty acid (NEFA), triglycerides (TGs) and total cholesterol (TC) were significantly higher following RS consumption compared with CS consumption (Fig. [1k](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). As there was no significant difference in fat intake during the RS and CS consumption (Supplementary Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)), these results suggested that the RS intervention may decrease lipid absorption from the diet. Circulating levels of angiopoietin-like 4 (ANGPTL4), a potential connection between gut and lipid metabolism <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR3">3</a></sup>, were significantly increased in the study participants after the RS consumption compared with CS consumption (Fig. [1l](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). Serum fibroblast growth factor 21 (FGF21), which was reported to increase in obese status, significantly reduced after RS consumption (Fig. [1m](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR24">24</a></sup>.

### RS intervention reshapes gut microbiota

To investigate the dynamics of the gut microbiota during the interventions, a shotgun metagenomic sequencing was conducted. The average throughput for each sample was 5.74 (s.d. 0.87) Gbp. Species-level taxonomic profiles based on the top 50 species by mean abundance across samples revealed that *Prevotella* *copri*, *Bacteroides* *stercoris* and *Faecalibacterium* *prausnitzii*, were the most prevalent across samples. The other species were present at a much lower abundance (on average 1.1% per species) for the majority of the samples (Extended Data Fig. [3a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig10)). To evaluate the extent to which the composition of the gut microbiome was altered in response to the RS intervention, we calculated the Bray–Curtis distance based on the profile of species variation (as fold change in abundance after the 8-week intervention) between samples using the following procedures. The profile of species variation was composed of log <sub>2</sub> fold change (log <sub>2</sub> FC) in abundance after treatment for each species. Fold change values were normalized ranging from 0 to 1 with the sum as one in each sample. We further calculated the Bray–Curtis distance based on this profile of species variation. As shown in the non-metric multidimensional scaling (NMDS) plot based on the Bray–Curtis distance, RS and CS samples were separated to a certain extent, indicating significantly different variation patterns in the microbiota after the RS and CS interventions, respectively (*P*  < 0.001 with the Adonis test from VEGAN) (Fig. [2a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2)). This finding thus suggests that the RS intervention influences the dynamics and restructures the composition of the gut microbiota. There was no significant difference in microbial composition between the baseline of CS and RS intervention (Extended Data Fig. [3b–e](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig10)). We found that seven species were significantly correlated (Benjamini–Hochberg (BH) false discovery rate (FDR) adjusted *P*  < 0.01) with the overall variation profiles (for all RS and CS samples) (Fig. [2a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2)). We found that BMI significantly correlated (*P*  < 0.001 with the envfit test in R package VEGAN) with the overall taxonomic variation profiles (Fig. [2a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2)), suggesting that the change in body weight is associated with the restructured gut microbiome.

![Extended Data Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/058af939c952/42255_2024_988_Fig10_ESM.jpg)

( a ) The taxonomic profile of the gut microbiome at the species level for all the samples from RS and CS intervention. The samples were named using the rules of “Intervention (RS or CS) individual ID\_time point (week 0 or 8). ( b-e ) Analysis of baseline microbial composition between RS intervention baseline and CS intervention baseline. ( b ) Beta-diversity for the CS-Washout-RS group using the Bray–Curtis distance method. ( c ) Beta-diversity for the RS-Washout-CS group using the Bray–Curtis distance method. ( d ) Beta-diversity for the CS-Washout-RS group using the weighted-Unifrac distance method. ( e ) Beta-diversity for the RS-Washout-CS group using the weighted-Unifrac distance method. All the beta-diversity plots used Non-metric multidimensional scaling (NMDS) based on the compositional profile of the species abundance. The ANOSIM results support the baseline comparability of microbial compositions ( P > 0.05). Source data

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/3c780d5eb9ba/42255_2024_988_Fig2_HTML.jpg)

a, NMDS of the samples based on the variation (fold change after intervention) profile of the species abundance. The red and grey circles surround (with 95% CI) RS and CS samples, respectively. The species and phenotypes correlated with the overall ordination significantly (FDR adjusted P < 0.01 with the envfit function in VEGAN R package) are highlighted with arrows (blue, species; black, phenotype) where the length of the arrows reflects the strength of the association. b, Seven species with significantly different variation profiles ( < 0.05, BH FDR < 0.2 with paired two-sided Wilcoxon signed-rank test) between the RS and CS interventions. Box-plots indicate the median and IQR. Whiskers extend to 1.5 × IQR. c, The association network between gut microbes and the host phenotypes. A total of 12 species with different variations ( < 0.05, BH FDR < 0.3 with two-sided Wilcoxon rank-sum test) between the RS and CS groups were used as predictors in the regression model. Node colour reflects either the phenotype (yellow), species with increased abundance (green) or species with decreased abundance (red) in response to RS. The colour of the connecting line reflects either the positive correlation (red) or negative correlation (blue) between microbes and the phenotypes. The width of the connecting line reflects the strength of the statistical linear correlation (model averaged importance) between the abundance variation (fold change after intervention) and the variation of the phenotypes (fold change after intervention). The importance is the summed AIC weights of the generalized linear models containing such variables during model selection. The node size of the species reflects its overall influence on all the phenotypes. d, Summary of the importance of the associated species for each phenotype. In –, 27 individuals before and after RS and 16 individuals before and after CS were randomly selected from all participants for metagenomic analysis. Source data

To identify the gut microbiota signature associated with the RS intervention, we compared the changes in the abundance of each species after the RS and CS treatments (as a fold change in the abundance after the 8-week intervention). The results showed that three species, *Bifidobacterium* *adolescentis*, *Bifidobacterium* *longum* and *Ruminococcus* *bromii*, were significantly increased after the RS intervention but remained stable or decreased after the CS intervention. Conversely, four species, including *Alisipes* *putredinis*, *Bacteroides* *vulgatus*, *Odoribacter* *splanchnicus* and *Parabacteroides* *merdae* were decreased (*P*  < 0.05, BH FDR < 0.2 with a Wilcoxon signed-rank test) after the RS treatment but remained stable or increased after the CS treatment. Among the above, *R.**bromii, A.**putredinis* and *B.**adolescentis* were the most significantly varied species after the RS intervention compared with the CS intervention (FDR = 0.05) (Fig. [2b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2) and Supplementary Table [5](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)). Other degrading species were not significantly varied after the RS intervention in this study (Supplementary Table [6](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR25">25</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR26">26</a></sup>.

To identify species associated with RS intervention benefits, we investigated the correlation of 12 species (FDR < 0.3 via Wilcoxon rank-sum test) with six significantly altered parameters of the host, including BMI, fat mass, waist circumference, VFA, GIR and serum ANGPTL4 levels. A generalized linear model quantified the association between variation of species abundance and variation of the phenotypes. The strength of the association between the species and phenotypes was described by the importance of the variables in the linear model based on the model selection using corrected Akaike information criterion (AIC), which quantified the relative quality/weight of the statistical model used for given data. The importance of a variable reflects the overall support of all possible weighted linear models or the proportion of the total weights for all possible linear models containing this variable. Among the 12 species with differential abundance, *B.**adolescentis* had the most frequent associations with the various phenotypes (total number of links with the phenotypes) (Fig. [2c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2)). Among the phenotypes, BMI and GIR had the strongest association (average importance > 0.5 for the associated species) with the gut microbes (Fig. [2d](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig2)). At the single species level, we found that the increased abundance of *R.**bromii* after supplementation of RS was associated with an increase in GIR (importance 0.67 with *P*  < 0.001). Combined with the aforementioned significant increase in *R.**bromii*, our findings suggested that *R.**bromii* could be a key species in responding to the RS intervention, consistent with a previous study <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR27">27</a></sup>. *B.**adolescentis* may play a crucial role in alleviating obesity, evident by its strong correlation with lower BMI, waist circumference and VFA (importance 0.87, 0.47 and 0.27, respectively; *P*  < 0.001). Its abundance increases also positively correlated higher serum ANGPTL4 levels (importance 0.252, *P*  < 0.001), suggesting potential lipid metabolism effect. As primary RS degraders, the presence of *R.**bromii* and *B.**adolescentis* at baseline and their relationship with the key outcomes were explored. The fat mass of *B.**adolescentis* -positive individuals decreased even more and their ANGPTL4 levels increased even more after RS intervention (both *P*  < 0.05), demonstrating that the initial composition of gut microbiota, especially *B.**adolescentis*, was closely correlated with the benefits of RS (Extended Data Fig. [4](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig11)). Among the top ten metabolism-related Kyoto Encyclopaedia of Genes and Genomes (KEGG) pathways with the most differently varied abundances between the RS and CS intervention, the linoleic acid metabolism (ko00591) and bisphenol degradation (ko00363) pathways decreased during the RS treatment, whereas they increased significantly in the CS treatment (FDR = 0.057 for both pathways) (Extended Data Fig. [5](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig12)).

![Extended Data Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/4a34f0e930ac/42255_2024_988_Fig11_ESM.jpg)

( a-d ) The two-tailed Pearson correlation between ( a ) the baseline abundances of B. adolescentis and R. bromii. ( b and the abundance change of c d ) the abundance changes of. The changes in are calculated by the log 2 fold change after treatment. Baseline abundance values have undergone centred log-ratio (CLR) transformation before correlation analysis. The lines and grey zones show the fitted linear regression lines with 95% confidence intervals. ( e ) The relationship between the presence of key species ( ) and the change of key outcomes after RS treatment. The changes of key outcomes were presented by the log fold change after treatment. The violin plots illustrate the kernel density estimation of the data distribution, and the embedded box plots indicate the median and interquartile range. Whiskers extend to 1.5 times the IQR. n = 16 for positive individuals, n = 11 for negative individuals, n = 9 for positive individuals, and n = 18 for negative individuals. Statistical differences between the positive and negative groups were assessed using the two-sided Wilcoxon rank-sum test, with P values displayed above the plots. \* = 0.033 and 0.046 for fat mass and ANGPTL4 ( ). The colour indicates the presence (positive, orange) or absence (negative, cyan) of key species. Source data

![Extended Data Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/016788820474/42255_2024_988_Fig12_ESM.jpg)

Data are presented by the log 2 fold change after treatment, which are expressed as mean ± s.e.m (27 individuals before and after RS and 16 individuals before and after CS). Statistical differences between interventions were assessed using two-sided Wilcoxon rank-sum test, with P adjusted for multiple comparisons using the Benjamini–Hochberg method. Source data

### Association of RS-altered gut microbiota with metabolites

While it remains unclear how changes in the gut microbiota contribute to benefits in the host, a possible mechanism is through altered metabolic production <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR28">28</a></sup>. Thus, we further performed non-targeted metabolomics profiling in the serum of study participants. Serum metabolites related to obesity, such as carnitine and methionine <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR29">29</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR30">30</a></sup>, were decreased after the RS intervention compared with CS intervention (Supplementary Table [7](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)). We used a generalized linear model to investigate the relationship between gut microbes and serum metabolite variation during the RS intervention. The shift of *B.**adolescentis* during the RS intervention negatively correlated with the shift of serum carnitine, energy metabolism and lipid metabolites (average importance 0.15; Fisher’s combined *P*  < 0.05; Extended Data Fig. [6a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig13)). In contrast, the shift of *B.**vulgatus* during the RS intervention positively correlated with the shift of metabolites from energy metabolism, including xanthine and uric acid (*P*  < 0.01; Extended Data Fig. [6a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig13)).

![Extended Data Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/96bac5e117c8/42255_2024_988_Fig13_ESM.jpg)

( a ) The strength of the linear correlation (model averaged Akaike information criterion \[AlC\] importance) between the species abundance variation (fold change after intervention) and the variation (fold change after intervention) of the circulating metabolites. The importance is the summed AlC weights of the linear models containing such variable during model selection. ( b-g ) Changes of faecal SCFAs after the 8-week RS or CS intervention (n = 16 individuals per group). ( b ) Acetate. ( c ) Propionate. ( d ) Butyrate. ( e ) Isobutyrate. ( f ) Valerate. ( g ) Hexanoate. Data are expressed as median with interquartile range. \* P = 0.030 ( ), \*\* = 0.006 ( ) for the difference between RS and CS intervention by linear mixed model adjusted by intervention order and † = 0.047 ( ), †† = 0.004 ( ) for the difference before and after RS intervention (RS 0W vs. RS 8W) by linear mixed model adjusted by intervention order followed by Bonferroni’s test. Source data

As bile acids are significant signalling metabolites linking gut microbiota with the host, we conducted quantitative analysis of bile acid profiles. Glycodesoxycholic acid (GDCA) was the only bile acid with a statistically significant difference between the RS and CS interventions. Other bile acids, such as glycocholic acid (GCA), deoxycholic acid (DCA), 7-ketolithocholic acid (7-ketoLCA) and taurodeoxycholic acid (TDCA) exhibited a tendency to increase, whereas 3β-chenodeoxycholic acid (βCDCA) level tended to decrease after the RS intervention (Supplementary Table [8](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3) and Fig. [3a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig3)). Through correlation analysis, it was discerned that the secondary bile acids GDCA and TDCA had the closest correlation with metabolic phenotypes. The change of GDCA and TDCA was negatively correlated with obesity-related phenotypes, whereas positively correlated with insulin sensitivity and ANGPTL4 levels (Fig. [3b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig3)). The change of GDCA and TDCA after RS treatment was significantly and positively associated with four species, including *B.**adolescentis*, *B.**longum*, *Dorea* *longicatena* and *R.**bromii*, whereas negatively associated with another eight species, including *A.**putredinis*, *Bacteroides* *coprophilus*, *Bacteroides* *ovatus*, *B.**vulgatus*, *Eubacterium* *eligens*, *O.**splanchnicus*, *P.**merdae* and *P.**copri* (Fig. [3c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig3)). Additionally, we identified some microbial pathways strongly associated with the variation of several metabolites, including GDCA, TDCA, βCDCA, DCA and 7-ketoLCA (Fig. [3d](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig3)). Our results suggest that RS-influenced gut microbiota increases the production of secondary bile acids, which might lead to alleviation of obesity and insulin resistance. To determine whether the species were associated with bile acid metabolism, we investigated the association between gut microbiome and bile salt hydrolase (BSH; [K01442](https://www.ncbi.nlm.nih.gov/nuccore/K01442)), which is the gatekeeper of bile acid metabolism and host–microbiome crosstalk <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR31">31</a></sup>. Our results revealed a significant reduction in the abundance of the *BSH* gene within the gut microbial community level in the RS intervention compared with the CS intervention (*P*  < 0.05, Wilcoxon rank-sum test). We found that the expression of *BSH* was significantly correlated with *B.**vulgatus* (*r*  = 0.561, *P*  < 0.001; Supplementary Table [9](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)).

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/bdf198921146/42255_2024_988_Fig3_HTML.jpg)

a, Fold change of serum bile acid levels evaluated in participants after an 8-week RS or CS intervention. Data are presented as box-and-whisker plots. The horizontal line in the middle of each box indicates the median value, the top and bottom borders of the boxes represent the 75th and 25th percentiles and the whiskers denote the lowest and highest values. \* P = 0.036 by mixed linear model. b, The association network between bile acids and the host phenotypes. c, The association network between gut microbes and the host phenotypes, respectively. Node colour reflects either the bile acids (blue), the phenotype (yellow in ), or the microbes (yellow in ). The colour of the connecting line reflects either the positive correlation (red) or negative correlation (blue) between bile acids and phenotypes or between bile acids and microbes. The width of connecting line reflects the strength of the statistical linear association (model averaged importance) between the variation of bile acids (fold change after intervention) and the variation of the phenotypes (fold change after intervention) or between the abundance variation (fold change after intervention) and the variation of the bile acids (fold change after intervention). The importance is the summed AIC weights of the generalized linear models containing such variables during model selection. The node size of the bile acid reflects its overall influence on all the phenotypes or the microbes. d, KEGG pathway analysis showing the correlation between the variation of bile acids and the variation of KEGG pathway abundance. Source data

Short-chain fatty acids (SCFAs), primarily produced by gut microbial fermentation of dietary fibre, play an important role in the maintenance of gut and metabolic health <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR28">28</a></sup>. Therefore, we conducted targeted metabolomics to investigate the changes in faecal and circulating SCFAs after the RS and CS treatments. We found that faecal concentrations of isobutyrate and valerate significantly decreased after the RS intervention compared with the CS intervention, whereas other faecal SCFAs, including acetate, propionate, butyrate and hexanoate did not differ significantly after the RS intervention (Extended Data Fig. [6b–g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig13)). There was no significant difference in circulating levels of SCFAs between the RS and CS interventions (Supplementary Table [10](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)).

### RS-reshaped gut microbiota alleviates obesity in mice

To investigate the potential of the RS-related gut microbiota to trigger improvements in host abdominal obesity and glucose metabolism, we performed FMT in antibiotic-treated mice fed with a western diet, utilizing samples from human donors after RS or CS intervention (with changes in body weight post-intervention approximating the respective group average; *n*  = 4 per group) (Fig. [4a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig4)). We observed a consistent trend of RS-induced alterations between donors and mice receiving FMT (Extended Data Fig. [7a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig14)). Two weeks after FMT, the body weight and fat mass of the mice that received the RS microbiota were lower than those of mice receiving the CS microbiota (*P*  < 0.001) (Fig. [4b,c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig4)). The depot mass percentage of epididymal white adipose tissue (eWAT), peri-renal white adipose tissue (pWAT) and mesenteric white adipose tissue (mWAT) in mice colonized with the RS microbiota was significantly lower than that of mice colonized with CS microbiota (Fig. [4d,e](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig4)). Histological analysis of eWAT and mWAT presented a substantially reduced size of adipocytes in mice colonized with the RS microbiota (Fig. [4f,g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig4)). A comprehensive laboratory animal monitoring system indicated that the two groups of mice did not show significant differences in energy consumption, respiratory exchange ratio, CO <sub>2</sub> production, O <sub>2</sub> consumption, activity and food intake (Extended Data Fig. [7b–g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig14)). The glucose tolerance in these mice was at least partially improved by transplanting the gut microbiota from humans after the RS intervention (Extended Data Fig. [7h,i](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig14)). Circulating adiponectin levels in the mice colonized with the RS microbiota were significantly higher than in those colonized with the CS microbiota (Extended Data Fig. [7j](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig14)). These results suggested that RS-induced changes in the gut microbiota could be sufficient to alleviate obesity and improve host glucose metabolism.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/6966b852e236/42255_2024_988_Fig4_HTML.jpg)

a, Schematic diagram of FMT. Faecal samples from human donors after an 8-week RS or CS intervention ( n = 4 per intervention) were transplanted to antibiotic (Abx)-treated C57BL/6 mice. The mice were fed an irradiated western diet (WD) for 2 weeks before and during the 14 d of colonization ( = 16 per group). b, c, Body weight and fat mass changes after FMT. d, Representative photographs of visceral fat in mice colonized for 14 d with microbiota from RS or CS donors. e, Adipose depot mass including inguinal subcutaneous fat (iSAT), eWAT, pWAT and mWAT in mice colonized for 14 d with microbiota. f g, Representative images and quantification data of haematoxylin and eosin-stained sections of iSAT, eWAT and mWAT in the two groups of mice colonized with CS (top) or RS microbiota (bottom). Scale bar, 100 µm. Data were reproduced in three independent experiments. Data are presented as mean ± s.e.m. Significance was determined by paired two-tailed Student’s t -test ( ) and unpaired two-tailed Student’s ) (normally distributed) or nonparametric two-sided Wilcoxon rank-sum test (non-normally distributed). \* P = 0.02 and 0.01 ( ), \*\* = 0.002 ( ), \*\*\* < 0.001 ( ). Source data

![Extended Data Fig. 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/b087c9329793/42255_2024_988_Fig14_ESM.jpg)

( a ) Whole-genome shotgun metagenomic sequencing was performed for caecum content samples from mice receiving faecal slurry collected from donors collected after RS or CS intervention. Species showing a similar trend of RS-induced alterations between donors and mice receiving faecal microbial transplantation (FMT) were shown. Data are presented as log 10 (RS/CS). RS/CS was defined as the ratio of relative microbial abundance after RS intervention (FMT from RS donors for mice) to that after CS intervention (FMT from CS donors for mice). ( b-i ) Mice were grouped and treated as in Fig. 4 (n = 8 biological replicates per group). ( b ) 24-hour energy expenditure, ( c ) respiratory exchange ratio (RER), ( d ) oxygen consumption, ( e ) carbon dioxide production, ( f ) activity, and ( g ) food intake were measured using the comprehensive laboratory animal monitoring system at the end of the experiment. ( h ) Glucose tolerance test (GTT) was performed in mice colonized with microbiota from RS or CS donors. ( i ) Area under the curve (AUC) analysis of GTT. ( j ) Serum adiponectin levels were measured at the end of faecal microbiota transplantation (n = 15 biological replicates per group). Data were reproduced in three independent experiments. Data are presented as mean ± s.e.m. Significance was determined by two-tailed Student’s t-test. \* P = 0.03 ( ), and \*\*\* < 0.001 ( ). Source data

### RS-reshaped gut microbiota restores gut barrier

Consistent with the clinical trial findings, RS-altered gut microbiota ameliorated systemic inflammation. Two weeks after FMT, circulating MCP-1, IL-1β and IL-6 levels in the mice colonized with RS microbiota were significantly lower than in those colonized with CS microbiota (all *P*  < 0.01) (Fig. [5a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). Serum levels of the anti-inflammatory factor IL-10 in mice colonized with RS microbiota increased (Fig. [5a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). In mesenteric adipose tissues, the expression of *MCP-1*, *IL-1β* and *IL-6* was also significantly lower, whereas that of *IL-10* was higher in the mice receiving RS microbiota compared with those receiving CS microbiota (Fig. [5b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)).

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/62585819c19c/42255_2024_988_Fig5_HTML.jpg)

Mice were grouped and treated as in Fig. 4 ( n = 16 per group). a, Serum levels of inflammatory cytokines in mice colonized with microbiota from RS or CS donors. b, The expression of inflammatory genes in mWAT and in mice colonized with microbiota from RS or CS donors. c, Gut permeability in vivo. d, The expression of ZO-1 and occludin in the ileum. e, The localizations and levels of ZO-1 (red) and occludin (green) in intestinal villus were visualized by immunofluorescence and counterstaining with 4,6-diamidino-2-phenylindole (DAPI) (blue). Representative images of each group are shown (scale bar, 50 µm). f, Quantitative analysis of the positive stained area of ZO-1 and occludin was performed by ImageJ software and calculated as the percentage of total lesion area. g, h, LPS levels in mWAT and in circulation (serum). i, Expression levels of ANGPTL4 in the ileum. j, Relative intestinal luminal lipase activity. k, TG levels in the ileum. l, Faecal TG levels. m, Serum ANGPTL4 levels. Data were reproduced in three independent experiments. Data are presented as mean ± s.e.m. Significance was determined by unpaired two-tailed Student’s t -test (normally distributed) or nonparametric two-sided Wilcoxon rank-sum test (non-normally distributed). \*\* P = 0.005 ( ) and 0.001 and 0.010 ( ), \*\*\* < 0.001. Source data

Metabolic endotoxaemia, which may be attributable to the increased penetration of lipopolysaccharide (LPS) from the gut to circulation, serves as a pivotal factor in obesity-induced chronic inflammation <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. To investigate how RS-altered gut microbiota interact with metabolic endotoxaemia, we further studied the in vivo gut permeability in mice colonized with microbiota from human donors following RS and CS interventions. The mice were orally gavaged with fluorescein isothiocyanate-labelled dextran (DX-4000-FITC) after a 6-h fasting period and subsequently, serum samples were collected via the tail vein. The concentration of DX-4000-FITC in the serum was determined using a fluorescence spectrophotometer. Intestinal permeability was significantly lower in mice colonized with RS microbiota compared with those colonized with CS microbiota (Fig. [5c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). Gut permeability is regulated by intestinal tight junctions of that block the invasion of pathogens and bacterial products <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. We evaluated the effect of the RS-induced gut microbiota changes on the expression levels of the major tight junction proteins in the intestine, zona occludens protein (ZO-1) and occludin. The expressions of both *ZO-1* and *occludin* genes were significantly increased in the mice that received the RS microbiota compared with those that received CS microbiota (Fig. [5d–f](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). Furthermore, in mice colonized with gut microbiota from donors after the RS intervention, restoration of the gut barrier led to a significant reduction in LPS levels in both mesenteric adipose tissue and circulation (Fig. [5g,h](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). These results suggested that one of the beneficial effects of the altered microbiota was to block LPS penetration by preserving the gut barrier.

### RS-altered gut microbiota reduces lipid absorption

Our clinical study showed significantly increased circulating levels of ANGPTL4 and faecal lipids in participants after RS consumption compared with CS consumption. To investigate whether the increase in ANGPTL4 was regulated by the RS microbiota, we measured the levels of *ANGPTL4* expression in the ileum of mice from the FMT experiments. Compared with mice colonized with CS microbiota, mice colonized with RS microbiota had significantly higher ileal *ANGPTL4* expression (Fig. [5i](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). ANGPTL4 is an endogenous inhibitor of lipoprotein lipase (LPL) and also pancreatic lipase, which functions as the principal intestinal lipase <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR32">32</a></sup>. We found that the mice colonized with the RS microbiota had a significantly lower intestinal luminal lipase activity compared with mice that received CS microbiota (Fig. [5j](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). In accordance with the inhibited luminal lipase activity, mice colonized with RS microbiota exhibited lower TG levels in the ileum and higher TG levels in the faeces than mice colonized with CS microbiota (Fig. [5k,l](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). Circulating ANGPTL4 levels were also significantly increased in mice colonized with RS microbiota compared with mice colonized with CS microbiota (Fig. [5m](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig5)). These results indicated that RS-induced changes in the gut microbiota were able to reduce lipid absorption through modulating intestinal ANGPTL4.

### Effects of B.adolescentis on obesity

The increase in *B.**adolescentis* caused by RS was strongly correlated with the alleviation of abdominal obesity. Given this, we further investigated the potential causal association between *B.**adolescentis* in the gut and host obesity in conventionally raised mice. Mice were randomized into three groups, drinking sterile water fortified with live *B.**adolescentis*, heat-killed *B.**adolescentis* or saline for 5 weeks (Fig. [6a,b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)). Supplementing drinking water with live *B.**adolescentis* significantly alleviated body weight gain (Fig. [6c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)) and adiposity (Fig. [6d](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)) in mice fed a western diet. We further investigated whether *B.**adolescentis* regulated *ANGPTL4* expression in the intestine. In comparison with mice given drinking water supplemented with heat-killed *B.**adolescentis*, ileal *ANGPTL4* expression and secretion were significantly higher in mice given drinking water supplemented with live *B.**adolescentis* (Fig. [6e–g](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)). Consistent with our clinical trial, *B.**adolescentis* positively correlated with serum ANGPTL4 levels. Mice receiving live *B.**adolescentis* had significantly decreased luminal lipase activity (Fig. [6h](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)). Correspondingly, these mice had decreased ileal TG and increased faecal TG levels (Fig. [6i,j](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)). Circulating ANGPTL4 levels were also significantly increased in the mice treated with live *B.**adolescentis* (Fig. [6k](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig6)). Our findings suggest that *B.**adolescentis*, one of the key species induced by RS, may protect mice against diet-induced obesity by affecting intestinal ANGPTL4. In addition, the level of FGF21 significantly decreased in the participants after RS intervention, indicating the improvement of FGF21 sensitivity (Fig. [1m](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). We found that the treatment with *B.**adolescentis* in mice increased the sensitivity to FGF21 in adipose tissue by suppressing the LPS–TLR4–NF-κB pathway (Extended Data Fig. [8](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig15)).

![Fig. 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/181907b56de1/42255_2024_988_Fig6_HTML.jpg)

a, Schematic diagram of B. adolescentis ( B.a ) supplementation strategy. WD-induced obese C57BL/6 mice were supplemented with drinking water with live (WD + ) or heat-killed (WD + hk- ) or saline (WD) for 5 weeks. b, The abundance of quantified by qPCR using the specific primers in caecum content among three groups. c – k, Parameters measured in the three groups after 5 weeks of treatment: body weight ( ). Adipose depot mass including iSAT, eWAT, pWAT and mWAT ( d ). The localizations and levels of ANGPTL4 in ileum of mice were visualized by immunohistochemistry staining (scale bar, 100 µm) ( e ). Representative images of each group were shown. ANGPTL4 + area as a percentage of total ileal mucosa area ( f ). ANGPTL4 mRNA expression levels in the ileum ( g ). Relative intestinal luminal lipase activity ( h ). TG levels in the ileum ( i ). Faecal TG levels ( j ). Serum ANGPTL4 levels ( n = 8 biological replicates for each group (, ). Data were reproduced in three independent experiments. Data are presented as mean ± s.e.m. Significance was determined by one-way ANOVA (normally distributed) followed by Tukey’s post hoc test or Kruskal–Wallis test (non-normally distributed) followed by Dunn’s test. \* P = 0.04 ( ), 0.02, 0.04, 0.03 ( ) and 0.05 and 0.05 ( ). \*\* = 0.001, 0.004 ( ), 0.003, 0.004 and 0.002 ( ), 0.001 ( ), 0.009 ( ), 0.002 ( ) and 0.003 and 0.003 ( ). \*\*\* < 0.001. Source data

![Extended Data Fig. 8](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/73e2ac813000/42255_2024_988_Fig15_ESM.jpg)

The mice were fed the western diet for 8 weeks before administration of B. adolescentis and were randomized into three groups, drinking sterile water fortified with (WD+B.a), heat-killed (WD+hk-B.a) or PBS (WD) for 5 weeks. After an 8-week intervention period, a fibroblast growth factor 21 (FGF21) response test was conducted on mice. ( a ) The level of serum lipopolysaccharide (LPS). ( b ) The transcription of cluster of differentiation 14 (CD14) in white adipose tissue (WAT). ( c ) The transcription of toll-like receptor 4 (TLR4) in WAT. ( d ) The serum level of FGF21. ( e ) The transcription of fibroblast growth factor receptor 1 (FGFR1) in WAT. ( f ) The transcription of βklotho (KLB) in WAT. ( g ) The transcription of early growth response factor 1 (EGR1) in WAT. ( h ) In vivo phosphorylation of extracellular signal-regulated kinase 1/2 (Erk1/2) (Thr202/Tyr204) and total Erk1/2 before and after rmFGF21 administration (2mg/kg, i.v.) detected by western blot analysis. ( i ) Quantification analysis of the fold change of Erk1/2 phosphorylation. The samples derive from the same experiment and that gels were processed in parallel. Data are mean ± s.e.m. WD+B.a, n = 5 biological replicates; WD+hk-B.a, n = 5 biological replicates; WD, n = 6 biological replicates. \* P = 0.04 ( ), 0.01 ( ), and 0.02 ( ), \*\* = 0.004 ( ), 0.009 ( ), 0.001 ( ), 0.002 ( ), and 0.002 ( ), \*\*\* < 0.001, based on one-way ANOVA (normally distributed) followed by Tukey’s post hoc test or Kruskal–Wallis test (non-normally distributed) followed by Dunn’s test. Source data

We further conducted a targeted metabolomics analysis in conventionally raised mice treated with *B.**adolescentis* (Extended Data Fig. [9a–c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)). The primary bile acids, including tauro-β muricholic acid (TβMCA), taurochenodeoxycholic acid (TCDCA) and taurocholic acid (TCA) significantly decreased and the secondary bile acid taurolithocholic acid significantly increased after *B.**adolescentis* intervention (Extended Data Fig. [9a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)). Regarding SCFAs, the faecal levels of isobutyrate and valerate also decreased significantly after *B.**adolescentis* intervention (Extended Data Fig. [9b](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)). In addition, the levels of serum branched-chain amino acids (BCAAs), including valine and leucine, significantly decreased after *B.**adolescentis* intervention (Extended Data Fig. [9c](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)).

![Extended Data Fig. 9](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/909acfcedf07/42255_2024_988_Fig16_ESM.jpg)

Metabolomics profile of ( a-c ) SPF mice were grouped and treated as in Extended Data Fig. 7 (WD+B.a, n = 5 biological replicates; WD+hk-B.a, n = 5 biological replicates; WD, n = 6 biological replicates). and ( d-f ) germ-free mice were grouped and treated as in Fig. 7 (n = 5 biological replicates per group). ( a ) Levels of GCA, DCA, TDCA, TβMCA, TCDCA, TCA, and TLCA in serum. ( b ) Levels of SCFAs in faeces. ( c ) Levels of valine, isoleucine, and leucine in serum. ( d ) Levels of GCA, DCA, TβMCA, TCDCA, and TCA in serum of germ-free mice. ( e ) Levels of SCFAs in faeces of germ-free mice. ( f ) Levels of valine, isoleucine, and leucine in serum of germ-free mice. Data are presented as box-and-whisker plots. The horizontal line in the middle of each box indicates the median value, the top and bottom borders of the boxes represent the 75th and 25th percentiles, and the whiskers denote the lowest and highest values. Analysed by one-way ANOVA followed by Tukey’s post hoc test (in -c ) and two-tailed Student’s unpaired t-test (in ). \* P = 0.05, 0.03, and 0.04 ( ), 0.02 ( ), 0.016 and 0.024 ( ), 0.015 ( ), and 0.034, 0.028, and 0.025 ( ), \*\* = 0.006 and 0.003 ( ), 0.008 ( ), 0.006 ( ), 0.002 and 0.005 ( ), \*\*\* < 0.001. SPF, specific pathogen-free; GCA, glycocholic acid; DCA, deoxycholic acid; TDCA, taurodeoxycholate; TβMCA, tauro-β muricholic acid; TCDCA, taurochenodeoxycholic acid; TCA, taurocholic acid; TLCA, taurolithocholic acid. Source data

### RS facilitates weight loss partially through B.adolescentis

To explore whether the effects of RS depend on gut microbiota, we carried out the RS and CS interventions in germ-free mice (Fig. [7a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig7)). Male germ-free mice were fed a C35 diet (35% carbohydrate sourced from 20% RS or 20% CS) for 10 weeks. Meanwhile, these mice were treated with PBS or live *B.**adolescentis* for 8 weeks. As intestinal contents and the thickness of the lumen were significantly increased after RS feeding, the disembowelled body weight was commonly used to evaluate the effect of weight loss <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR12">12</a></sup>. Unlike the protecting effect of RS against diet-induced obesity in conventional environment <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR33">33</a></sup>, RS treatment showed no significant change on disembowelled body weight, glucose tolerance and insulin sensitivity compared with CS treatment in germ-free mice (Fig. [7c–f](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig7)). To further investigated the effects of RS with and without *B.**adolescentis*, we fed the germ-free mice with RS together with *B.**adolescentis*. These mice showed significant decrease in disembowelled body weight and fat mass, as well as the improvement on glucose tolerance and insulin sensitivity compared with mice without *B.**adolescentis* (Fig. [7c–f](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig7)). *B.**adolescentis* reduced gut permeability and increased the intestinal production of ANGPTL4 in germ-free mice, which was consistent with our finding in a conventional environment (Fig. [7g–k](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig7)). Germ-free mice gavaged with *B.**adolescentis* exhibited reduced luminal lipase activity and ileum TG levels, alongside increased faecal TG levels (Fig. [7l,m](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig7)). These results indicated that gut microbiota was essential in the action of RS in protecting against diet-induced obesity. RS played the role in facilitating weight loss, at least partially, through *B.**adolescentis* via reducing inflammation by restoring the gut barrier and modulating ANGPTL4 production in the intestine, thereby impeding lipid absorption.

![Fig. 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/e7da89842b4b/42255_2024_988_Fig7_HTML.jpg)

a, Schematic diagram of B. adolescentis ( B.a ) supplementation strategy. Orally inoculating or PBS into germ-free (GF) mice on diets with 20% protein, 45% fat and 35% carbohydrate sourced (from 20% CS or 20% RS and the remaining 80% from maltodextrin) for 8 weeks. b, Abundance of quantified as total plate count in caecum content. c.f.u., colony-forming unit. c, Disembowelled body weight. d, Glucose excursion curves of intraperitoneal glucose tolerance tests (GTTs). e, Glucose excursion curves of intraperitoneal insulin tolerance tests (ITTs). f, Body mass composition. g, After an 8-week supplementation, in vivo gut permeability was determined by measurement of serum concentrations of DX-4000-FITC at 1 h after oral gavage. h, Expression of ZO-1 and occludin in the ileum. i, Expression of inflammatory genes in mWAT. j, mRNA expression levels of ANGPTL4 k, Relative intestinal luminal lipase activity. l, TG levels in the ileum. m, Faecal TG levels. Data are mean ± s.e.m. ( n = 5 biological replicates per group). \* P = 0.04 and 0.03 ( ), 0.02 ( ), 0.05, 0.04 ( ) and 0.03 ( ), \*\* = 0.002 ( ), 0.002 ( ), and 0.004 ( ), \*\*\* < 0.001 based on one-way ANOVA (normally distributed) followed by Dunnett’s test or Kruskal–Wallis test (non-normally distributed) followed by Dunn’s test. \* indicates the comparison between RS + and RS + PBS (, ). Source data

In addition, the serum levels of primary bile acids in germ-free mice were higher than those in the conventional environment and the levels of secondary bile acids were relatively low <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR34">34</a></sup>. GCA, a conjugated bile acid and DCA, a secondary bile acid, exhibit a tendency to increase after RS intervention in humans. GCA and DCA significantly increased after *B.**adolescentis* intervention in germ-free mice. The primary bile acids, including TβMCA, TCDCA and TCA significantly decreased after *B.**adolescentis* intervention in germ-free mice (Extended Data Fig. [9d](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)). The levels of serum BCAAs and faecal isobutyrate were decreased after *B.**adolescentis* intervention (Extended Data Fig. [9e,f](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig16)).

## Discussion

This placebo-controlled, double-blinded and crossover-designed trial was conducted in individuals with excess body weight. We found that RS supplementation, coupled with isoenergetic and balanced diets, significantly reduced body weight and improved insulin sensitivity in humans. RS treatment reshaped the microbiome structure and altered metabolites. Gut microbiota plays a key role in RS’s efficacy for weight loss. Monocolonization of mice with *B.**adolescentis*, linked to the benefits of RS in humans, prevented diet-induced obesity in mice. Mechanistically, the RS-induced changes in gut microbiota influenced bile acid metabolism, reduced inflammation through gut barrier restoration, inhibited lipid absorption by modulating ANGPTL4 and improved FGF21 sensitivity (Extended Data Fig. [10](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig17)). RS played a role in facilitating weight loss at least partially through *B.**adolescentis*. Specifically, this study provides evidence that RS (40 g d <sup>−1</sup>, type 2) as a dietary supplement for 8 weeks can help to achieve weight loss in individuals with excess body weight. In this feeding study, we provided the participants with a background diet throughout the study. The background diets, which were isoenergetic and balanced <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR20">20</a></sup>, did not influence body weight or fat content over an 8-week period, as observed following CS treatment. In previous trials using RS, relatively insufficient RS intake (fat:fibre ratio ≈53:27 g d <sup>−1</sup>) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR35">35</a></sup> or high fat intake (fat:RS ratio ≈77:40 g d <sup>−1</sup>) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR14">14</a></sup> resulted in reduction of insulin resistance or fat percentage without weight loss. Other trials did not prescribe a background diet and the fat intake was either too high or not assessed <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR15">15</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR16">16</a></sup>. High-fat diets induce changes in the intestinal microbiota that lead to impaired gut health <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR36">36</a></sup>. In microbiome-based therapies, a background diet is critical for efficacy <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. In rodents, low-fat diets (18% of energy) supplemented with RS have beneficial effects on the hosts, but high-fat diets (42% of energy) attenuated RS fermentation and the beneficial effects <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR18">18</a></sup>. As 40 g d <sup>−1</sup> RS was deemed to be a relatively high dose that could be delivered without adverse effects <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR37">37</a></sup>, the fat content in the daily diet is important for the effect of RS. Our study provided an effective dietary recommendation using RS as a supplement (40 g d <sup>−1</sup> with a balanced background diet containing 25–30% fat), which may help to achieve significant weight loss. The design of the background diet and good adherence enabled us to mitigate the influence of significant confounding variables known to have impacts on the gut microbiome and metabolome.

![Extended Data Fig. 10](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a4f6/10963277/722f578429ac/42255_2024_988_Fig17_ESM.jpg)

Resistant starch (RS, 40 g d -1 ) accompanied with isoenergetic and balanced diets led to an obvious reduction in body weight and improvement of insulin sensitivity, as well as alteration in metagenomics and metabolomics. Faecal microbiota transplantation (FMT) showed benefits of RS were associated with the reshaped gut microbiota composition. Monocolonization of mice with B. adolescentis, which was closely correlated with the benefits of RS in human protected mice from diet-induced obesity. Mechanistically, the RS-induced changes in the gut microbiota influenced metabolites of gut microbiome, reduced chronic low-grade inflammation by improving intestinal integrity, inhibited lipid absorption by modulating angiopoietin-like 4 (ANGPTL4), and improved the sensitivity of fibroblast growth factor 21 (FGF21) in adipose tissue. SPF, specific-pathogen-free; LPS, lipopolysaccharide; BCAAs, branched-chain amino acids; Erk1/2, extracellular signal-regulated kinase 1/2; FGFR1, fibroblast growth factor receptor 1. Created with BioRender.com.

This study represents one of the few that reports a specific microbiota signature after RS2 intervention. The taxonomic groups commonly enriched post-RS2 consumption include *R.**bromii*, *B.**adolescentis*, *Faecalibacterium* *prausnitzii* and *Eubacterium* *rectale*. *R.**bromii*, which acts as primary RS2 degrader was enriched during previous RS2 interventions <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR38">38</a></sup>. In this study, *R.**bromii* and *B.**adolescentis* notably increased after RS intervention in individuals with excess body weight. The increased abundance of *B.**adolescentis* strongly correlated with decreased BMI and VFA, suggesting its role in RS’s weight-loss benefits. Participants with *B.**adolescentis* in their gut microbiome at baseline exhibited a greater decrease in fat mass after RS treatment. Furthermore, we transmitted the beneficial effects of RS on host obesity and glucose metabolism in mice via human microbiota engraftment, reinforcing our hypothesis that RS-induced changes in the microbiota can drive beneficial host outcomes. Furthermore, gut microbiota was essential in the benefits of RS, which was verified through RS intervention in germ-free mice.

The production of microbiota-derived metabolites is involved in the function of prebiotics <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. Bile acids are significant signalling metabolites linking gut microbiota with the host <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR28">28</a></sup>. Herein, the secondary bile acid, GDCA, was significantly increased after RS treatment, other secondary bile acids, such as DCA, 7-ketoLCA and TDCA, exhibited a tendency to increase. Secondary bile acids have been reported to ameliorate hepatic steatosis and augment insulin sensitivity <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR39">39</a> – <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR41">41</a></sup>. BSH carries out bile acid deconjugation <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR31">31</a></sup>. In our study, the abundance of the *BSH* gene decreased and showed a significant correlation with *B.**vulgatus*. Inhibiting BSH activity emerged as a potentially beneficial approach to regulate lipid and energy homoeostasis, leading to an increase in conjugated bile acids <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR31">31</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR34">34</a></sup>. In our cohort, conjugated bile acids, including GCA, GDCA and TDCA, demonstrated an increasing trend after RS treatment. Consistent with the findings in humans, GCA and DCA significantly increased after *B.**adolescentis* intervention in germ-free mice. GCA and DCA are the agonists to FXR and Takeda G protein-coupled receptor 5 (TGR5), respectively, which are known receptors regulating glucose, lipid and energy metabolism <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR42">42</a></sup>. The serum level of BCAAs, including valine and leucine, significantly decreased after *B.**adolescentis* intervention in both conventional and germ-free environment. Serum levels of BCAAs decreased after the RS intervention in participants with non-alcoholic fatty liver disease (NAFLD) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR43">43</a></sup>. Faecal isobutyrate, a microbial product of the proteolysis of valine <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR44">44</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR45">45</a></sup>, decreased after RS intervention in our study.

The interplay between energy regulation and inflammation was modulated by the gut microbiome, influencing intestinal permeability and generating pro-inflammatory factors that exhibited variable effects on metabolic health <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. Integrity of the intestinal mucosa is maintained via intercellular tight junctions, which are essential regulators of intestinal permeability <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR22">22</a></sup>. The levels of the two tight junction proteins, occludin and ZO-1, elevated in the ileum and circulating LPS decreased in mice that received RS-induced microbiota, suggesting an additional mechanism of protecting the intestinal barrier by RS-altered gut microbiota. Chronic low-grade inflammation and immune system activation were involved in the development of obesity and insulin resistance <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR22">22</a></sup>. Weight management influenced by gut microbiome may also through modulation of secretory protein involved in energy utilization and homoeostasis <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR7">7</a></sup>. Notably, the metagenomics analysis demonstrated that the increased abundance of *B.**adolescentis* after the RS treatment was positively associated with serum ANGPTL4. ANGPTL4 has garnered substantial interest in relation to the gut microbiota. Germ-free mice exhibited elevated intestinal *ANGPTL4* expression in compared with mice raised conventionally <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR3">3</a></sup>. The reduction in ANGPTL4 levels contributes to increased fat accumulation upon conventionalization through increased LPL activity in adipose tissue <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR3">3</a></sup>. Colonization of germ-free mice with acetate-producing *Bacteroides* *thetaiotaomicron* or methane-producing *Methanobrevibacter* *smithii*, resulted in decreased intestinal *ANGPTL4* expression, whereas colonization with the butyrate-producing bacteria *Clostridium* *tyrobutyricum* significantly elevated intestinal *ANGPTL4* expression <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR46">46</a></sup>. A possible explanation for this result is that the modulation of intestinal ANGPTL4 is contingent on the gut microbiota composition, which determines the mixture of microbial metabolites formed, together with substrate provision <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR46">46</a></sup>. Using FMT, we observed that RS-induced changes in gut microbiota reduced lipid absorption through modulating intestinal ANGPTL4. Furthermore, colonization of diet-induced obese mice raised in both conventional and germ-free environments with *B.**adolescentis* markedly elevated intestinal ANGPTL4 levels. Thus, manipulating pancreatic lipase activity via ANGPTL4, a key regulator of lipid absorption, through gut microbiota modulation, might be a crucial way to alter body fat storage. *B.**adolescentis* supplementation could mitigate NAFLD by enhancing FGF21 sensitivity in the liver <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR47">47</a></sup>. Our results also revealed that treatment with *B.**adolescentis* increased FGF21 sensitivity in adipose tissue by suppressing the LPS–TLR4–NF-κB pathway. Multiple clinical trials of FGF21 analogues and mimetics showed improved lipid profiles, increased adiponectin and decreased body weight <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR24">24</a></sup>. It was uncovered that the reversal of obesity-induced FGF21 resistance in adipose tissue could serve as an alternative approach for treating obesity and related diseases <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR48">48</a></sup>.

The main limitation of the study was its relatively small sample size and stringent inclusion criteria for participants, limiting result generalizability. Our database-dependent, taxon-based analysis, while offering comprehensive and well-annotated information for taxonomic assignments, also faces limitations such as discarding sequences that are difficult to classify and overlooking strain-level functional diversity. We acknowledge that integrating metagenome-assembled genomes could enhance our understanding of microbial ecology and potentially improve the identification of robust biomarkers for clinical translation. The broader generalizability of our results requires additional validation in larger and more diverse cohorts. Future studies will need to pay special attention to the individual dynamics and functional responses of microbiota in RS supplementation. The in-depth insight into the crosstalk between RS-altered bile acids and gut microbiota and its impact on host metabolism are warranted in further investigations. Although the applicability to other populations remains unconfirmed, our study provides the evidence that RS (40 g d <sup>−1</sup>, type 2) as a dietary supplement facilitates weight loss in individuals with excess body weight. Our study also provides an effective dietary design along with RS to mitigate the influence of major confounding variables affecting the gut microbiome. Gut microbiota change driven by RS was essential for its effect. The gut microbiota played a pivotal role in this weight-loss mechanism, potentially modulating obesity through interactions with low-grade inflammation and regulation of energy-balance-related secretory proteins. Through the crossover design, we found that body weight was regained during the washout period. Weight regain is one of the biggest challenges of weight loss treatment, which occurs after stopping medication, such as GLP-1 agonists and even after bariatric surgery <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR49">49</a></sup>. Long-term adherence to an RS-rich dietary pattern to maintain the composition of the microbiome may be crucial for weight maintenance. As RS occurs naturally in foods and can also be added to daily diets, our findings provide a pragmatic lifestyle to treat obesity and its related metabolic disorders. Manipulating the gut microbial composition through diet may represent a strategy for modifying host energy balance to promote health.

## Methods

### Study participants

The inclusion criteria were (1) age 18–55 years and (2) overweight or obesity, as defined as BMI ≥ 24 kg m <sup>−2</sup> or waist circumference ≥85 cm in men and ≥80 cm in women <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR20">20</a></sup>. The exclusion criteria were acute illness, antibiotic or probiotic supplement use within 3 weeks previously, diagnosis of hyperthyroidism or hypothyroidism, diabetes, current systemic corticosteroid treatment or medications affecting glucose metabolism, and participation in other clinical trials within 4 weeks before the study.

This study was approved by the ethics committee of the Shanghai Sixth People’s Hospital and conformed with the Declaration of Helsinki. Written informed consent was obtained from all participants. Complete clinical trial registration was deposited in the Chinese Clinical Trial Registry (ChiCTR-TTRCC 13003333) (Extended Data Fig. [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig8)).

### General protocol of the clinical trial

The detailed study protocol was submitted to the Chinese Clinical Trial Management Public Platform and is available in the [Supplementary Information](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM1). Individuals with excess body weight consumed either RS derived from maize (HAM-RS2, Hi-Maize 260 resistant starch, 22000B00, provided by Ingredion) or energy-matched CS (AMIOCA cornstarch, 04400110, also provided by Ingredion) alternately for 8 weeks and separated by a 4-week washout period. An independent researcher performed the randomization and participant allocation to the CS-Washout-RS or RS-Washout-CS intervention schemes with at a 1:1 allocation ratio. The randomization schedule was produced by SAS PROC PLAN in SAS software. RS and CS were packaged in identical sealed bags with an identical appearance and participants and investigators were blinded to the group allocations during the double-blind period. Only the research designer was aware of the randomization scheme, whereas the participants, investigators, clinical staff and outcome assessors were blinded to it. The blinding was lifted during the bioinformatics analysis to explore the potential mechanism by which the gut microbiota conferred the physiological benefits of RS.

During this feeding study, participants received a uniform background diet, following the Chinese and American guidelines for the prevention and control of adults with overweight and obesity <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR20">20</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR21">21</a></sup>. All participants were either lightly active or led sedentary lives. The diet provided 25 kcal kg <sup>−1</sup> ideal body weight (ideal body weight (kg) = height (cm) − 105) daily, with 50–60% carbohydrate, 25–30% fat and 15–20% protein. Participants were permitted one item of fruit daily and advised against extra-sugary beverages or snacks. They maintained a diary recording three consecutive 24-h dietary intake (two weekdays and one weekend day) at baseline and at the beginning and end of each intervention period, noting any dietary deviations. A trained dietician assessed starch consumption and dietary regimen adherence during visits. Diet data revealed similar average calorie intake and macronutrient percentages in the RS and CS intervention periods (Supplementary Table [1](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3)).

Participants attended the institute for visits (V) 1–10 (Fig. [1a](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Fig1)). At the start and the end of each intervention (V1, V5, V6 and V10), participants had a hyperinsulinemic–euglycemic clamp, MTT, MRI scans and sample collections <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR37">37</a></sup>. Anthropometric and biochemical assessments were conducted at each visit. The primary outcome was body weight and secondary outcomes included VFA and SFA, body fat, waist circumference, lipid profiles, insulin sensitivity, metabolome and gut microbiome.

### Anthropometric and biochemical assessments

After overnight fasting for at least 10 h, anthropometric test and samples (venous blood, urine and faeces) collection was conducted in the morning following the study protocol.

Serum samples after centrifugation were stored at −80 °C until measuring AST, ALT, GGT, HDL-C, LDL-C and insulin <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR37">37</a></sup>. Serum A-FABP (AIS, HKU, 31030), adiponectin (AIS, HKU, 31010), inflammatory cytokines (Ebiosciense, TNFα, BMS223HS; MCP-1, BMS281; IL-1β, BMS224HS; IL-6, BMS213HS; IL-10, BMS215HS), ANGPTL4 (BioVendor, RD191073200R) and FGF21 (AIS, HKU, 31180) were quantified by enzyme-linked immunosorbent assay (ELISA). Serum LPS levels were measured through sensitive limulus amoebocyte lysate (LAL) assay (Hycult Biotechnology, MAK109).

### MTT experiment

To evaluate glucose metabolism, serial blood samples were collected in both fasting and postprandial states for laboratory tests following a standard meal (315.2 kcal, including 68.4 g carbohydrate and 10.4 g protein) (China Oil & Foodstuffs Corporation).

### Hyperinsulinemic–euglycemic clamp

Insulin sensitivity was evaluated through a hyperinsulinemic–euglycemic clamp. Insulin levels were maintained at approximately 100 μU ml <sup>−1</sup> through a prime-continuous infusion of insulin. To maintain the concentration of plasma glucose at basal levels, variable glucose infusion following a negative feedback principle was conducted. The glucose clamp technique was performed as previously described <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR50">50</a></sup>.

### Faecal DNA extraction and sequencing

Faecal samples were collected through a tube with a DNA stabilizer (STRATEC Molecular) and stored at −80 °C. Faecal genomic DNA from both humans and mice receiving FMT from human donors was extracted using PSP Spin Stool DNA kit (STRATEC Molecular, 1038100) following the manufacturer’s instructions. Faecal DNA extracts from 86 samples (27 samples before and after RS and 16 samples before and after CS) were randomly selected from all participants for shotgun metagenomic sequencing. HiSeq 1500 was used for 100-bp paired-end shotgun metagenomic sequencing for human samples at the Centre for Genomic Sciences, the University of Hong Kong. An Illumina platform Novaseq 6000 was used for 150-bp paired-end sequencing for mice samples at Novogene. Raw sequences were stored in the National Center for Biotechnology (NCBI) Sequence Read Archive (project ID [PRJNA414688](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA414688)).

### Metagenomics analysis

#### Quality control for metagenomic data

We used a series of quality control steps to remove low-quality reads/bases as described previously <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR51">51</a></sup>. In brief, all Illumina primer/adaptor/linker sequences were removed and low-quality regions (quality score <20) and reads were trimmed. Subsequently, we mapped all reads to the human genome using BWA v.0.7.4 (ref. <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR52">52</a></sup>). Reads with >95% identity and 90% coverage were removed as human DNA contamination.

#### Taxonomy and functional profiling

Taxonomy affiliations for reads were determined by MetaPhlAn <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR53">53</a></sup>. We distilled the top 50 most-abundant species based on the average abundance across all samples (minimum average abundance > 0.3%), which accounted for 91% of the total relative abundance in each sample on average. The proportion of unclassified reads was 8.06 ± 6.71% (mean ± s.d.) across all samples. To overcome potential bias from baseline differences between individuals within groups and to facilitate cross-group comparisons, we compared community-level variation patterns (based on the fold change in abundance of species), instead of raw community structures between two groups (based on the relative abundances of species). We calculated the log <sub>2</sub> FC in abundance after treatment for each species. Fold change values were normalized (within all samples in each group) ranging from 0 to 1 and further rescaled to make the sum of the normalized fold change as one. We further calculated the Bray–Curtis distance based on the profile of the normalized fold change of species abundance.

Differentially varied species after treatment with RS or CS were tested using a Wilcoxon signed-rank test, with an FDR cutoff of 0.2. The NMDS ordination analysis based on the normalized Bray–Curtis dissimilarity of species-level abundance fold change, described above, was performed with R package VEGAN <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR54">54</a></sup>. Network visualization was performed by Cytoscape 3 (ref. <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR55">55</a></sup>).

IDBA-UD <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR56">56</a></sup> was adopted de novo assemblies with *k* -mer size ranging from 20 to 100 bp. The method for functional annotation and abundance calculations for KEGG pathways is described in an online server that we developed <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR57">57</a></sup>. In brief, MetaGeneMark <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR58">58</a></sup> was used for microbial gene prediction. Metagenomic reads were mapped to assembled contigs to get gene-level DNA abundance levels with reads per kilobase per million (RPKM) mapped reads. We used KOBAS <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR59">59</a></sup> to annotate KEGG orthologues and pathways based on annotated genes. Scripts from our publish servers <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR57">57</a></sup> were used to calculate the aggregated RPKM for each pathway.

#### Metagenomics association analyses

Generalized linear modelling was performed using the R package glmulti. For each phenotype, the response variable was the fold change of the phenotype; the independent variables were from a subset of the fold change of 12 species with FDR < 0.3 in the differentially varied analysis using a Wilcoxon rank-sum test. The linear models started from ‘PHENOTYPE ~’, generating all possible models to reach a best model with feature selections. The importance for a particular species variable was calculated as the proportion of the total weights/probabilities for the models containing the variable during automatic model selection based on corrected AIC. This importance index reflects overall support of all possible weighted linear models. Only the main effects in the generalized linear models were considered. The *P* value of a particular species variable was calculate using Fisher’s method to combine the *P* value among the top ten models.

To quantify the association between (1) phenotypes and metabolites, (2) gut microbes and blood metabolites and (3) gut microbial pathways and blood metabolites, we adopted partial least-squares discriminant analysis (PLS-DA) using the mixOmics package <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR60">60</a></sup> in R. The number of components was set to 2 and number of features for each component was estimated based on hyperparameter optimization.

### Metabolomics profiling of human serum samples

The serum untargeted metabolomics data were quantified through a Shimadzu Prominence HPLC system (Shimadzu) coupled to an LTQ Orbitrap Velos instrument (Thermo Fisher Scientific) at the CAS Key Laboratory of Separation Science for Analytical Chemistry. The details of analytical conditions were as described in a previous study <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR61">61</a></sup>. The quantitation of bile acid profiles was performed by Metabo-Profile <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR62">62</a> – <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR64">64</a></sup>.

### Metabolomics profiling of human faecal samples

The metabolomics data of human faecal samples were quantified by GC–TOF–MS (Agilent 6890N GC coupled with a LECO Pegasus HT TOF–MS). The details of analytical condition as described in previous study <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR65">65</a></sup>.

### Faecal lipid assessment

Faecal lipids were extracted from the faeces of participants after 8-week RS or CS interventions using published methods <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR66">66</a>,<a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR67">67</a></sup>. Approximately 100 mg lyophilized faeces was acidified with 100 μl ethanol and 500 μl 8 N hydrochloric acid, then kept in a water bath at 80 °C for 40 min. Fatty acids were subsequently extracted with 1,250 μl diethyl ether and 1,250 μl petroleum ether. After filtering the ether-fat upper phase, the clear ether solution was collected and kept in a rotary evaporator for 1 h at 40 °C. Fatty acids were resolubilized in ethanol and NEFA, TC and TG were subsequently measured using enzymatic assays (Jiancheng, A042-1-1, A111-1-1 and A110-1-1, respectively).

### Mouse model

The Committee on the Use of Live Animals for Teaching and Research of the University of Hong Kong and the Animal Ethics Committee of Shanghai Sixth People’s Hospital approved the animal experimental procedures. Healthy C57BL/6J male mice were conventionally raised in a specific-pathogen-free barrier facility in individually ventilated cages with a maximum of four mice per cage, with free access to food and water under a strict 12-h light–dark cycle at a controlled temperature (23 ± 2°C) and 60–70% humidity. For FMT experiments, 10-week-old male C57BL/6J antibiotic-treated mice (GemParmatech, N000013) were fed a WD (Research Diet, D12451) for 2 weeks before and during the colonization. For administrating of *B.**adolescentis* to conventionally raised mice, eight-week-old males were similarly fed the WD for 8 weeks before *B.**adolescentis* administration. For the intervention of RS with *B.**adolescentis* in mice housed in a GF environment, 5-week-old male C57BL/6J mice were kept in isolators (GemParmatech, N000295) and fed a C35 diet (45% fat, 20% protein and 35% carbohydrate sourced from 20% RS or 20% CS, and the remaining 80% from maltodextrin) for 10 weeks. Meanwhile, the mice were orally inoculated with *B.**adolescentis* or PBS for the remaining 8 weeks.

### FMT experiment

Antibiotic-treated mice with microbiota depletion were given daily oral gavage with a 200-µl antibiotics cocktail (ampicillin 1 g l <sup>−1</sup>, neomycin 1 g l <sup>−1</sup>, metronidazole 1 g l <sup>−1</sup> and vancomycin 0.5 g l <sup>−1</sup>) for 7 d, followed by a 4-d antibiotic washout period before FMT. For FMT, the preparation of fresh stool samples and the subsequent operation in mice was conducted as previously described <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR43">43</a></sup>.

Body weight was measured every 3 or 4 d. Stool samples were collected before and after faecal transplantation and instantly stored at −80 °C until further analysis. Body composition was determined with a Minispec LF90 Body Composition Analyzer (Bruker). In vivo gut permeability, GTTs, ITTs and whole-body oxygen consumption were accessed via a comprehensive laboratory animal monitoring system (Columbus Instruments) <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR68">68</a></sup>. Fat mass in inguinal subcutaneous, epididymal, peri-renal and mesenteric white adipose tissue was determined after death by wet weight measurements. Blood and various tissues were collected for further biochemical evaluations. The study did not blind investigators to group allocations and no mice were excluded.

### Bacterial strain

*B.**adolescentis* strain E 298 b (Variant c) (DSMZ no. 20086) was purchased from DSMZ. The culture medium was prepared according to the DSMZ website and *B.**adolescentis* was grown in an anaerobic workstation (Gene Science AG300). *B.**adolescentis* was cultured anaerobically in sterilized DSMZ Medium 58 containing casein peptone (tryptic digest, Sigma-Aldrich), yeast extract (Sigma-Aldrich), meat extract (Sigma-Aldrich), bacto soytone (BD), glucose (Sigma-Aldrich), K <sub>2</sub> HPO <sub>4</sub>, MgSO <sub>4</sub>  × 7 H <sub>2</sub> O, MnSO <sub>4</sub>  × H <sub>2</sub> O, Tween 80, NaCl, cysteine-HCl × H <sub>2</sub> O (Sigma-Aldrich), salt solution and 0.025% resazurin (Sigma-Aldrich). The purity of cultures was monitored by Gram staining and the c.f.u. was counted by plating serial dilutions on agar plates.

### Administration of B.adolescentis

*B.**adolescentis* was collected through centrifugation at 1,500 *g* for 30 min at 4 °C, followed by a double washing with sterile PBS and subsequently resuspended in 2 ml anaerobic sterile PBS containing 20% glycerol, achieving of 5 × 10 <sup>11</sup> c.f.u. per ml. This suspension was then preserved by storage at −80 °C. Before supplementing the drinking water of mice, live *B.**adolescentis* stock was thawed at 4 °C and diluted in 100 ml autoclaved water. On average, each mouse received at least 4 × 10 <sup>10</sup> c.f.u. per day. *B.**adolescentis* was heat-killed at 121 °C under pressure of 225 kPa for 15 min. Viability confirmation tests were performed by culturing, showing that heat-killed *B.**adolescentis* did not grow, whereas live *B.**adolescentis* grew well. Drinking water was changed daily in the three groups during the experiment. In addition, GF mice were treated with PBS or 1 × 10 <sup>9</sup> c.f.u. of live *B.**adolescentis* in 200 μl sterile anaerobic PBS by gavage three times per week for 8 weeks.

### Biochemical and immunological assays in mice

Serum levels of TC and TG as well as intrahepatic, intestinal and faecal TG were measured by enzymatic analysis (Stanbio Laboratory, 2100430 and 1100430). Serum NEFA was measured by enzymatic analysis (Roche Diagnostics). Serum adiponectin (AIS, HKU, 32010), ANGPTL4 (Abcam, ab210577), inflammatory serum molecules (eBioscience, MCP-1, BMS6005; IL-1β, BMS6002; IL-6, BMS603-2; IL-10, BMS614INST) and FGF21 (Immunodiagnostics Limited, 32180) were measured using ELISA. LPS levels in mesenteric white adipose tissue and serum were measured by LAL assay (Hycult Biotechnology, HIT302).

### Metabolomics profiling of mice samples

#### Amio acid profiling

Target amino acids were quantified using Waters AccQ-Tag derivation kit with 20 μl serum by A Nexera X2 ultra HPLC equipped with a triple quadrupole 8050 (Shimadzu). The details of analytical condition as described in previous study <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR61">61</a></sup>.

#### Bile acid profiling

Target bile acids were analysed by A Vanquish UPLC-Q Exactive (Thermo Fisher Scientific) system. Bile acids in THE serum sample (50 μl) was extracted by 200 μl methanol containing internal standards. Peak detection and integration were carried out by using Trancefinder (Thermo Fisher). The results quantified by the standard curve were used for feature analysis. The details of sample preparation and analytical conditions are as described in a previous study <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR69">69</a></sup>.

#### SCFA profiling

For sample pretreatment, 30 μl serum was mixed with 60 μl acetonitrile containing the internal standard 0.3 μg ml <sup>−1</sup> butyric acid-d7. After vortexing and centrifuging, the supernatant was aspirated for subsequent analysis. For the faecal sample, 40 mg faecal with grinding ball was mixed with 200 μl of acetonitrile containing internal standard extractant, internal standard Butyric-d7 acid 20 μmol l <sup>−1</sup>. After grinding and centrifugation, the upper supernatant was aspirated for subsequent analysis.

For analytical measurement, an Agilent 5977A instrument equipped with an MSD gas chromatography mass spectrometry (GC–MS) system (Agilent) was used for SCFA analysis. A DB-FFAP column (30 m × 250 μm × 0.25 μm, Agilent) was adopted for GC separation. The initial temperature was 50 °C and maintained for 1 min, then increased to 180 °C at 10 °C min <sup>−1</sup>, immediately raised to 250 °C at 30 °C min <sup>−1</sup> and maintained for 2 min. The flow rate of helium carrier gas was kept at 40 cm s <sup>−1</sup>. The temperatures of the injector, ion source, quadrupole and interface were 250, 230, 150 and 250 °C, respectively. The selected ion monitoring mode was performed by detecting the protonated molecules at *m* / *z* 43 for acetic acid and isobutyric acid, *m* / *z* 74 for propionic acid and *m* / *z* 60 for other SCFAs. Peak detection and integration were performed using Quantitative Analysis (Agilent). The results quantified by the standard curve are used for feature analysis. All the metabolomic profiles in mice were performed at the CAS Key Laboratory of Separation Science for Analytical Chemistry.

### FGF21 response test

After an 8-week *B.**adolescentis* intervention period, an FGF21 response test was conducted in conventionally raised mice. Under general anaesthesia, a subcutaneous fat biopsy was taken and flash-frozen in liquid nitrogen. Subsequently, 2 mg kg <sup>−1</sup> recombinant mouse FGF21 or a vehicle was intravenously injected via the inferior vena cava. At 15 min later, a second subcutaneous fat biopsy was taken to assess gene expression and Erk1/2 phosphorylation.

### Quantitative PCR assays

The total RNA of tissue samples was extracted by TRIzol (Invitrogen) and reverse transcribed into complementary DNA using ImProm-II reverse transcriptase (Promega). Quantitative PCR with reverse transcription (RT–qPCR) reactions were performed using the Quantifast SYBR Green master mix (QIAGEN) on a Light Cycler 480 system (Roche), normalized with mouse GAPDH. Primers for each gene and *B.**adolescentis* <sup><a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#CR70">70</a></sup> are listed in Supplementary Table [11](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM3).

### Histological examination

Ileum sections were treated with ZO-1 or occludin antibodies (Abcam, 1:500 dilution, ab96587; 1:100 dilution, ab216327), followed by incubation with Alex Fluor-596 or FITC-conjugated secondary antibodies (Life Technologies) and counterstaining with DAPI. Ileum sections were treated with ANGPTL4 antibody (Proteintech, 1:500 dilution, 18374-1-AP), followed by incubation with horseradish peroxidase-labelled secondary antibodies (DAKO), developed by DAB following the manufacturer’s recommendations. Five random fields were selected from different regions of individual sections for image quantitation. ImageJ software was used to analyse the intensity of positive staining, represented as a percentage of the total area of the lesion or villi in every field.

### Luminal lipase activity

The mixture of luminal content from the small intestine and ice-cold PBS was vortexed thoroughly, followed centrifugation at 15,000 *g* at 4 °C for 10 min. Supernatants were collected and diluted 1:100 in PBS for lipase activity measurement via lipase activity assay kit (Sigma, MAK109) following the manufacturer’s protocol.

### Western blotting

WAT proteins were extracted using a lysis buffer containing protease inhibitors (ST505, Beyotime Biotechnology) and phosphatase inhibitors (P1082, Beyotime Biotechnology). Protein samples were resolved by SDS–PAGE (8%) and immunoblotted onto polyvinylidene difluoride membranes. Immunoblotting was conducted using extracellular signal-regulated protein kinase 1/2 (Erk1/2) (Cell Signalling Technology, 1:1,000 dilution, 9102) and phosphorylated Erk1/2 (Thr202/Tyr204) (Cell Signalling Technology, 1:1,000 dilution, 9101). Immunoreactivity was detected using enhanced chemiluminescent autoradiography (Millipore). Band quantification was performed using ImageJ software.

### Statistical analysis

The statistical analysis plan for this study is available in the [Supplementary Information](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM1). EpiData v.3.1 facilitated clinical data entry and documentation. Statistical analyses unless otherwise stated, utilized SPSS v.17 and R v.3.3.2 software. Normally distributed clinical data were presented as mean ± s.d. Non-normally distributed data, verified by the Kolmogorov–Smirnov test, were presented as medians and IQRs. The intervention’s effects on outcomes were analysed using a linear mixed model adjusted for age, sex and intervention order, with Bonferroni adjustment for multiple testing correction. In animal studies, data distribution and variance equality were assessed using a Shapiro–Wilk test and Levene’s test. For two-group comparisons, a two-tailed Student’s unpaired *t-* test (normally distributed) or nonparametric Wilcoxon rank-sum test (non-normally distributed) was applied. Among multiple groups, a one-way ANOVA (normally distributed) followed by Tukey’s post hoc test or Kruskal–Wallis test (non-normally distributed) followed by Dunn’s test was used. In the clinical study, based on literature and preliminary data, the expected change in body weight and its s.d. after RS intervention was estimated as 2 ± 2.8 kg. A two-tailed test with a 0.05 significance level and 80% power required a minimum sample size of 31. Thirty-seven participants were invited to account for an expected 17% dropout post-randomization. For animal studies, sample sizes were estimated based on previous research and assay variability. Two-sided *P* values < 0.05 were deemed statistically significant.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#MOESM2) linked to this article.

### Supplementary information

[Supplementary Information](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM1_ESM.pdf) <sup>(681.4KB, pdf)</sup>

Study protocol and statistical analysis plan.

[Reporting Summary](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM2_ESM.pdf) <sup>(2.9MB, pdf)</sup>

[Supplementary Table 1](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM3_ESM.xlsx) <sup>(111.9KB, xlsx)</sup>

Supplementary Tables 1–11.

### Source data

[Source Data Fig. 2](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM4_ESM.xlsx) <sup>(13.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 3](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM5_ESM.xlsx) <sup>(14.5KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 4](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM6_ESM.xlsx) <sup>(15.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 5](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM7_ESM.xlsx) <sup>(22.5KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 6](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM8_ESM.xlsx) <sup>(17.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 7](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM9_ESM.xlsx) <sup>(21.8KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 3](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM10_ESM.xlsx) <sup>(45.9KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 4](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM11_ESM.xlsx) <sup>(11.6KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 5](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM12_ESM.xlsx) <sup>(10.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 6](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM13_ESM.xlsx) <sup>(12.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 7](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM14_ESM.xlsx) <sup>(31.7KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 8](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM15_ESM.xlsx) <sup>(16.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 8](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM16_ESM.pdf) <sup>(781.1KB, pdf)</sup>

Unprocessed western gels.

[Source Data Extended Data Fig. 9](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM17_ESM.xlsx) <sup>(16.9KB, xlsx)</sup>

Statistical Source Data.

## Acknowledgements

We thank the medical staff and study participants who took part in the trial. The authors thank J. Wu and R. Zeng for valuable suggestions. This work was supported by the National Key Research and Development Program of China (2022YFA1004804), Shanghai Municipal Key Clinical Specialty, Shanghai Research Center for Endocrine and Metabolic Diseases (2022ZZ01002) and National Natural Science Foundation of China (NSFC) major international (regional) joint research project (81220108006) to W.J.; Excellent Young Scientists Fund of NSFC (82022012), General Fund of NSFC (82270907), Major Program of NSFC (92357305), Innovative Research Team of High-level Local Universities in Shanghai (SHSMU-ZDCX20212700), Hong Kong Scholars Program (XJ2013035) and Two Hundred Program from Shanghai Jiao Tong University School of Medicine to H.L.; Hong Kong Research Grant Council (AOE/M/707-18) to A.X.; Marie Sklodowska-Curie Actions and Innovative Training Networks (H2020-MSCA-ITN-2018 813781) to G.P., DFG under Germany’s Excellence Strategy (EXC 2051) project ID 390713860 to G.P. and Y.N.; Shenzhen Basic Research Program (JCYJ20190808182402941) and Guangdong Basic and Applied Research Major Program (2019B030302005) to J.L.; Key foundation of NSFC (21934006), Strategic Priority Research Program (B) of the Chinese Academy of Sciences (XDB38020200) to G.X.; the Youth Innovation Promotion Association of the Chinese Academy of Sciences (2021186) to X. Liu.; and The Hong Kong Polytechnic University (P0042740) to P.L.

### Extended data

## Author contributions

W.J., A.X., G.P., H.L. and X. Liu conceived and designed the project. H.L., L.Z., J.L. and Q.W. managed the study. H.L., L.Z. and L.Q. made clinical diagnoses, recruited participants and performed interventions. L.S. supported the study as a clinical nutritionist. L.Z., L.Q. and L.W. collected samples and clinical phenotypes. G.P., J.L., Y.N., K.K. and J.H. performed bioinformatics analyses. X. Liu., G.X., P.K.-D., X.W., Y.L. and Y.Y. performed metabolomics profiling and data analysis. L.Z., Q.W., L.Q., X. Long., M.Z., Q.F., R.Y., S.L. and Y.Z. conducted animal experiments. B.S. and P.L. performed medical image analysis. H.L., L.Z., J.L. and Q.W. wrote the manuscript. W.J., A.X., G.P., P.K.-D., J.Y. and Y.B. reviewed and edited the manuscript. The contributions and authorship criteria have been carefully considered in this multi-region collaboration.

## Peer review

### Peer review information

*Nature Metabolism* thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editor: Yanina-Yasmin Pesch, in collaboration with the *Nature Metabolism* team.

## Data availability

Individual-level patient data are accessible with the consent of the Data Management Committee from the institutions and are not publicly available. The Data Management Committee will review all requests and grant access (if successful). A formal data transfer agreement will be required upon approval. Generally, all requests for access to data will be responded to within 1 month. All data shared will be de-identified. Raw metagenomic sequencing data have been deposited in the NCBI Sequence Read Archive under accession ID [PRJNA414688](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA414688) and are publicly available as of the date of publication. Any additional information required to reanalyse the data reported in this paper is available from the lead contact W.J. (wpjia@sjtu.edu.cn) upon request. [Source data](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Sec44) are provided with this paper.

## Code availability

No custom code was used in this paper. All publicly available codes and tools used to analyse the data are reported and referenced in the [Methods](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Sec12).

## Competing interests

The authors declare no competing interests.

## Footnotes

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

These authors contributed equally: Huating Li, Lei Zhang, Jun Li, Qian Wu.

## Contributor Information

Huating Li, Email: huarting99@sjtu.edu.cn.

Xinyu Liu, Email: liuxy2012@dicp.ac.cn.

Gianni Panagiotou, Email: Gianni.Panagiotou@hki-jena.de.

Aimin Xu, Email: amxu@hku.hk.

Weiping Jia, Email: wpjia@sjtu.edu.cn.

### Extended data

is available for this paper at 10.1038/s42255-024-00988-y.

### Supplementary information

The online version contains supplementary material available at 10.1038/s42255-024-00988-y.

## References

- 1.Waxman A, World Health A. WHO global strategy on diet, physical activity and health. Food Nutr. Bull. 2004;25:292–302. doi: 10.1177/156482650402500310. \[[DOI](https://doi.org/10.1177/156482650402500310)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15460274/)\] \[\]
- 2.Torres-Fuentes C, Schellekens H, Dinan TG, Cryan JF. The microbiota-gut-brain axis in obesity. Lancet Gastroenterol. Hepatol. 2017;2:747–756. doi: 10.1016/S2468-1253(17)30147-4. \[[DOI](https://doi.org/10.1016/S2468-1253\(17\)30147-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28844808/)\] \[\]
- 3.Backhed F, et al. The gut microbiota as an environmental factor that regulates fat storage. Proc. Natl Acad. Sci. USA. 2004;101:15718–15723. doi: 10.1073/pnas.0407076101. \[[DOI](https://doi.org/10.1073/pnas.0407076101)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC524219/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15505215/)\] \[\]
- 4.Turnbaugh PJ, et al. An obesity-associated gut microbiome with increased capacity for energy harvest. Nature. 2006;444:1027–1031. doi: 10.1038/nature05414. \[[DOI](https://doi.org/10.1038/nature05414)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17183312/)\] \[\]
- 5.Kootte RS, et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal microbiota composition. Cell Metab. 2017;26:611–619 e616. doi: 10.1016/j.cmet.2017.09.008. \[[DOI](https://doi.org/10.1016/j.cmet.2017.09.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28978426/)\] \[\]
- 6.de Groot P, et al. Donor metabolic characteristics drive effects of faecal microbiota transplantation on recipient insulin sensitivity, energy expenditure and intestinal transit time. Gut. 2020;69:502–512. doi: 10.1136/gutjnl-2019-318320. \[[DOI](https://doi.org/10.1136/gutjnl-2019-318320)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7034343/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31147381/)\] \[\]
- 7.Carmody RN, Bisanz JE. Roles of the gut microbiome in weight management. Nat. Rev. Microbiol. 2023;21:535–550. doi: 10.1038/s41579-023-00888-0. \[[DOI](https://doi.org/10.1038/s41579-023-00888-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/37138047/)\] \[\]
- 8.Gibson GR, et al. Expert consensus document: The International Scientific Association for Probiotics and Prebiotics (ISAPP) consensus statement on the definition and scope of prebiotics. Nat. Rev. Gastroenterol. Hepatol. 2017;14:491–502. doi: 10.1038/nrgastro.2017.75. \[[DOI](https://doi.org/10.1038/nrgastro.2017.75)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28611480/)\] \[\]
- 9.Delannoy-Bruno O, et al. Evaluating microbiome-directed fibre snacks in gnotobiotic mice and humans. Nature. 2021;595:91–95. doi: 10.1038/s41586-021-03671-4. \[[DOI](https://doi.org/10.1038/s41586-021-03671-4)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8324079/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34163075/)\] \[\]
- 10.Bindels LB, Delzenne NM, Cani PD, Walter J. Towards a more comprehensive concept for prebiotics. Nat. Rev. Gastroenterol. Hepatol. 2015;12:303–310. doi: 10.1038/nrgastro.2015.47. \[[DOI](https://doi.org/10.1038/nrgastro.2015.47)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25824997/)\] \[\]
- 11.Englyst HN, Cummings JH. Digestion of the polysaccharides of some cereal foods in the human small intestine. Am. J. Clin. Nutr. 1985;42:778–787. doi: 10.1093/ajcn/42.5.778. \[[DOI](https://doi.org/10.1093/ajcn/42.5.778)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2998174/)\] \[\]
- 12.Higgins JA. Resistant starch and energy balance: impact on weight loss and maintenance. Crit. Rev. Food Sci. Nutr. 2014;54:1158–1166. doi: 10.1080/10408398.2011.629352. \[[DOI](https://doi.org/10.1080/10408398.2011.629352)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4220782/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24499148/)\] \[\]
- 13.Wali JA, et al. Impact of dietary carbohydrate type and protein-carbohydrate interaction on metabolic health. Nat. Metab. 2021;3:810–828. doi: 10.1038/s42255-021-00393-9. \[[DOI](https://doi.org/10.1038/s42255-021-00393-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34099926/)\] \[\]
- 14.Robertson MD, et al. Insulin-sensitizing effects on muscle and adipose tissue after dietary fiber intake in men and women with metabolic syndrome. J. Clin. Endocrinol. Metab. 2012;97:3326–3332. doi: 10.1210/jc.2012-1513. \[[DOI](https://doi.org/10.1210/jc.2012-1513)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22745235/)\] \[\]
- 15.Johnston KL, Thomas EL, Bell JD, Frost GS, Robertson MD. Resistant starch improves insulin sensitivity in metabolic syndrome. Diabet. Med. 2010;27:391–397. doi: 10.1111/j.1464-5491.2010.02923.x. \[[DOI](https://doi.org/10.1111/j.1464-5491.2010.02923.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20536509/)\] \[\]
- 16.Maki KC, et al. Resistant starch from high-amylose maize increases insulin sensitivity in overweight and obese men. J. Nutr. 2012;142:717–723. doi: 10.3945/jn.111.152975. \[[DOI](https://doi.org/10.3945/jn.111.152975)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3301990/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22357745/)\] \[\]
- 17.Coate KC, Huggins KW. Consumption of a high glycemic index diet increases abdominal adiposity but does not influence adipose tissue pro-oxidant and antioxidant gene expression in C57BL/6 mice. Nutr. Res. 2010;30:141–150. doi: 10.1016/j.nutres.2010.01.003. \[[DOI](https://doi.org/10.1016/j.nutres.2010.01.003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20227000/)\] \[\]
- 18.Charrier JA, et al. High fat diet partially attenuates fermentation responses in rats fed resistant starch from high-amylose maize. Obesity. 2013;21:2350–2355. doi: 10.1002/oby.20362. \[[DOI](https://doi.org/10.1002/oby.20362)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5225625/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23512798/)\] \[\]
- 19.DeMartino P, Cockburn DW. Resistant starch: impact on the gut microbiome and health. Curr. Opin. Biotechnol. 2020;61:66–71. doi: 10.1016/j.copbio.2019.10.008. \[[DOI](https://doi.org/10.1016/j.copbio.2019.10.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31765963/)\] \[\]
- 20.Chen C, Lu FC, Department of Disease Control Ministry of Health, PR China. The guidelines for prevention and control of overweight and obesity in Chinese adults. Biomed. Environ. Sci. 2004;17:1–36. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15807475/)\] \[\]
- 21.Jensen MD, et al. 2013 AHA/ACC/TOS guideline for the management of overweight and obesity in adults: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines and The Obesity Society. Circulation. 2014;129:S102–S138. doi: 10.1161/01.cir.0000437739.71477.ee. \[[DOI](https://doi.org/10.1161/01.cir.0000437739.71477.ee)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5819889/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24222017/)\] \[\]
- 22.Cox AJ, West NP, Cripps AW. Obesity, inflammation, and the gut microbiota. Lancet Diabetes Endocrinol. 2015;3:207–215. doi: 10.1016/S2213-8587(14)70134-2. \[[DOI](https://doi.org/10.1016/S2213-8587\(14\)70134-2)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25066177/)\] \[\]
- 23.Martinez-Guryn K, et al. Small intestine microbiota regulate host digestive and absorptive adaptive responses to dietary lipids. Cell Host Microbe. 2018;23:458–469. doi: 10.1016/j.chom.2018.03.011. \[[DOI](https://doi.org/10.1016/j.chom.2018.03.011)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5912695/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29649441/)\] \[\]
- 24.Geng L, Lam KSL, Xu A. The therapeutic potential of FGF21 in metabolic diseases: from bench to clinic. Nat. Rev. Endocrinol. 2020;16:654–667. doi: 10.1038/s41574-020-0386-0. \[[DOI](https://doi.org/10.1038/s41574-020-0386-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32764725/)\] \[\]
- 25.Baxter NT, et al. Dynamics of human gut microbiota and short-chain fatty acids in response to dietary interventions with three fermentable fibers. mBio. 2019;10:e02566–18. doi: 10.1128/mBio.02566-18. \[[DOI](https://doi.org/10.1128/mBio.02566-18)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6355990/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30696735/)\] \[\]
- 26.Kadyan S, Sharma A, Arjmandi BH, Singh P, Nagpal R. Prebiotic potential of dietary beans and pulses and their resistant starch for aging-associated gut and metabolic health. Nutrients. 2022;14:1726. doi: 10.3390/nu14091726. \[[DOI](https://doi.org/10.3390/nu14091726)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC9100130/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35565693/)\] \[\]
- 27.Vital M, et al. Metagenomic insights into the degradation of resistant starch by human gut microbiota. Appl. Environ. Microbiol. 2018;84:e01562–18. doi: 10.1128/AEM.01562-18. \[[DOI](https://doi.org/10.1128/AEM.01562-18)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6238065/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30266729/)\] \[\]
- 28.Krautkramer KA, Fan J, Backhed F. Gut microbial metabolites as multi-kingdom intermediates. Nat. Rev. Microbiol. 2021;19:77–94. doi: 10.1038/s41579-020-0438-4. \[[DOI](https://doi.org/10.1038/s41579-020-0438-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32968241/)\] \[\]
- 29.Heianza Y, et al. Changes in gut microbiota-related metabolites and long-term successful weight loss in response to weight-loss diets: the POUNDS Lost trial. Diabetes Care. 2018;41:413–419. doi: 10.2337/dc17-2108. \[[DOI](https://doi.org/10.2337/dc17-2108)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5829970/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29305401/)\] \[\]
- 30.Adams SH. Emerging perspectives on essential amino acid metabolism in obesity and the insulin-resistant state. Adv. Nutr. 2011;2:445–456. doi: 10.3945/an.111.000737. \[[DOI](https://doi.org/10.3945/an.111.000737)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3226382/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22332087/)\] \[\]
- 31.Foley MH, O’Flaherty S, Barrangou R, Theriot CM. Bile salt hydrolases: gatekeepers of bile acid metabolism and host-microbiome crosstalk in the gastrointestinal tract. PLoS Pathog. 2019;15:e1007581. doi: 10.1371/journal.ppat.1007581. \[[DOI](https://doi.org/10.1371/journal.ppat.1007581)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6405046/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30845232/)\] \[\]
- 32.Mattijssen F, et al. Angptl4 serves as an endogenous inhibitor of intestinal lipid digestion. Mol. Metab. 2014;3:135–144. doi: 10.1016/j.molmet.2013.11.004. \[[DOI](https://doi.org/10.1016/j.molmet.2013.11.004)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3953698/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24634819/)\] \[\]
- 33.Higgins JA, et al. Resistant starch and exercise independently attenuate weight regain on a high fat diet in a rat model of obesity. Nutr. Metab. 2011;8:49. doi: 10.1186/1743-7075-8-49. \[[DOI](https://doi.org/10.1186/1743-7075-8-49)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3152509/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21736742/)\] \[\]
- 34.Wahlstrom A, Sayin SI, Marschall HU, Backhed F. Intestinal crosstalk between bile acids and microbiota and its impact on host metabolism. Cell Metab. 2016;24:41–50. doi: 10.1016/j.cmet.2016.05.005. \[[DOI](https://doi.org/10.1016/j.cmet.2016.05.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27320064/)\] \[\]
- 35.Upadhyaya B, et al. Impact of dietary resistant starch type 4 on human gut microbiota and immunometabolic functions. Sci. Rep. 2016;6:28797. doi: 10.1038/srep28797. \[[DOI](https://doi.org/10.1038/srep28797)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4928084/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27356770/)\] \[\]
- 36.Ravussin Y, et al. Responses of gut microbiota to diet composition and weight loss in lean and obese mice. Obesity. 2012;20:738–747. doi: 10.1038/oby.2011.111. \[[DOI](https://doi.org/10.1038/oby.2011.111)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3871199/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21593810/)\] \[\]
- 37.Zhang L, et al. Metabolic phenotypes and the gut microbiota in response to dietary resistant starch type 2 in normal-weight subjects: a randomized crossover trial. Sci. Rep. 2019;9:4736. doi: 10.1038/s41598-018-38216-9. \[[DOI](https://doi.org/10.1038/s41598-018-38216-9)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6426958/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30894560/)\] \[\]
- 38.Bendiks ZA, Knudsen KEB, Keenan MJ, Marco ML. Conserved and variable responses of the gut microbiome to resistant starch type 2. Nutr. Res. 2020;77:12–28. doi: 10.1016/j.nutres.2020.02.009. \[[DOI](https://doi.org/10.1016/j.nutres.2020.02.009)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7295659/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32251948/)\] \[\]
- 39.Golden JM, et al. Ursodeoxycholic acid protects against intestinal barrier breakdown by promoting enterocyte migration via EGFR- and COX-2-dependent mechanisms. Am. J. Physiol. Gastrointest. Liver Physiol. 2018;315:G259–G271. doi: 10.1152/ajpgi.00354.2017. \[[DOI](https://doi.org/10.1152/ajpgi.00354.2017)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6139640/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29672156/)\] \[\]
- 40.Quintero P, et al. Bile acid supplementation improves established liver steatosis in obese mice independently of glucagon-like peptide-1 secretion. J. Physiol. Biochem. 2014;70:667–674. doi: 10.1007/s13105-014-0336-1. \[[DOI](https://doi.org/10.1007/s13105-014-0336-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24816727/)\] \[\]
- 41.Nie B, et al. Specific bile acids inhibit hepatic fatty acid uptake in mice. Hepatology. 2012;56:1300–1310. doi: 10.1002/hep.25797. \[[DOI](https://doi.org/10.1002/hep.25797)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3445775/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22531947/)\] \[\]
- 42.Schneider KM, Albers S, Trautwein C. Role of bile acids in the gut–liver axis. J. Hepatol. 2018;68:1083–1085. doi: 10.1016/j.jhep.2017.11.025. \[[DOI](https://doi.org/10.1016/j.jhep.2017.11.025)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29519549/)\] \[\]
- 43.Ni Y, et al. Resistant starch decreases intrahepatic triglycerides in patients with NAFLD via gut microbiome alterations. Cell Metab. 2023;35:1530–1547. doi: 10.1016/j.cmet.2023.08.002. \[[DOI](https://doi.org/10.1016/j.cmet.2023.08.002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/37673036/)\] \[\]
- 44.Jaskiewicz J, et al. Catabolism of isobutyrate by colonocytes. Arch. Biochem. Biophys. 1996;327:265–270. doi: 10.1006/abbi.1996.0120. \[[DOI](https://doi.org/10.1006/abbi.1996.0120)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8619613/)\] \[\]
- 45.Oliphant K, Allen-Vercoe E. Macronutrient metabolism by the human gut microbiome: major fermentation by-products and their impact on host health. Microbiome. 2019;7:91. doi: 10.1186/s40168-019-0704-8. \[[DOI](https://doi.org/10.1186/s40168-019-0704-8)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC6567490/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31196177/)\] \[\]
- 46.Dijk W, Kersten S. Regulation of lipoprotein lipase by Angptl4. Trends Endocrinol. Metab. 2014;25:146–155. doi: 10.1016/j.tem.2013.12.005. \[[DOI](https://doi.org/10.1016/j.tem.2013.12.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24397894/)\] \[\]
- 47.Long X, et al. Bifidobacterium adolescentis alleviates liver steatosis and steatohepatitis by increasing fibroblast growth factor 21 sensitivity. Front. Endocrinol. 2021;12:773340. doi: 10.3389/fendo.2021.773340. \[[DOI](https://doi.org/10.3389/fendo.2021.773340)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC8756294/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35035378/)\] \[\]
- 48.Geng L, et al. Exercise alleviates obesity-induced metabolic dysfunction via enhancing FGF21 sensitivity in adipose tissues. Cell Rep. 2019;26:2738–2752. doi: 10.1016/j.celrep.2019.02.014. \[[DOI](https://doi.org/10.1016/j.celrep.2019.02.014)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30840894/)\] \[\]
- 49.van Baak MA, Mariman ECM. Mechanisms of weight regain after weight loss - the role of adipose tissue. Nat. Rev. Endocrinol. 2019;15:274–287. doi: 10.1038/s41574-018-0148-4. \[[DOI](https://doi.org/10.1038/s41574-018-0148-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30655624/)\] \[\]
- 50.Jia W, et al. Association of serum retinol-binding protein 4 and visceral adiposity in Chinese subjects with and without type 2 diabetes. J. Clin. Endocrinol. Metab. 2007;92:3224–3229. doi: 10.1210/jc.2007-0209. \[[DOI](https://doi.org/10.1210/jc.2007-0209)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17550959/)\] \[\]
- 51.Li J, et al. Probiotics modulated gut microbiota suppresses hepatocellular carcinoma growth in mice. Proc. Natl Acad. Sci. USA. 2016;113:E1306–E1315. doi: 10.1073/pnas.1518189113. \[[DOI](https://doi.org/10.1073/pnas.1518189113)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4780612/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26884164/)\] \[\]
- 52.Li H, Durbin R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics. 2009;25:1754–1760. doi: 10.1093/bioinformatics/btp324. \[[DOI](https://doi.org/10.1093/bioinformatics/btp324)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC2705234/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19451168/)\] \[\]
- 53.Segata N, et al. Metagenomic microbial community profiling using unique clade-specific marker genes. Nat. Methods. 2012;9:811–814. doi: 10.1038/nmeth.2066. \[[DOI](https://doi.org/10.1038/nmeth.2066)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3443552/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22688413/)\] \[\]
- 54.Dixon P. VEGAN, a package of R functions for community ecology. J. Vegetation Sci. 2003;14:927–930. \[\]
- 55.Shannon P, et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2003;13:2498–2504. doi: 10.1101/gr.1239303. \[[DOI](https://doi.org/10.1101/gr.1239303)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC403769/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14597658/)\] \[\]
- 56.Peng Y, Leung HC, Yiu SM, Chin FY. IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth. Bioinformatics. 2012;28:1420–1428. doi: 10.1093/bioinformatics/bts174. \[[DOI](https://doi.org/10.1093/bioinformatics/bts174)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22495754/)\] \[\]
- 57.Ni Y, Li J, Panagiotou G. COMAN: a web server for comprehensive metatranscriptomics analysis. BMC Genomics. 2016;17:622. doi: 10.1186/s12864-016-2964-z. \[[DOI](https://doi.org/10.1186/s12864-016-2964-z)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4982211/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27515514/)\] \[\]
- 58.Zhu W, Lomsadze A, Borodovsky M. Ab initio gene identification in metagenomic sequences. Nucleic Acids Res. 2010;38:e132. doi: 10.1093/nar/gkq275. \[[DOI](https://doi.org/10.1093/nar/gkq275)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC2896542/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20403810/)\] \[\]
- 59.Xie C, et al. KOBAS 2.0: a web server for annotation and identification of enriched pathways and diseases. Nucleic Acids Res. 2011;39:W316–W322. doi: 10.1093/nar/gkr483. \[[DOI](https://doi.org/10.1093/nar/gkr483)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3125809/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21715386/)\] \[\]
- 60.Rohart F, Gautier B, Singh A, Le Cao KA. mixOmics: an R package for 'omics feature selection and multiple data integration. PLoS Comput. Biol. 2017;13:e1005752. doi: 10.1371/journal.pcbi.1005752. \[[DOI](https://doi.org/10.1371/journal.pcbi.1005752)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5687754/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29099853/)\] \[\]
- 61.Liu R, et al. Gut microbiome and serum metabolome alterations in obesity and after weight-loss intervention. Nat. Med. 2017;23:859–868. doi: 10.1038/nm.4358. \[[DOI](https://doi.org/10.1038/nm.4358)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28628112/)\] \[\]
- 62.Xie G, et al. Dysregulated hepatic bile acids collaboratively promote liver carcinogenesis. Int J. Cancer. 2016;139:1764–1775. doi: 10.1002/ijc.30219. \[[DOI](https://doi.org/10.1002/ijc.30219)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5493524/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27273788/)\] \[\]
- 63.Xie G, et al. Profiling of serum bile acids in a healthy Chinese population using UPLC-MS/MS. J. Proteome Res. 2015;14:850–859. doi: 10.1021/pr500920q. \[[DOI](https://doi.org/10.1021/pr500920q)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25581415/)\] \[\]
- 64.Lan K, et al. Key role for the 12-hydroxy group in the negative ion fragmentation of unconjugated c24 bile acids. Anal. Chem. 2016;88:7041–7048. doi: 10.1021/acs.analchem.6b00573. \[[DOI](https://doi.org/10.1021/acs.analchem.6b00573)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5511518/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27322813/)\] \[\]
- 65.Zhao L, et al. High throughput and quantitative measurement of microbial metabolome by gas chromatography/mass spectrometry using automated alkyl chloroformate derivatization. Anal. Chem. 2017;89:5565–5577. doi: 10.1021/acs.analchem.7b00660. \[[DOI](https://doi.org/10.1021/acs.analchem.7b00660)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC5663283/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28437060/)\] \[\]
- 66.Govers MJ, Van der Meet R. Effects of dietary calcium and phosphate on the intestinal interactions between calcium, phosphate, fatty acids, and bile acids. Gut. 1993;34:365–370. doi: 10.1136/gut.34.3.365. \[[DOI](https://doi.org/10.1136/gut.34.3.365)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC1374143/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8472985/)\] \[\]
- 67.Cho KD, Han CK, Lee BH. Loss of body weight and fat and improved lipid profiles in obese rats fed apple pomace or apple juice concentrate. J. Med. Food. 2013;16:823–830. doi: 10.1089/jmf.2013.2784. \[[DOI](https://doi.org/10.1089/jmf.2013.2784)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC3778949/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23909905/)\] \[\]
- 68.Wong CM, et al. Adropin is a brain membrane-bound protein regulating physical activity via the NB-3/Notch signaling pathway in mice. J. Biol. Chem. 2014;289:25976–25986. doi: 10.1074/jbc.M114.576058. \[[DOI](https://doi.org/10.1074/jbc.M114.576058)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC4162195/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25074942/)\] \[\]
- 69.Zhang Y, et al. Gut microbiome-related effects of berberine and probiotics on type 2 diabetes (the PREMOTE study) Nat. Commun. 2020;11:5015. doi: 10.1038/s41467-020-18414-8. \[[DOI](https://doi.org/10.1038/s41467-020-18414-8)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC7538905/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33024120/)\] \[\]
- 70.Haarman M, Knol J. Quantitative real-time PCR assays to identify and quantify fecal Bifidobacterium species in infants receiving a prebiotic infant formula. Appl. Environ. Microbiol. 2005;71:2318–2324. doi: 10.1128/AEM.71.5.2318-2324.2005. \[[DOI](https://doi.org/10.1128/AEM.71.5.2318-2324.2005)\] \[[PMC free article](https://pmc.ncbi.nlm.nih.gov/articles/PMC1087546/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15870317/)\] \[\]

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

[Supplementary Information](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM1_ESM.pdf) <sup>(681.4KB, pdf)</sup>

Study protocol and statistical analysis plan.

[Reporting Summary](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM2_ESM.pdf) <sup>(2.9MB, pdf)</sup>

[Supplementary Table 1](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM3_ESM.xlsx) <sup>(111.9KB, xlsx)</sup>

Supplementary Tables 1–11.

[Source Data Fig. 2](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM4_ESM.xlsx) <sup>(13.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 3](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM5_ESM.xlsx) <sup>(14.5KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 4](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM6_ESM.xlsx) <sup>(15.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 5](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM7_ESM.xlsx) <sup>(22.5KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 6](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM8_ESM.xlsx) <sup>(17.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Fig. 7](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM9_ESM.xlsx) <sup>(21.8KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 3](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM10_ESM.xlsx) <sup>(45.9KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 4](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM11_ESM.xlsx) <sup>(11.6KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 5](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM12_ESM.xlsx) <sup>(10.3KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 6](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM13_ESM.xlsx) <sup>(12.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 7](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM14_ESM.xlsx) <sup>(31.7KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 8](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM15_ESM.xlsx) <sup>(16.1KB, xlsx)</sup>

Statistical Source Data.

[Source Data Extended Data Fig. 8](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM16_ESM.pdf) <sup>(781.1KB, pdf)</sup>

Unprocessed western gels.

[Source Data Extended Data Fig. 9](https://pmc.ncbi.nlm.nih.gov/articles/instance/10963277/bin/42255_2024_988_MOESM17_ESM.xlsx) <sup>(16.9KB, xlsx)</sup>

Statistical Source Data.

### Data Availability Statement

Individual-level patient data are accessible with the consent of the Data Management Committee from the institutions and are not publicly available. The Data Management Committee will review all requests and grant access (if successful). A formal data transfer agreement will be required upon approval. Generally, all requests for access to data will be responded to within 1 month. All data shared will be de-identified. Raw metagenomic sequencing data have been deposited in the NCBI Sequence Read Archive under accession ID [PRJNA414688](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA414688) and are publicly available as of the date of publication. Any additional information required to reanalyse the data reported in this paper is available from the lead contact W.J. (wpjia@sjtu.edu.cn) upon request. [Source data](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Sec44) are provided with this paper.

No custom code was used in this paper. All publicly available codes and tools used to analyse the data are reported and referenced in the [Methods](https://pmc.ncbi.nlm.nih.gov/articles/PMC10963277/#Sec12).